diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index d167c7ab..876315c8 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -23,7 +23,6 @@ class ArticleAdminController extends BaseController
     public function new(TranslatorInterface $translator, EntityManagerInterface $em, Request $request)
     {
         $article = new Article();
-        $article->setCurrentLocale($request->getSession()->get(TranslatableLocaleSubscriber::SESSION_KEY));
         $form = $this->createForm(ArticleFormType::class, $article);
 
         $form->handleRequest($request);
@@ -49,7 +48,6 @@ class ArticleAdminController extends BaseController
      */
     public function edit(TranslatorInterface $translator, Article $article, Request $request, EntityManagerInterface $em)
     {
-        $article->setCurrentLocale($request->getSession()->get(TranslatableLocaleSubscriber::SESSION_KEY));
         $form = $this->createForm(ArticleFormType::class, $article, [
             'include_published_at' => true
         ]);
diff --git a/templates/article_admin/_form.html.twig b/templates/article_admin/_form.html.twig
index f6523fec..b614084c 100644
--- a/templates/article_admin/_form.html.twig
+++ b/templates/article_admin/_form.html.twig
@@ -1,15 +1,3 @@
-<div>
-    {% for locale in app_locales_array %}
-        <a href="{{ path(app.request.get('_route', 'app_homepage'), app.request.get('_route_params', [])|merge({_translatable_locale: locale})) }}">
-            {% if locale == app.session.get('_translatable_locale') %}
-                <strong>{{ locale|upper }}</strong>
-            {% else %}
-                {{ locale|upper }}
-            {% endif %}
-        </a>
-    {% endfor %}
-</div>
-
 {{ form_start(articleForm) }}
     {{ form_row(articleForm.author) }}
     {{ form_row(articleForm.location, {
