diff --git a/templates/article_admin/_form.html.twig b/templates/article_admin/_form.html.twig
index b599901..eb385c7 100644
--- a/templates/article_admin/_form.html.twig
+++ b/templates/article_admin/_form.html.twig
@@ -13,7 +13,9 @@
         </div>
         <div class="col-sm-3">
             {% if articleForm.vars.data.imageFilename %}
-                <img src="{{ articleForm.vars.data.imagePath|imagine_filter('squared_thumbnail_small') }}" height="100">
+                <a href="{{ uploaded_asset(articleForm.vars.data.imagePath) }}" target="_blank">
+                    <img src="{{ articleForm.vars.data.imagePath|imagine_filter('squared_thumbnail_small') }}" height="100">
+                </a>
             {% endif %}
         </div>
     </div>
