diff --git a/src/Entity/Article.php b/src/Entity/Article.php
index 0eabc133..c9f6d3c3 100644
--- a/src/Entity/Article.php
+++ b/src/Entity/Article.php
@@ -59,7 +59,7 @@ class Article
     private $imageFilename;
 
     /**
-     * @ORM\OneToMany(targetEntity="App\Entity\Comment", mappedBy="article")
+     * @ORM\OneToMany(targetEntity="App\Entity\Comment", mappedBy="article", fetch="EXTRA_LAZY")
      * @ORM\OrderBy({"createdAt" = "DESC"})
      */
     private $comments;
