diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index 29eefcf2..aee4f563 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -85,7 +85,6 @@ class ArticleReferenceAdminController extends BaseController
 
         if (is_file($uploadedFile->getPathname())) {
             unlink($uploadedFile->getPathname());
-            dd($uploadedFile->getPathname());
         }
 
         $entityManager->persist($articleReference);
