diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index e4c69539..d9284cf5 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -44,10 +44,11 @@ class ArticleReferenceAdminController extends BaseController
             $tmpPath = sys_get_temp_dir().'/sf_upload'.uniqid();
             file_put_contents($tmpPath, $uploadApiModel->getDecodedData());
             $uploadedFile = new FileObject($tmpPath);
-            dd($uploadedFile);
+            $originalFilename = $uploadApiModel->filename;
         } else {
             /** @var UploadedFile $uploadedFile */
             $uploadedFile = $request->files->get('reference');
+            $originalFilename = $uploadedFile->getClientOriginalName();
         }
 
         $violations = $validator->validate(
@@ -80,7 +81,7 @@ class ArticleReferenceAdminController extends BaseController
 
         $articleReference = new ArticleReference($article);
         $articleReference->setFilename($filename);
-        $articleReference->setOriginalFilename($uploadedFile->getClientOriginalName() ?? $filename);
+        $articleReference->setOriginalFilename($originalFilename ?? $filename);
         $articleReference->setMimeType($uploadedFile->getMimeType() ?? 'application/octet-stream');
 
         $entityManager->persist($articleReference);
