diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index f724366e..3b1bcc6c 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -84,7 +84,6 @@ class ArticleReferenceAdminController extends BaseController
 
         if (is_file($uploadedFile->getPathname())) {
             unlink($uploadedFile->getPathname());
-            dd($uploadedFile->getPathname());
         }
 
         $entityManager->persist($articleReference);
