diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index d9284cf..b67f002 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -84,6 +84,11 @@ class ArticleReferenceAdminController extends BaseController
         $articleReference->setOriginalFilename($originalFilename ?? $filename);
         $articleReference->setMimeType($uploadedFile->getMimeType() ?? 'application/octet-stream');
 
+        if (is_file($uploadedFile->getPathname())) {
+            unlink($uploadedFile->getPathname());
+            dd($uploadedFile->getPathname());
+        }
+
         $entityManager->persist($articleReference);
         $entityManager->flush();
 
