diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index ce7e1a03..5f2110fe 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -83,6 +83,11 @@ class ArticleReferenceAdminController extends BaseController
         $articleReference->setOriginalFilename($originalFilename ?? $filename);
         $articleReference->setMimeType($uploadedFile->getMimeType() ?? 'application/octet-stream');
 
+        if (is_file($uploadedFile->getPathname())) {
+            unlink($uploadedFile->getPathname());
+            dd($uploadedFile->getPathname());
+        }
+
         $entityManager->persist($articleReference);
         $entityManager->flush();
 
