diff --git a/src/Api/ArticleReferenceUploadApiModel.php b/src/Api/ArticleReferenceUploadApiModel.php
index acddd1fc..5f27316e 100644
--- a/src/Api/ArticleReferenceUploadApiModel.php
+++ b/src/Api/ArticleReferenceUploadApiModel.php
@@ -14,5 +14,13 @@ class ArticleReferenceUploadApiModel
     /**
      * @Assert\NotBlank()
      */
-    public $data;
+    private $data;
+
+    private $decodedData;
+
+    public function setData(?string $data)
+    {
+        $this->data = $data;
+        $this->decodedData = base64_decode($data);
+    }
 }
