diff --git a/src/Controller/AccountController.php b/src/Controller/AccountController.php
index 15478e2c..18db4af3 100644
--- a/src/Controller/AccountController.php
+++ b/src/Controller/AccountController.php
@@ -29,6 +29,8 @@ class AccountController extends BaseController
     {
         $user = $this->getUser();
 
-        return $this->json($user);
+        return $this->json($user, 200, [], [
+            'groups' => ['main'],
+        ]);
     }
 }
diff --git a/src/Entity/User.php b/src/Entity/User.php
index d3eebbe8..e84f4d38 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -4,6 +4,7 @@ namespace App\Entity;
 
 use Doctrine\ORM\Mapping as ORM;
 use Symfony\Component\Security\Core\User\UserInterface;
+use Symfony\Component\Serializer\Annotation\Groups;
 
 /**
  * @ORM\Entity(repositoryClass="App\Repository\UserRepository")
@@ -19,6 +20,7 @@ class User implements UserInterface
 
     /**
      * @ORM\Column(type="string", length=180, unique=true)
+     * @Groups("main")
      */
     private $email;
 
@@ -29,6 +31,7 @@ class User implements UserInterface
 
     /**
      * @ORM\Column(type="string", length=255)
+     * @Groups("main")
      */
     private $firstName;
 
@@ -39,6 +42,7 @@ class User implements UserInterface
 
     /**
      * @ORM\Column(type="string", length=255, nullable=true)
+     * @Groups("main")
      */
     private $twitterUsername;
 
