diff --git a/src/Controller/AccountController.php b/src/Controller/AccountController.php
index b9026f91..15478e2c 100644
--- a/src/Controller/AccountController.php
+++ b/src/Controller/AccountController.php
@@ -21,4 +21,14 @@ class AccountController extends BaseController
 
         ]);
     }
+
+    /**
+     * @Route("/api/account", name="api_account")
+     */
+    public function accountApi()
+    {
+        $user = $this->getUser();
+
+        return $this->json($user);
+    }
 }
