diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index c0157824..a1d2ae59 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -3,7 +3,11 @@
 namespace App\Controller;
 
 use App\Entity\Article;
+use App\Entity\ArticleReference;
+use App\Service\UploaderHelper;
+use Doctrine\ORM\EntityManagerInterface;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Routing\Annotation\Route;
 
@@ -13,8 +17,23 @@ class ArticleReferenceAdminController extends BaseController
      * @Route("/admin/article/{id}/references", name="admin_article_add_reference", methods={"POST"})
      * @IsGranted("MANAGE", subject="article")
      */
-    public function uploadArticleReference(Article $article, Request $request)
+    public function uploadArticleReference(Article $article, Request $request, UploaderHelper $uploaderHelper, EntityManagerInterface $entityManager)
     {
-        dd($request->files->get('reference'));
+        /** @var UploadedFile $uploadedFile */
+        $uploadedFile = $request->files->get('reference');
+
+        $filename = $uploaderHelper->uploadArticleReference($uploadedFile);
+
+        $articleReference = new ArticleReference($article);
+        $articleReference->setFilename($filename);
+        $articleReference->setOriginalFilename($uploadedFile->getClientOriginalName() ?? $filename);
+        $articleReference->setMimeType($uploadedFile->getMimeType() ?? 'application/octet-stream');
+
+        $entityManager->persist($articleReference);
+        $entityManager->flush();
+
+        return $this->redirectToRoute('admin_article_edit', [
+            'id' => $article->getId(),
+        ]);
     }
 }
diff --git a/src/Entity/ArticleReference.php b/src/Entity/ArticleReference.php
index 9687fb73..7b078940 100644
--- a/src/Entity/ArticleReference.php
+++ b/src/Entity/ArticleReference.php
@@ -37,6 +37,11 @@ class ArticleReference
      */
     private $mimeType;
 
+    public function __construct(Article $article)
+    {
+        $this->article = $article;
+    }
+
     public function getId(): ?int
     {
         return $this->id;
@@ -47,13 +52,6 @@ class ArticleReference
         return $this->article;
     }
 
-    public function setArticle(?Article $article): self
-    {
-        $this->article = $article;
-
-        return $this;
-    }
-
     public function getFilename(): ?string
     {
         return $this->filename;
diff --git a/src/Service/UploaderHelper.php b/src/Service/UploaderHelper.php
index 989ab93a..b671a215 100644
--- a/src/Service/UploaderHelper.php
+++ b/src/Service/UploaderHelper.php
@@ -68,6 +68,11 @@ class UploaderHelper
         return $newFilename;
     }
 
+    public function uploadArticleReference(File $file): string
+    {
+        dd($file);
+    }
+
     public function getPublicPath(string $path): string
     {
         // needed if you deploy under a subdirectory
