diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 5fea0f82..7a36e7b9 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -81,7 +81,8 @@ class ArticleAdminController extends BaseController
         }
 
         return $this->render('article_admin/edit.html.twig', [
-            'articleForm' => $form->createView()
+            'articleForm' => $form->createView(),
+            'article' => $article,
         ]);
     }
 
diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
new file mode 100644
index 00000000..c0157824
--- /dev/null
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Article;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ArticleReferenceAdminController extends BaseController
+{
+    /**
+     * @Route("/admin/article/{id}/references", name="admin_article_add_reference", methods={"POST"})
+     * @IsGranted("MANAGE", subject="article")
+     */
+    public function uploadArticleReference(Article $article, Request $request)
+    {
+        dd($request->files->get('reference'));
+    }
+}
diff --git a/templates/article_admin/edit.html.twig b/templates/article_admin/edit.html.twig
index f353a02e..1bff3fa0 100644
--- a/templates/article_admin/edit.html.twig
+++ b/templates/article_admin/edit.html.twig
@@ -14,6 +14,13 @@
         </div>
         <div class="col-sm-4">
             <h3>References</h3>
+
+            <form action="{{ path('admin_article_add_reference', {
+                id: article.id
+            }) }}" method="POST" enctype="multipart/form-data">
+                <input type="file" name="reference">
+                <button type="submit" class="btn btn-sm btn-primary">Upload</button>
+            </form>
         </div>
     </div>
 {% endblock %}
