diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index a1d2ae59e..882f2e2d8 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -10,6 +10,8 @@ use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Symfony\Component\HttpFoundation\File\UploadedFile;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Validator\Constraints\File;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
 
 class ArticleReferenceAdminController extends BaseController
 {
@@ -17,11 +19,22 @@ class ArticleReferenceAdminController extends BaseController
      * @Route("/admin/article/{id}/references", name="admin_article_add_reference", methods={"POST"})
      * @IsGranted("MANAGE", subject="article")
      */
-    public function uploadArticleReference(Article $article, Request $request, UploaderHelper $uploaderHelper, EntityManagerInterface $entityManager)
+    public function uploadArticleReference(Article $article, Request $request, UploaderHelper $uploaderHelper, EntityManagerInterface $entityManager, ValidatorInterface $validator)
     {
         /** @var UploadedFile $uploadedFile */
         $uploadedFile = $request->files->get('reference');
 
+        $violations = $validator->validate(
+            $uploadedFile,
+            new File([
+                'maxSize' => '1k'
+            ])
+        );
+
+        if ($violations->count() > 0) {
+            dd($violations);
+        }
+
         $filename = $uploaderHelper->uploadArticleReference($uploadedFile);
 
         $articleReference = new ArticleReference($article);
