diff --git a/templates/article_admin/edit.html.twig b/templates/article_admin/edit.html.twig
index f353a02e5..a49c5365b 100644
--- a/templates/article_admin/edit.html.twig
+++ b/templates/article_admin/edit.html.twig
@@ -14,6 +14,11 @@
         </div>
         <div class="col-sm-4">
             <h3>References</h3>
+
+            <form action="" method="POST" enctype="multipart/form-data">
+                <input type="file" name="reference">
+                <button type="submit" class="btn btn-sm btn-primary">Upload</button>
+            </form>
         </div>
     </div>
 {% endblock %}
