diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index ec03cb1bc..44f7b6ae3 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -28,7 +28,17 @@ class ArticleReferenceAdminController extends BaseController
         $violations = $validator->validate(
             $uploadedFile,
             new File([
-                'maxSize' => '1k'
+                'maxSize' => '5M',
+                'mimeTypes' => [
+                    'image/*',
+                    'application/pdf',
+                    'application/msword',
+                    'application/vnd.ms-excel',
+                    'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+                    'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+                    'application/vnd.openxmlformats-officedocument.presentationml.presentation',
+                    'text/plain'
+                ]
             ])
         );
 
