diff --git a/src/Entity/Article.php b/src/Entity/Article.php
index b4345d33..3637fcc9 100644
--- a/src/Entity/Article.php
+++ b/src/Entity/Article.php
@@ -320,27 +320,4 @@ class Article
     {
         return $this->articleReferences;
     }
-
-    public function addArticleReference(ArticleReference $articleReference): self
-    {
-        if (!$this->articleReferences->contains($articleReference)) {
-            $this->articleReferences[] = $articleReference;
-            $articleReference->setArticle($this);
-        }
-
-        return $this;
-    }
-
-    public function removeArticleReference(ArticleReference $articleReference): self
-    {
-        if ($this->articleReferences->contains($articleReference)) {
-            $this->articleReferences->removeElement($articleReference);
-            // set the owning side to null (unless already changed)
-            if ($articleReference->getArticle() === $this) {
-                $articleReference->setArticle(null);
-            }
-        }
-
-        return $this;
-    }
 }
