diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 2a07f9d2..bd23fdfd 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -15,6 +15,8 @@ use Symfony\Component\HttpFoundation\File\UploadedFile;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Validator\Constraints\File;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
 
 class ArticleAdminController extends BaseController
 {
@@ -91,11 +93,22 @@ class ArticleAdminController extends BaseController
      * @Route("/admin/article/{id}/reference", name="admin_article_add_reference", methods={"POST"})
      * @IsGranted("MANAGE", subject="article")
      */
-    public function uploadArticleReference(Article $article, Request $request, UploaderHelper $uploaderHelper, EntityManagerInterface $entityManager)
+    public function uploadArticleReference(Article $article, Request $request, UploaderHelper $uploaderHelper, EntityManagerInterface $entityManager, ValidatorInterface $validator)
     {
         /** @var UploadedFile $uploadedFile */
         $uploadedFile = $request->files->get('reference');
 
+        $violations = $validator->validate(
+            $uploadedFile,
+            new File([
+                'maxSize' => '1k'
+            ])
+        );
+
+        if ($violations->count() > 0) {
+            dd($violations);
+        }
+
         $filename = $uploaderHelper->uploadArticleReference($uploadedFile);
 
         $articleReference = new ArticleReference($article);
