diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index 882f2e2d8..ec03cb1bc 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -11,6 +11,7 @@ use Symfony\Component\HttpFoundation\File\UploadedFile;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\Validator\Constraints\File;
+use Symfony\Component\Validator\ConstraintViolation;
 use Symfony\Component\Validator\Validator\ValidatorInterface;
 
 class ArticleReferenceAdminController extends BaseController
@@ -32,7 +33,13 @@ class ArticleReferenceAdminController extends BaseController
         );
 
         if ($violations->count() > 0) {
-            dd($violations);
+            /** @var ConstraintViolation $violation */
+            $violation = $violations[0];
+            $this->addFlash('error', $violation->getMessage());
+
+            return $this->redirectToRoute('admin_article_edit', [
+                'id' => $article->getId(),
+            ]);
         }
 
         $filename = $uploaderHelper->uploadArticleReference($uploadedFile);
diff --git a/templates/base.html.twig b/templates/base.html.twig
index f360a280e..827883e74 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -71,6 +71,12 @@
             </div>
         {% endfor %}
 
+        {% for message in app.flashes('error') %}
+            <div class="alert alert-danger">
+                {{ message }}
+            </div>
+        {% endfor %}
+
         {% block body %}{% endblock %}
 
         <footer class="footer">
