diff --git a/config/packages/security.yaml b/config/packages/security.yaml
index e4800f24..6da73fc5 100644
--- a/config/packages/security.yaml
+++ b/config/packages/security.yaml
@@ -13,6 +13,10 @@ security:
         main:
             anonymous: true
 
+            guard:
+                authenticators:
+                    - App\Security\LoginFormAuthenticator
+
             # activate different ways to authenticate
 
             # http_basic: true
diff --git a/src/Security/LoginFormAuthenticator.php b/src/Security/LoginFormAuthenticator.php
index 5831ef28..833fad93 100644
--- a/src/Security/LoginFormAuthenticator.php
+++ b/src/Security/LoginFormAuthenticator.php
@@ -12,7 +12,7 @@ class LoginFormAuthenticator extends AbstractFormLoginAuthenticator
 {
     public function supports(Request $request)
     {
-        // todo
+        die('Our authenticator is alive!');
     }
 
     public function getCredentials(Request $request)
