diff --git a/src/Security/LoginFormAuthenticator.php b/src/Security/LoginFormAuthenticator.php
index 833fad93..f186f9e3 100644
--- a/src/Security/LoginFormAuthenticator.php
+++ b/src/Security/LoginFormAuthenticator.php
@@ -12,12 +12,14 @@ class LoginFormAuthenticator extends AbstractFormLoginAuthenticator
 {
     public function supports(Request $request)
     {
-        die('Our authenticator is alive!');
+        // do your work when we're POSTing to the login page
+        return $request->attributes->get('_route') === 'app_login'
+            && $request->isMethod('POST');
     }
 
     public function getCredentials(Request $request)
     {
-        // todo
+        dump($request->request->all());die;
     }
 
     public function getUser($credentials, UserProviderInterface $userProvider)
