diff --git a/templates/base.html.twig b/templates/base.html.twig
index 7c9ec4a8..9d548967 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -81,10 +81,10 @@
                         </li>
                     {% else %}
                         <li class="nav-item">
-                            <a style="color: #fff;" class="nav-link" href="{{ path('app_login') }}">Login</a>
+                            <a style="color: #fff;" class="nav-link" href="{{ path('app_login') }}">{{ 'Login'|trans }}</a>
                         </li>
                         <li class="nav-item">
-                            <a style="color: #fff;" class="nav-link" href="{{ path('app_register') }}">Register</a>
+                            <a style="color: #fff;" class="nav-link" href="{{ path('app_register') }}">{{ 'Register'|trans }}</a>
                         </li>
                     {% endif %}
                 </ul>
