diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index a63520a6..2241da8f 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -18,7 +18,7 @@ class ArticleController extends AbstractController
      */
     private $isDebug;
 
-    public function __construct(bool $isDebug)
+    public function __construct(bool $isDebug, Client $slack)
     {
         $this->isDebug = $isDebug;
     }
