diff --git a/assets/js/components/algolia-autocomplete.js b/assets/js/components/algolia-autocomplete.js
index fa62249c..86ed5646 100644
--- a/assets/js/components/algolia-autocomplete.js
+++ b/assets/js/components/algolia-autocomplete.js
@@ -1,3 +1,6 @@
+import $ from 'jquery';
+import autocomplete from 'autocomplete.js';
+
 $(document).ready(function() {
     $('.js-user-autocomplete').each(function() {
         var autocompleteUrl = $(this).data('autocomplete-url');
