diff --git a/src/Entity/Article.php b/src/Entity/Article.php
index 5e68091..9fd06d8 100644
--- a/src/Entity/Article.php
+++ b/src/Entity/Article.php
@@ -29,7 +29,7 @@ class Article
     private $id;
 
     /**
-     * @ORM\Column(type="string", length=255)
+     * @ORM\Column(type="string", length=255, nullable=true)
      */
     private $title;
     // @Assert\NotBlank(message="Get creative and think of a title!")
