diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index ad70d8ba..ac344501 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -67,6 +67,14 @@ class ArticleAdminController extends BaseController
         ]);
     }
 
+    /**
+     * @Route("/admin/upload/test", name="upload_test")
+     */
+    public function temporaryUploadAction(Request $request)
+    {
+        dd($request->files->get('image'));
+    }
+
     /**
      * @Route("/admin/article/location-select", name="admin_article_location_select")
      * @IsGranted("ROLE_USER")
diff --git a/templates/article_admin/edit.html.twig b/templates/article_admin/edit.html.twig
index 342703c8..4baecba2 100644
--- a/templates/article_admin/edit.html.twig
+++ b/templates/article_admin/edit.html.twig
@@ -3,6 +3,14 @@
 {% block content_body %}
     <h1>Edit the Article! 🚀</h1>
 
+    <form method="POST" action="{{ path('upload_test') }}">
+        <input type="file" name="image" />
+
+        <button type="submit" class="btn btn-primary">Upload!</button>
+    </form>
+
+    <hr/>
+
     {{ include('article_admin/_form.html.twig', {
         button_text: 'Update!'
     }) }}
