diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index cd54acbb..c665f73e 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -30,10 +30,30 @@ class ArticleController extends AbstractController
             'I like bacon too! Buy some from my site! bakinsomebacon.com',
         ];
 
+        $articleContent = <<<EOF
+Spicy **jalapeno bacon** ipsum dolor amet veniam shank in dolore. Ham hock nisi landjaeger cow,
+lorem proident [beef ribs](https://baconipsum.com/) aute enim veniam ut cillum pork chuck picanha. Dolore reprehenderit
+labore minim pork belly spare ribs cupim short loin in. Elit exercitation eiusmod dolore cow
+turkey shank eu pork belly meatball non cupim.
+
+Laboris beef ribs fatback fugiat eiusmod jowl kielbasa alcatra dolore velit ea ball tip. Pariatur
+laboris sunt venison, et laborum dolore minim non meatball. Shankle eu flank aliqua shoulder,
+capicola biltong frankfurter boudin cupim officia. Exercitation fugiat consectetur ham. Adipisicing
+picanha shank et filet mignon pork belly ut ullamco. Irure velit turducken ground round doner incididunt
+occaecat lorem meatball prosciutto quis strip steak.
+
+Meatball adipisicing ribeye bacon strip steak eu. Consectetur ham hock pork hamburger enim strip steak
+mollit quis officia meatloaf tri-tip swine. Cow ut reprehenderit, buffalo incididunt in filet mignon
+strip steak pork belly aliquip capicola officia. Labore deserunt esse chicken lorem shoulder tail consectetur
+cow est ribeye adipisicing. Pig hamburger pork belly enim. Do porchetta minim capicola irure pancetta chuck
+fugiat.
+EOF;
+
         return $this->render('article/show.html.twig', [
             'title' => ucwords(str_replace('-', ' ', $slug)),
             'slug' => $slug,
             'comments' => $comments,
+            'articleContent' => $articleContent,
         ]);
     }
 
diff --git a/templates/article/show.html.twig b/templates/article/show.html.twig
index 346fe33c..a52823f3 100644
--- a/templates/article/show.html.twig
+++ b/templates/article/show.html.twig
@@ -26,33 +26,7 @@
                 <div class="row">
                     <div class="col-sm-12">
                         <div class="article-text">
-                            <p>Spicy jalapeno bacon ipsum dolor amet veniam shank in dolore. Ham hock nisi landjaeger cow,
-                                lorem proident beef ribs aute enim veniam ut cillum pork chuck picanha. Dolore reprehenderit
-                                labore minim pork belly spare ribs cupim short loin in. Elit exercitation eiusmod dolore cow
-                                turkey shank eu pork belly meatball non cupim.</p>
-
-                            <p>Laboris beef ribs fatback fugiat eiusmod jowl kielbasa alcatra dolore velit ea ball tip. Pariatur
-                                laboris sunt venison, et laborum dolore minim non meatball. Shankle eu flank aliqua shoulder,
-                                capicola biltong frankfurter boudin cupim officia. Exercitation fugiat consectetur ham. Adipisicing
-                                picanha shank et filet mignon pork belly ut ullamco. Irure velit turducken ground round doner incididunt
-                                occaecat lorem meatball prosciutto quis strip steak.</p>
-
-                            <p>Meatball adipisicing ribeye bacon strip steak eu. Consectetur ham hock pork hamburger enim strip steak
-                                mollit quis officia meatloaf tri-tip swine. Cow ut reprehenderit, buffalo incididunt in filet mignon
-                                strip steak pork belly aliquip capicola officia. Labore deserunt esse chicken lorem shoulder tail consectetur
-                                cow est ribeye adipisicing. Pig hamburger pork belly enim. Do porchetta minim capicola irure pancetta chuck
-                                fugiat.</p>
-
-                            <p>Sausage tenderloin officia jerky nostrud. Laborum elit pastrami non, pig kevin buffalo minim ex quis. Pork belly
-                                pork chop officia anim. Irure tempor leberkas kevin adipisicing cupidatat qui buffalo ham aliqua pork belly
-                                exercitation eiusmod. Exercitation incididunt rump laborum, t-bone short ribs buffalo ut shankle pork chop
-                                bresaola shoulder burgdoggen fugiat. Adipisicing nostrud chicken consequat beef ribs, quis filet mignon do.
-                                Prosciutto capicola mollit shankle aliquip do dolore hamburger brisket turducken eu.</p>
-
-                            <p>Do mollit deserunt prosciutto laborum. Duis sint tongue quis nisi. Capicola qui beef ribs dolore pariatur.
-                                Minim strip steak fugiat nisi est, meatloaf pig aute. Swine rump turducken nulla sausage. Reprehenderit pork
-                                belly tongue alcatra, shoulder excepteur in beef bresaola duis ham bacon eiusmod. Doner drumstick short loin,
-                                adipisicing cow cillum tenderloin.</p>
+                            {{ articleContent }}
                         </div>
                     </div>
                 </div>
