diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index c665f73e..a24f89b6 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -2,6 +2,7 @@
 
 namespace App\Controller;
 
+use Michelf\MarkdownInterface;
 use Psr\Log\LoggerInterface;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
@@ -22,7 +23,7 @@ class ArticleController extends AbstractController
     /**
      * @Route("/news/{slug}", name="article_show")
      */
-    public function show($slug)
+    public function show($slug, MarkdownInterface $markdown)
     {
         $comments = [
             'I ate a normal rock once. It did NOT taste like bacon!',
@@ -49,6 +50,8 @@ cow est ribeye adipisicing. Pig hamburger pork belly enim. Do porchetta minim ca
 fugiat.
 EOF;
 
+        $articleContent = $markdown->transform($articleContent);
+
         return $this->render('article/show.html.twig', [
             'title' => ucwords(str_replace('-', ' ', $slug)),
             'slug' => $slug,
