diff --git a/src/Entity/Article.php b/src/Entity/Article.php
index 64a188e4..c670d0e7 100644
--- a/src/Entity/Article.php
+++ b/src/Entity/Article.php
@@ -4,6 +4,7 @@ namespace App\Entity;
 
 use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\Common\Collections\Collection;
+use Doctrine\Common\Collections\Criteria;
 use Doctrine\ORM\Mapping as ORM;
 use Gedmo\Mapping\Annotation as Gedmo;
 use Gedmo\Timestampable\Traits\TimestampableEntity;
@@ -184,7 +185,12 @@ class Article
      */
     public function getNonDeletedComments(): Collection
     {
-        return $this->comments;
+        $criteria = Criteria::create()
+            ->andWhere(Criteria::expr()->eq('isDeleted', false))
+            ->orderBy(['createdAt' => 'DESC'])
+        ;
+
+        return $this->comments->matching($criteria);
     }
 
     public function addComment(Comment $comment): self
