diff --git a/src/Entity/Article.php b/src/Entity/Article.php
index c670d0e..5d61c77 100644
--- a/src/Entity/Article.php
+++ b/src/Entity/Article.php
@@ -2,6 +2,7 @@
 
 namespace App\Entity;
 
+use App\Repository\ArticleRepository;
 use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\Common\Collections\Collection;
 use Doctrine\Common\Collections\Criteria;
@@ -185,10 +186,7 @@ class Article
      */
     public function getNonDeletedComments(): Collection
     {
-        $criteria = Criteria::create()
-            ->andWhere(Criteria::expr()->eq('isDeleted', false))
-            ->orderBy(['createdAt' => 'DESC'])
-        ;
+        $criteria = ArticleRepository::createNonDeletedCriteria();
 
         return $this->comments->matching($criteria);
     }
diff --git a/src/Repository/ArticleRepository.php b/src/Repository/ArticleRepository.php
index 2435455..f5eac5c 100644
--- a/src/Repository/ArticleRepository.php
+++ b/src/Repository/ArticleRepository.php
@@ -4,6 +4,7 @@ namespace App\Repository;
 
 use App\Entity\Article;
 use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Common\Collections\Criteria;
 use Doctrine\ORM\QueryBuilder;
 use Symfony\Bridge\Doctrine\RegistryInterface;
 
@@ -32,6 +33,14 @@ class ArticleRepository extends ServiceEntityRepository
         ;
     }
 
+    public static function createNonDeletedCriteria(): Criteria
+    {
+        return Criteria::create()
+            ->andWhere(Criteria::expr()->eq('isDeleted', false))
+            ->orderBy(['createdAt' => 'DESC'])
+        ;
+    }
+
     /*
     public function findOneBySomeField($value): ?Article
     {
