diff --git a/src/Entity/Article.php b/src/Entity/Article.php
index c9f6d3c3..64a188e4 100644
--- a/src/Entity/Article.php
+++ b/src/Entity/Article.php
@@ -179,6 +179,14 @@ class Article
         return $this->comments;
     }
 
+    /**
+     * @return Collection|Comment[]
+     */
+    public function getNonDeletedComments(): Collection
+    {
+        return $this->comments;
+    }
+
     public function addComment(Comment $comment): self
     {
         if (!$this->comments->contains($comment)) {
diff --git a/templates/article/show.html.twig b/templates/article/show.html.twig
index 3720dbde..4e7a3dfc 100644
--- a/templates/article/show.html.twig
+++ b/templates/article/show.html.twig
@@ -39,7 +39,7 @@
                 </div>
                 <div class="row">
                     <div class="col-sm-12">
-                        <h3><i class="pr-3 fa fa-comment"></i>{{ article.comments|length }} Comments</h3>
+                        <h3><i class="pr-3 fa fa-comment"></i>{{ article.nonDeletedComments|length }} Comments</h3>
                         <hr>
 
                         <div class="row mb-5">
@@ -55,7 +55,7 @@
                             </div>
                         </div>
 
-                        {% for comment in article.comments %}
+                        {% for comment in article.nonDeletedComments %}
                         <div class="row">
                             <div class="col-sm-12">
                                 <img class="comment-img rounded-circle" src="{{ asset('images/alien-profile.png') }}">
