diff --git a/src/Controller/CommentAdminController.php b/src/Controller/CommentAdminController.php
index e6971cf1..75c82734 100644
--- a/src/Controller/CommentAdminController.php
+++ b/src/Controller/CommentAdminController.php
@@ -2,6 +2,7 @@
 
 namespace App\Controller;
 
+use App\Repository\CommentRepository;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
@@ -10,10 +11,12 @@ class CommentAdminController extends Controller
     /**
      * @Route("/admin/comment", name="comment_admin")
      */
-    public function index()
+    public function index(CommentRepository $repository)
     {
+        $comments = $repository->findBy([], ['createdAt' => 'DESC']);
+
         return $this->render('comment_admin/index.html.twig', [
-            'controller_name' => 'CommentAdminController',
+            'comments' => $comments,
         ]);
     }
 }
diff --git a/templates/comment_admin/index.html.twig b/templates/comment_admin/index.html.twig
index b3cf8dfc..ab6d40c8 100644
--- a/templates/comment_admin/index.html.twig
+++ b/templates/comment_admin/index.html.twig
@@ -8,6 +8,37 @@
     <div class="row">
         <div class="col-sm-12">
             <h1>Manage Comments</h1>
+
+            <table class="table table-striped">
+                <thead>
+                <tr>
+                    <th>Article</th>
+                    <th>Author</th>
+                    <th>Comment</th>
+                    <th>Created</th>
+                </tr>
+                </thead>
+                <tbody>
+                {% for comment in comments %}
+                    <tr>
+                        <td>
+                            <a href="{{ path('article_show', {'slug': comment.article.slug}) }}">
+                                {{ comment.article.title }}
+                            </a>
+                        </td>
+                        <td>
+                            {{ comment.authorName }}
+                        </td>
+                        <td style="width: 20%;">
+                            {{ comment.content }}
+                        </td>
+                        <td>
+                            {{ comment.createdAt|ago }}
+                        </td>
+                    </tr>
+                {% endfor %}
+                </tbody>
+            </table>
         </div>
     </div>
 {% endblock %}
