diff --git a/src/Command/AuthorWeeklyReportSendCommand.php b/src/Command/AuthorWeeklyReportSendCommand.php
index 5f9b66c0..49edae93 100644
--- a/src/Command/AuthorWeeklyReportSendCommand.php
+++ b/src/Command/AuthorWeeklyReportSendCommand.php
@@ -2,6 +2,7 @@
 
 namespace App\Command;
 
+use App\Repository\ArticleRepository;
 use App\Repository\UserRepository;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
@@ -15,12 +16,14 @@ class AuthorWeeklyReportSendCommand extends Command
     protected static $defaultName = 'app:author-weekly-report:send';
 
     private $userRepository;
+    private $articleRepository;
 
-    public function __construct(UserRepository $userRepository)
+    public function __construct(UserRepository $userRepository, ArticleRepository $articleRepository)
     {
         parent::__construct(null);
 
         $this->userRepository = $userRepository;
+        $this->articleRepository = $articleRepository;
     }
 
     protected function configure()
@@ -39,6 +42,13 @@ class AuthorWeeklyReportSendCommand extends Command
         $io->progressStart(count($authors));
         foreach ($authors as $author) {
             $io->progressAdvance();
+
+            $articles = $this->articleRepository
+                ->findAllPublishedLastWeekByAuthor($author);
+            // Skip authors who do not have published articles for the last week
+            if (count($articles) === 0) {
+                continue;
+            }
         }
         $io->progressFinish();
 
diff --git a/src/Repository/ArticleRepository.php b/src/Repository/ArticleRepository.php
index 77953620..5d3323ae 100644
--- a/src/Repository/ArticleRepository.php
+++ b/src/Repository/ArticleRepository.php
@@ -3,6 +3,7 @@
 namespace App\Repository;
 
 use App\Entity\Article;
+use App\Entity\User;
 use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
 use Doctrine\ORM\QueryBuilder;
 use Symfony\Bridge\Doctrine\RegistryInterface;
@@ -34,6 +35,20 @@ class ArticleRepository extends ServiceEntityRepository
         ;
     }
 
+    /**
+     * @return Article[]
+     */
+    public function findAllPublishedLastWeekByAuthor(User $author): array
+    {
+        return $this->createQueryBuilder('a')
+            ->andWhere('a.author = :author')
+            ->andWhere('a.publishedAt > :week_ago')
+            ->setParameter('author', $author)
+            ->setParameter('week_ago', new \DateTime('-1 week'))
+            ->getQuery()
+            ->getResult();
+    }
+
     /*
     public function findOneBySomeField($value): ?Article
     {
