diff --git a/templates/email/base.html.twig b/templates/email/base.html.twig
new file mode 100644
index 00000000..15704336
--- /dev/null
+++ b/templates/email/base.html.twig
@@ -0,0 +1,34 @@
+<!doctype html>
+{% filter inline_css(source('@styles/email.css')) %}
+<html lang="en">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>{{ email.subject }}</title>
+</head>
+<body>
+<div class="body">
+    <div class="container">
+        <div class="header text-center">
+            <a href="{{ url('app_homepage') }}">
+                <img src="{{ email.image('@images/email/logo.png') }}" class="logo" alt="SpaceBar Logo">
+            </a>
+        </div>
+        <div class="content">
+            {% block content %}
+            {% endblock %}
+        </div>
+        <div class="footer">
+            <p class="block">
+                Cheers,<br><br>
+                Your friendly <em>Space Bar Team</em>
+            </p>
+            <p class="block bottom text-center">
+                Sent with ❤️ from the friendly folks at The Space Bar
+            </p>
+        </div>
+    </div>
+</div>
+</body>
+</html>
+{% endfilter %}
diff --git a/templates/email/welcome.html.twig b/templates/email/welcome.html.twig
index 63796a82..ab011719 100644
--- a/templates/email/welcome.html.twig
+++ b/templates/email/welcome.html.twig
@@ -1,51 +1,23 @@
-<!doctype html>
-{% filter inline_css(source('@styles/email.css')) %}
-<html lang="en">
-<head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-    <meta name="viewport" content="width=device-width">
-    <title>{{ email.subject }}</title>
-</head>
-<body>
-<div class="body">
-    <div class="container">
-        <div class="header text-center">
-            <a href="{{ url('app_homepage') }}">
-                <img src="{{ email.image('@images/email/logo.png') }}" class="logo" alt="SpaceBar Logo">
-            </a>
-        </div>
-        <div class="content">
-            <h1 class="text-center">Nice to meet you {{ email.toName }}!</h1>
-            <p class="block">
-                Welcome to <strong>the Space Bar</strong>, we can't wait to read what you have to write.
-                Get started on your first article and connect with the space bar community.
-            </p>
-            <p class="block text-center">
-                <a href="{{ url('admin_article_new') }}" class="btn">Get writing!</a>
-            </p>
-            <p class="block">
-                Check out our existing articles and share your thoughts in the comments!
-            </p>
-            <p class="block text-center">
-                <a href="{{ url('app_homepage') }}" class="btn">Get reading!</a>
-            </p>
-            <p class="block">
-                We're so excited that you've decided to join us in our corner of the universe,
-                it's a friendly one with other creative and insightful writers just like you!
-                Need help from a friend? We're always just a message away.
-            </p>
-        </div>
-        <div class="footer">
-            <p class="block">
-                Cheers,<br><br>
-                Your friendly <em>Space Bar Team</em>
-            </p>
-            <p class="block bottom text-center">
-                Sent with ❤️ from the friendly folks at The Space Bar
-            </p>
-        </div>
-    </div>
-</div>
-</body>
-</html>
-{% endfilter %}
+{% extends 'email/base.html.twig' %}
+
+{% block content %}
+    <h1 class="text-center">Nice to meet you {{ email.toName }}!</h1>
+    <p class="block">
+        Welcome to <strong>the Space Bar</strong>, we can't wait to read what you have to write.
+        Get started on your first article and connect with the space bar community.
+    </p>
+    <p class="block text-center">
+        <a href="{{ url('admin_article_new') }}" class="btn">Get writing!</a>
+    </p>
+    <p class="block">
+        Check out our existing articles and share your thoughts in the comments!
+    </p>
+    <p class="block text-center">
+        <a href="{{ url('app_homepage') }}" class="btn">Get reading!</a>
+    </p>
+    <p class="block">
+        We're so excited that you've decided to join us in our corner of the universe,
+        it's a friendly one with other creative and insightful writers just like you!
+        Need help from a friend? We're always just a message away.
+    </p>
+{% endblock %}
