diff --git a/templates/email/author-weekly-report.html.twig b/templates/email/author-weekly-report.html.twig
index 67e314da9..ab4a8131d 100644
--- a/templates/email/author-weekly-report.html.twig
+++ b/templates/email/author-weekly-report.html.twig
@@ -1,40 +1,38 @@
-{% apply inky_to_html|inline_css(source('@styles/foundation-emails.css'), source('@styles/email.css')) %}
-    <container>
-        {# Header #}
-        <hr>
-        <spacer size="20"></spacer>
-        <row>
-            <columns>
-                <p>
-                    What a week {{ email.toName }}! Here's a quick review of what you've been up to on the Space Bar this week
-                </p>
-            </columns>
-        </row>
-        <row>
-            <columns>
-                <table>
-                    <tr>
-                        <th>#</th>
-                        <th>Title</th>
-                        <th>Comments</th>
-                    </tr>
-                    <tr>
-                        <td>1</td>
-                        <td>Article Title</td>
-                        <td>99</td>
-                    </tr>
-                </table>
-            </columns>
-        </row>
-        <row>
-            <columns>
-                <center>
-                    <spacer size="20"></spacer>
-                    <button href="{{ url('app_homepage') }}">Check on the Space Bar</button>
-                    <spacer size="20"></spacer>
-                </center>
-            </columns>
-        </row>
-        {# Footer #}
-    </container>
-{% endapply %}
+{% extends 'email/emailBase.html.twig' %}
+
+{% block content %}
+    <hr>
+    <spacer size="20"></spacer>
+    <row>
+        <columns>
+            <p>
+                What a week {{ email.toName }}! Here's a quick review of what you've been up to on the Space Bar this week
+            </p>
+        </columns>
+    </row>
+    <row>
+        <columns>
+            <table>
+                <tr>
+                    <th>#</th>
+                    <th>Title</th>
+                    <th>Comments</th>
+                </tr>
+                <tr>
+                    <td>1</td>
+                    <td>Article Title</td>
+                    <td>99</td>
+                </tr>
+            </table>
+        </columns>
+    </row>
+    <row>
+        <columns>
+            <center>
+                <spacer size="20"></spacer>
+                <button href="{{ url('app_homepage') }}">Check on the Space Bar</button>
+                <spacer size="20"></spacer>
+            </center>
+        </columns>
+    </row>
+{% endblock %}
diff --git a/templates/email/emailBase.html.twig b/templates/email/emailBase.html.twig
new file mode 100644
index 000000000..2edeed117
--- /dev/null
+++ b/templates/email/emailBase.html.twig
@@ -0,0 +1,31 @@
+{% apply inky_to_html|inline_css(source('@styles/foundation-emails.css'), source('@styles/email.css')) %}
+    <container>
+        <row class="header">
+            <columns>
+                <a href="{{ url('app_homepage') }}">
+                    <img src="{{ email.image('@images/email/logo.png') }}" class="logo" alt="SpaceBar Logo">
+                </a>
+            </columns>
+        </row>
+
+        {% block content %}
+        {% endblock %}
+
+        <row class="footer">
+            <columns>
+                <p>Cheers,</p>
+                <p>Your friendly <em>Space Bar Team</em></p>
+            </columns>
+        </row>
+        <row class="bottom">
+            <columns>
+                <center>
+                    <spacer size="20"></spacer>
+                    <div>
+                        Sent with ❤️ from the friendly folks at The Space Bar
+                    </div>
+                </center>
+            </columns>
+        </row>
+    </container>
+{% endapply %}
diff --git a/templates/email/welcome.html.twig b/templates/email/welcome.html.twig
index 7e39a362c..06ca765ba 100644
--- a/templates/email/welcome.html.twig
+++ b/templates/email/welcome.html.twig
@@ -1,77 +1,54 @@
-{% apply inky_to_html|inline_css(source('@styles/foundation-emails.css'), source('@styles/email.css')) %}
-    <container>
-        <row class="header">
-            <columns>
-                <a href="{{ url('app_homepage') }}">
-                    <img src="{{ email.image('@images/email/logo.png') }}" class="logo" alt="SpaceBar Logo">
-                </a>
-            </columns>
-        </row>
-        <row class="welcome">
-            <columns>
-                <spacer size="35"></spacer>
-                <h1>
-                    <center>
-                        Nice to meet you {{ email.toName }}!
-                    </center>
-                </h1>
-                <spacer size="10"></spacer>
-            </columns>
-        </row>
-        <spacer size="30"></spacer>
-        <row>
-            <columns>
-                <p>
-                    Welcome to <strong>the Space Bar</strong>, we can't wait to read what you have to write.
-                    Get started on your first article and connect with the space bar community.
-                </p>
-            </columns>
-        </row>
-        <row>
-            <columns>
+{% extends 'email/emailBase.html.twig' %}
+
+{% block content %}
+    <row class="welcome">
+        <columns>
+            <spacer size="35"></spacer>
+            <h1>
                 <center>
-                    <button href="{{ url('admin_article_new') }}">Get writing!</button>
+                    Nice to meet you {{ email.toName }}!
                 </center>
-            </columns>
-        </row>
-        <row>
-            <columns>
-                <p>
-                    Check out our existing articles and share your thoughts in the comments!
-                </p>
-            </columns>
-        </row>
-        <row>
-            <columns>
-                <center>
-                    <button href="{{ url('app_homepage') }}">Get reading!</button>
-                </center>
-            </columns>
-        </row>
-        <row>
-            <columns>
-                <p>
-                    We're so excited that you've decided to join us in our corner of the universe,
-                    it's a friendly one with other creative and insightful writers just like you!
-                    Need help from a friend? We're always just a message away.
-                </p>
-            </columns>
-        </row>
-        <row class="footer">
-            <columns>
-                <p>Cheers,</p>
-                <p>Your friendly <em>Space Bar Team</em></p>
-            </columns>
-        </row>
-        <row class="bottom">
-            <columns>
-                <center>
-                    <spacer size="20"></spacer>
-                    <div>
-                        Sent with ❤️ from the friendly folks at The Space Bar
-                    </div>
-                </center>
-            </columns>
-        </row>
-    </container>
-{% endapply %}
+            </h1>
+            <spacer size="10"></spacer>
+        </columns>
+    </row>
+    <spacer size="30"></spacer>
+    <row>
+        <columns>
+            <p>
+                Welcome to <strong>the Space Bar</strong>, we can't wait to read what you have to write.
+                Get started on your first article and connect with the space bar community.
+            </p>
+        </columns>
+    </row>
+    <row>
+        <columns>
+            <center>
+                <button href="{{ url('admin_article_new') }}">Get writing!</button>
+            </center>
+        </columns>
+    </row>
+    <row>
+        <columns>
+            <p>
+                Check out our existing articles and share your thoughts in the comments!
+            </p>
+        </columns>
+    </row>
+    <row>
+        <columns>
+            <center>
+                <button href="{{ url('app_homepage') }}">Get reading!</button>
+            </center>
+        </columns>
+    </row>
+    <row>
+        <columns>
+            <p>
+                We're so excited that you've decided to join us in our corner of the universe,
+                it's a friendly one with other creative and insightful writers just like you!
+                Need help from a friend? We're always just a message away.
+            </p>
+        </columns>
+    </row>
+{% endblock %}
