diff --git a/src/Command/AuthorWeeklyReportSendCommand.php b/src/Command/AuthorWeeklyReportSendCommand.php
index 49edae93..e270b883 100644
--- a/src/Command/AuthorWeeklyReportSendCommand.php
+++ b/src/Command/AuthorWeeklyReportSendCommand.php
@@ -4,12 +4,15 @@ namespace App\Command;
 
 use App\Repository\ArticleRepository;
 use App\Repository\UserRepository;
+use Symfony\Bridge\Twig\Mime\TemplatedEmail;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\NamedAddress;
 
 class AuthorWeeklyReportSendCommand extends Command
 {
@@ -17,13 +20,15 @@ class AuthorWeeklyReportSendCommand extends Command
 
     private $userRepository;
     private $articleRepository;
+    private $mailer;
 
-    public function __construct(UserRepository $userRepository, ArticleRepository $articleRepository)
+    public function __construct(UserRepository $userRepository, ArticleRepository $articleRepository, MailerInterface $mailer)
     {
         parent::__construct(null);
 
         $this->userRepository = $userRepository;
         $this->articleRepository = $articleRepository;
+        $this->mailer = $mailer;
     }
 
     protected function configure()
@@ -49,6 +54,17 @@ class AuthorWeeklyReportSendCommand extends Command
             if (count($articles) === 0) {
                 continue;
             }
+
+            $email = (new TemplatedEmail())
+                ->from(new NamedAddress('alienmailcarrier@example.com', 'The Space Bar'))
+                ->to(new NamedAddress($author->getEmail(), $author->getFirstName()))
+                ->subject('Your weekly report on the Space Bar!')
+                ->htmlTemplate('email/author-weekly-report.html.twig')
+                ->context([
+                    'author' => $author,
+                    'articles' => $articles,
+                ]);
+            $this->mailer->send($email);
         }
         $io->progressFinish();
 
