diff --git a/templates/article/show.html.twig b/templates/article/show.html.twig
index 4e7a3dfc..48a559bb 100644
--- a/templates/article/show.html.twig
+++ b/templates/article/show.html.twig
@@ -1,13 +1,8 @@
-{% extends 'base.html.twig' %}
+{% extends 'content_base.html.twig' %}
 
 {% block title %}Read: {{ article.title }}{% endblock %}
 
-{% block body %}
-
-<div class="container">
-    <div class="row">
-        <div class="col-sm-12">
-            <div class="show-article-container p-3 mt-4">
+{% block content_body %}
                 <div class="row">
                     <div class="col-sm-12">
                         <img class="show-article-img" src="{{ asset(article.imagePath) }}">
@@ -75,10 +70,6 @@
 
                     </div>
                 </div>
-            </div>
-        </div>
-    </div>
-</div>
 
 {% endblock %}
 
diff --git a/templates/comment_admin/index.html.twig b/templates/comment_admin/index.html.twig
index 13625f4e..df334637 100644
--- a/templates/comment_admin/index.html.twig
+++ b/templates/comment_admin/index.html.twig
@@ -1,20 +1,11 @@
-{% extends 'base.html.twig' %}
+{% extends 'content_base.html.twig' %}
 
 {% block title %}Manage Comments{% endblock %}
 
-{% block body %}
-
-<div class="container">
+{% block content_body %}
     <div class="row">
         <div class="col-sm-12">
-            <div class="show-article-container p-3 mt-4">
-                <div class="row">
-                    <div class="col-sm-12">
-                        <h1>Manage Comments</h1>
-                    </div>
-                </div>
-            </div>
+            <h1>Manage Comments</h1>
         </div>
     </div>
-</div>
 {% endblock %}
diff --git a/templates/content_base.html.twig b/templates/content_base.html.twig
new file mode 100644
index 00000000..9d787ff0
--- /dev/null
+++ b/templates/content_base.html.twig
@@ -0,0 +1,13 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-12">
+                <div class="show-article-container p-3 mt-4">
+                    {% block content_body %}{% endblock %}
+                </div>
+            </div>
+        </div>
+    </div>
+{% endblock %}
\ No newline at end of file
