diff --git a/src/Command/AuthorWeeklyReportSendCommand.php b/src/Command/AuthorWeeklyReportSendCommand.php
index 0cbf5aea..22aca9d2 100644
--- a/src/Command/AuthorWeeklyReportSendCommand.php
+++ b/src/Command/AuthorWeeklyReportSendCommand.php
@@ -2,6 +2,7 @@
 
 namespace App\Command;
 
+use App\Entity\Article;
 use App\Entity\User;
 use Doctrine\ORM\EntityManagerInterface;
 use Symfony\Component\Console\Command\Command;
@@ -41,6 +42,14 @@ class AuthorWeeklyReportSendCommand extends Command
         $io->progressStart(count($authors));
         foreach ($authors as $author) {
             $io->progressAdvance();
+
+            $articles = $this->entityManager
+                ->getRepository(Article::class)
+                ->findAllPublishedLastWeekByAuthor($author);
+            // Skip authors who does not have published articles for the last week
+            if (!count($articles)) {
+                continue;
+            }
         }
         $io->progressFinish();
 
diff --git a/src/Repository/ArticleRepository.php b/src/Repository/ArticleRepository.php
index 77953620..5d3323ae 100644
--- a/src/Repository/ArticleRepository.php
+++ b/src/Repository/ArticleRepository.php
@@ -3,6 +3,7 @@
 namespace App\Repository;
 
 use App\Entity\Article;
+use App\Entity\User;
 use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
 use Doctrine\ORM\QueryBuilder;
 use Symfony\Bridge\Doctrine\RegistryInterface;
@@ -34,6 +35,20 @@ class ArticleRepository extends ServiceEntityRepository
         ;
     }
 
+    /**
+     * @return Article[]
+     */
+    public function findAllPublishedLastWeekByAuthor(User $author): array
+    {
+        return $this->createQueryBuilder('a')
+            ->andWhere('a.author = :author')
+            ->andWhere('a.publishedAt > :week_ago')
+            ->setParameter('author', $author)
+            ->setParameter('week_ago', new \DateTime('-1 week'))
+            ->getQuery()
+            ->getResult();
+    }
+
     /*
     public function findOneBySomeField($value): ?Article
     {
