diff --git a/src/Controller/CommentAdminController.php b/src/Controller/CommentAdminController.php
index c993e996..e6971cf1 100644
--- a/src/Controller/CommentAdminController.php
+++ b/src/Controller/CommentAdminController.php
@@ -8,7 +8,7 @@ use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 class CommentAdminController extends Controller
 {
     /**
-     * @Route("/comment/admin", name="comment_admin")
+     * @Route("/admin/comment", name="comment_admin")
      */
     public function index()
     {
