diff --git a/templates/comment_admin/index.html.twig b/templates/comment_admin/index.html.twig
index df334637..b3cf8dfc 100644
--- a/templates/comment_admin/index.html.twig
+++ b/templates/comment_admin/index.html.twig
@@ -2,6 +2,8 @@
 
 {% block title %}Manage Comments{% endblock %}
 
+{% block content_class %}{{ parent() }} show-article-container-border-green{% endblock %}
+
 {% block content_body %}
     <div class="row">
         <div class="col-sm-12">
diff --git a/templates/content_base.html.twig b/templates/content_base.html.twig
index 9d787ff0..01302b62 100644
--- a/templates/content_base.html.twig
+++ b/templates/content_base.html.twig
@@ -4,7 +4,7 @@
     <div class="container">
         <div class="row">
             <div class="col-sm-12">
-                <div class="show-article-container p-3 mt-4">
+                <div class="{% block content_class %}show-article-container p-3 mt-4{% endblock %}">
                     {% block content_body %}{% endblock %}
                 </div>
             </div>
