diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index bc96cbd0..5c5a591a 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -51,6 +51,7 @@ cow est ribeye adipisicing. Pig hamburger pork belly enim. Do porchetta minim ca
 fugiat.
 EOF;
 
+        dump($cache);die;
         $item = $cache->getItem('markdown_'.md5($articleContent));
         if (!$item->isHit()) {
             $item->set($markdown->transform($articleContent));
@@ -58,8 +59,6 @@ EOF;
         }
         $articleContent = $item->get();
 
-        dump($markdown);die;
-
         return $this->render('article/show.html.twig', [
             'title' => ucwords(str_replace('-', ' ', $slug)),
             'slug' => $slug,
