diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index ef7e0830..bc96cbd0 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -58,6 +58,8 @@ EOF;
         }
         $articleContent = $item->get();
 
+        dump($markdown);die;
+
         return $this->render('article/show.html.twig', [
             'title' => ucwords(str_replace('-', ' ', $slug)),
             'slug' => $slug,
