diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index 503e18f6..84ab279d 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -12,6 +12,11 @@ use Twig\Environment;
 
 class ArticleController extends AbstractController
 {
+    public function __construct(bool $isDebug)
+    {
+        dump($isDebug);die;
+    }
+
     /**
      * @Route("/", name="app_homepage")
      */
@@ -23,10 +28,8 @@ class ArticleController extends AbstractController
     /**
      * @Route("/news/{slug}", name="article_show")
      */
-    public function show($slug, MarkdownHelper $markdownHelper, $isDebug)
+    public function show($slug, MarkdownHelper $markdownHelper)
     {
-        dump($isDebug);die;
-
         $comments = [
             'I ate a normal rock once. It did NOT taste like bacon!',
             'Woohoo! I\'m going on an all-asteroid diet!',
