diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index 84ab279d..8f199d01 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -12,9 +12,14 @@ use Twig\Environment;
 
 class ArticleController extends AbstractController
 {
+    /**
+     * Currently unused: just showing a controller with a constructor!
+     */
+    private $isDebug;
+
     public function __construct(bool $isDebug)
     {
-        dump($isDebug);die;
+        $this->isDebug = $isDebug;
     }
 
     /**
