diff --git a/public/js/algolia-autocomplete.js b/public/js/algolia-autocomplete.js
index f7074647..fa62249c 100644
--- a/public/js/algolia-autocomplete.js
+++ b/public/js/algolia-autocomplete.js
@@ -6,7 +6,7 @@ $(document).ready(function() {
             {
                 source: function(query, cb) {
                     $.ajax({
-                        url: autocompleteUrl
+                        url: autocompleteUrl+'?query='+query
                     }).then(function(data) {
                         cb(data.users);
                     });
diff --git a/src/Controller/AdminUtilityController.php b/src/Controller/AdminUtilityController.php
index de6bd08f..4f86cca2 100644
--- a/src/Controller/AdminUtilityController.php
+++ b/src/Controller/AdminUtilityController.php
@@ -6,6 +6,7 @@ use App\Repository\UserRepository;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Routing\Annotation\Route;
 
 class AdminUtilityController extends AbstractController
@@ -14,9 +15,9 @@ class AdminUtilityController extends AbstractController
      * @Route("/admin/utility/users", methods="GET", name="admin_utility_users")
      * @IsGranted("ROLE_ADMIN_ARTICLE")
      */
-    public function getUsersApi(UserRepository $userRepository)
+    public function getUsersApi(UserRepository $userRepository, Request $request)
     {
-        $users = $userRepository->findAllEmailAlphabetical();
+        $users = $userRepository->findAllMatching($request->query->get('query'));
 
         return $this->json([
             'users' => $users
diff --git a/src/Repository/UserRepository.php b/src/Repository/UserRepository.php
index fe3ab5ee..70b7bed8 100644
--- a/src/Repository/UserRepository.php
+++ b/src/Repository/UserRepository.php
@@ -31,6 +31,19 @@ class UserRepository extends ServiceEntityRepository
         ;
     }
 
+    /**
+     * @return User[]
+     */
+    public function findAllMatching(string $query, int $limit = 5)
+    {
+        return $this->createQueryBuilder('u')
+            ->andWhere('u.email LIKE :query')
+            ->setParameter('query', '%'.$query.'%')
+            ->setMaxResults($limit)
+            ->getQuery()
+            ->getResult();
+    }
+
 //    /**
 //     * @return User[] Returns an array of User objects
 //     */
