diff --git a/templates/article_admin/edit.html.twig b/templates/article_admin/edit.html.twig
index 194a7ed8..962b02d7 100644
--- a/templates/article_admin/edit.html.twig
+++ b/templates/article_admin/edit.html.twig
@@ -1,18 +1,5 @@
 {% extends 'content_base.html.twig' %}
 
-{% block javascripts %}
-    {{ parent() }}
-
-    <script src="https://cdn.jsdelivr.net/autocomplete.js/0/autocomplete.jquery.min.js"></script>
-    <script src="{{ asset('js/algolia-autocomplete.js') }}"></script>
-{% endblock %}
-
-{% block stylesheets %}
-    {{ parent() }}
-
-    <link rel="stylesheet" href="{{ asset('css/algolia-autocomplete.css') }}">
-{% endblock %}
-
 {% block content_body %}
     <h1>Edit the Article! 🚀</h1>
 
diff --git a/templates/article_admin/new.html.twig b/templates/article_admin/new.html.twig
index 0765b918..b5a95b99 100644
--- a/templates/article_admin/new.html.twig
+++ b/templates/article_admin/new.html.twig
@@ -1,5 +1,18 @@
 {% extends 'content_base.html.twig' %}
 
+{% block javascripts %}
+    {{ parent() }}
+
+    <script src="https://cdn.jsdelivr.net/autocomplete.js/0/autocomplete.jquery.min.js"></script>
+    <script src="{{ asset('js/algolia-autocomplete.js') }}"></script>
+{% endblock %}
+
+{% block stylesheets %}
+    {{ parent() }}
+
+    <link rel="stylesheet" href="{{ asset('css/algolia-autocomplete.css') }}">
+{% endblock %}
+
 {% block content_body %}
     <h1>Launch a new Article! 🚀</h1>
 
