diff --git a/src/Entity/Article.php b/src/Entity/Article.php
index 44a569d2..3eb0141c 100644
--- a/src/Entity/Article.php
+++ b/src/Entity/Article.php
@@ -72,6 +72,7 @@ class Article
     /**
      * @ORM\ManyToOne(targetEntity="App\Entity\User", inversedBy="articles")
      * @ORM\JoinColumn(nullable=false)
+     * @Assert\NotNull(message="Please set an author")
      */
     private $author;
 
diff --git a/src/Form/DataTransformer/EmailToUserTransformer.php b/src/Form/DataTransformer/EmailToUserTransformer.php
index d4dca50b..ab5c2494 100644
--- a/src/Form/DataTransformer/EmailToUserTransformer.php
+++ b/src/Form/DataTransformer/EmailToUserTransformer.php
@@ -31,6 +31,10 @@ class EmailToUserTransformer implements DataTransformerInterface
 
     public function reverseTransform($value)
     {
+        if (!$value) {
+            return;
+        }
+
         $user = $this->userRepository->findOneBy(['email' => $value]);
 
         if (!$user) {
