diff --git a/src/Form/DataTransformer/EmailToUserTransformer.php b/src/Form/DataTransformer/EmailToUserTransformer.php
index 05542974..d4318a4a 100644
--- a/src/Form/DataTransformer/EmailToUserTransformer.php
+++ b/src/Form/DataTransformer/EmailToUserTransformer.php
@@ -2,6 +2,7 @@
 
 namespace App\Form\DataTransformer;
 
+use App\Entity\User;
 use Symfony\Component\Form\DataTransformerInterface;
 use Symfony\Component\Form\Exception\TransformationFailedException;
 
@@ -9,7 +10,15 @@ class EmailToUserTransformer implements DataTransformerInterface
 {
     public function transform($value)
     {
-        dd('transform', $value);
+        if (null === $value) {
+            return '';
+        }
+
+        if (!$value instanceof User) {
+            throw new \LogicException('The UserSelectTextType can only be used with User objects');
+        }
+
+        return $value->getEmail();
     }
 
     public function reverseTransform($value)
