diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index 671720a8..d9b56fbb 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -31,15 +31,7 @@ class ArticleFormType extends AbstractType
             ->add('publishedAt', null, [
                 'widget' => 'single_text'
             ])
-            ->add('author', EntityType::class, [
-                'class' => User::class,
-                'choice_label' => function(User $user) {
-                    return sprintf('(%d) %s', $user->getId(), $user->getEmail());
-                },
-                'placeholder' => 'Choose an author',
-                'choices' => $this->userRepository->findAllEmailAlphabetical(),
-                'invalid_message' => 'Symfony is too smart for your hacking!'
-            ])
+            ->add('author', UserSelectTextType::class)
         ;
     }
 
diff --git a/src/Form/UserSelectTextType.php b/src/Form/UserSelectTextType.php
new file mode 100644
index 00000000..63b057be
--- /dev/null
+++ b/src/Form/UserSelectTextType.php
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+class UserSelectTextType extends AbstractType
+{
+    public function getParent()
+    {
+        return TextType::class;
+    }
+}
