diff --git a/config/packages/security.yaml b/config/packages/security.yaml
index a48752a7..a16fc638 100644
--- a/config/packages/security.yaml
+++ b/config/packages/security.yaml
@@ -12,7 +12,8 @@ security:
                 property: email
 
     role_hierarchy:
-        ROLE_ADMIN: [ROLE_ADMIN_COMMENT, ROLE_ADMIN_ARTICLE, ROLE_ALLOWED_TO_SWITCH]
+        ROLE_ADMIN: [ROLE_ADMIN_COMMENT, ROLE_ADMIN_ARTICLE, ROLE_ALLOWED_TO_SWITCH, ROLE_ADMIN_UTILITY]
+        ROLE_ADMIN_ARTICLE: [ROLE_ADMIN_UTILITY]
 
     firewalls:
         dev:
diff --git a/src/Controller/AdminUtilityController.php b/src/Controller/AdminUtilityController.php
index 76458e3b..fbb31583 100644
--- a/src/Controller/AdminUtilityController.php
+++ b/src/Controller/AdminUtilityController.php
@@ -2,9 +2,26 @@
 
 namespace App\Controller;
 
+use App\Repository\UserRepository;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
 
+/**
+ * @IsGranted("ROLE_ADMIN_UTILITY")
+ */
 class AdminUtilityController extends AbstractController
 {
+    /**
+     * @Route("/admin/utility/users", methods="GET")
+     */
+    public function getUsersApi(UserRepository $userRepository)
+    {
+        $users = $userRepository->findAllEmailAlphabetical();
 
+        return $this->json([
+            'users' => $users
+        ], 200, [], ['groups' => ['main']]);
+    }
 }
\ No newline at end of file
