diff --git a/src/Controller/AdminUtilityController.php b/src/Controller/AdminUtilityController.php
index 76458e3b..c4e46c7a 100644
--- a/src/Controller/AdminUtilityController.php
+++ b/src/Controller/AdminUtilityController.php
@@ -2,9 +2,23 @@
 
 namespace App\Controller;
 
+use App\Repository\UserRepository;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
 
 class AdminUtilityController extends AbstractController
 {
+    /**
+     * @Route("/admin/utility/users", methods="GET")
+     */
+    public function getUsersApi(UserRepository $userRepository)
+    {
+        $users = $userRepository->findAllEmailAlphabetical();
 
+        return $this->json([
+            'users' => $users
+        ], 200, [], ['groups' => ['main']]);
+    }
 }
\ No newline at end of file
