diff --git a/src/Service/MarkdownHelper.php b/src/Service/MarkdownHelper.php
index 62964857..eb2eeca2 100644
--- a/src/Service/MarkdownHelper.php
+++ b/src/Service/MarkdownHelper.php
@@ -3,21 +3,28 @@
 namespace App\Service;
 
 use Michelf\MarkdownInterface;
+use Psr\Log\LoggerInterface;
 use Symfony\Component\Cache\Adapter\AdapterInterface;
 
 class MarkdownHelper
 {
     private $cache;
     private $markdown;
+    private $logger;
 
-    public function __construct(AdapterInterface $cache, MarkdownInterface $markdown)
+    public function __construct(AdapterInterface $cache, MarkdownInterface $markdown, LoggerInterface $logger)
     {
         $this->cache = $cache;
         $this->markdown = $markdown;
+        $this->logger = $logger;
     }
 
     public function parse(string $source): string
     {
+        if (stripos($source, 'bacon') !== false) {
+            $this->logger->info('They are talking about bacon again!');
+        }
+
         $item = $this->cache->getItem('markdown_'.md5($source));
         if (!$item->isHit()) {
             $item->set($this->markdown->transform($source));
