diff --git a/src/Form/UserSelectTextType.php b/src/Form/UserSelectTextType.php
index b90bd10d..39b9cd1b 100644
--- a/src/Form/UserSelectTextType.php
+++ b/src/Form/UserSelectTextType.php
@@ -7,6 +7,8 @@ use App\Repository\UserRepository;
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 use Symfony\Component\Routing\RouterInterface;
 
@@ -41,10 +43,17 @@ class UserSelectTextType extends AbstractType
             'finder_callback' => function(UserRepository $userRepository, string $email) {
                 return $userRepository->findOneBy(['email' => $email]);
             },
-            'attr' => [
-                'class' => 'js-user-autocomplete',
-                'data-autocomplete-url' => $this->router->generate('admin_utility_users')
-            ]
         ]);
     }
+
+    public function buildView(FormView $view, FormInterface $form, array $options)
+    {
+        $attr = $view->vars['attr'];
+        $class = isset($attr['class']) ? $attr['class'].' ' : '';
+        $class .= 'js-user-autocomplete';
+
+        $attr['class'] = $class;
+        $attr['data-autocomplete-url'] = $this->router->generate('admin_utility_users');
+        $view->vars['attr'] = $attr;
+    }
 }
