diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 3313f0fc..2f16f04d 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -24,11 +24,8 @@ class ArticleAdminController extends AbstractController
 
         $form->handleRequest($request);
         if ($form->isSubmitted() && $form->isValid()) {
-            $data = $form->getData();
-            dd($data);
-            $article = new Article();
-            $article->setTitle($data['title']);
-            $article->setContent($data['content']);
+            /** @var Article $article */
+            $article = $form->getData();
             $article->setAuthor($this->getUser());
 
             $em->persist($article);
