diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index 7cbd4bea..28f6c71f 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -36,6 +36,7 @@ class ArticleFormType extends AbstractType
                 'placeholder' => 'Choose an author',
                 'choices' => $this->userRepository->findAllEmailAlphabetical(),
             ])
+            ->add('publishedAt')
         ;
     }
 
