diff --git a/templates/article/homepage.html.twig b/templates/article/homepage.html.twig
index 4bbba64..5208728 100644
--- a/templates/article/homepage.html.twig
+++ b/templates/article/homepage.html.twig
@@ -20,10 +20,10 @@
 
                 {% for article in articles %}
                 <div class="article-container my-1">
-                    <a href="{{ path('article_show', {slug: article.slug}) }}">
+                    <a href="{{ path('article_show', {slug: article.translate.slug}) }}">
                         <img class="article-img" src="{{ asset(article.imagePath) }}">
                         <div class="article-title d-inline-block pl-3 align-middle">
-                            <span>{{ article.title }}</span>
+                            <span>{{ article.translate.title }}</span>
                             <small>({{ 'comments.count'|trans({'%count%': article.nonDeletedComments|length}) }})</small>
                             <br>
                             {% for tag in article.tags %}
