diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 47c3c12..9210433 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -21,7 +21,7 @@ class ArticleAdminController extends BaseController
      */
     public function new(TranslatorInterface $translator, EntityManagerInterface $em, Request $request)
     {
-        $form = $this->createForm(ArticleFormType::class);
+        $form = $this->createForm(ArticleFormType::class, new Article());
 
         $form->handleRequest($request);
         if ($form->isSubmitted() && $form->isValid()) {
