diff --git a/src/Entity/Article.php b/src/Entity/Article.php
index 595c10f..b2f7f01 100644
--- a/src/Entity/Article.php
+++ b/src/Entity/Article.php
@@ -9,6 +9,7 @@ use Doctrine\Common\Collections\Criteria;
 use Doctrine\ORM\Mapping as ORM;
 use Gedmo\Mapping\Annotation as Gedmo;
 use Gedmo\Timestampable\Traits\TimestampableEntity;
+use Knp\DoctrineBehaviors\Model\Translatable\Translatable;
 use Symfony\Component\Validator\Constraints as Assert;
 use Symfony\Component\Validator\Context\ExecutionContextInterface;
 
@@ -18,6 +19,7 @@ use Symfony\Component\Validator\Context\ExecutionContextInterface;
 class Article
 {
     use TimestampableEntity;
+    use Translatable;
 
     /**
      * @ORM\Id()
diff --git a/src/Entity/ArticleTranslation.php b/src/Entity/ArticleTranslation.php
index 675e96f..96dbf6d 100644
--- a/src/Entity/ArticleTranslation.php
+++ b/src/Entity/ArticleTranslation.php
@@ -3,10 +3,12 @@
 namespace App\Entity;
 
 use Doctrine\ORM\Mapping as ORM;
+use Knp\DoctrineBehaviors\Model\Translatable\Translation;
 
 /**
  * @ORM\Entity()
  */
 class ArticleTranslation
 {
+    use Translation;
 }
