diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index a3ce115a..1d7985f1 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -29,6 +29,7 @@ class ArticleAdminController extends BaseController
             $article = $form->getData();
 
             $em->persist($article);
+            $article->mergeNewTranslations();
             $em->flush();
 
             $message = $translator->trans('flash.success.article.created');
