diff --git a/src/DataFixtures/ArticleFixtures.php b/src/DataFixtures/ArticleFixtures.php
index 3f1ae085..bda813d7 100644
--- a/src/DataFixtures/ArticleFixtures.php
+++ b/src/DataFixtures/ArticleFixtures.php
@@ -26,8 +26,8 @@ class ArticleFixtures extends BaseFixture implements DependentFixtureInterface
     {
         $this->createMany(10, 'main_articles', function($count) use ($manager) {
             $article = new Article();
-            $article->setTitle($this->faker->randomElement(self::$articleTitles))
-                ->setContent(<<<EOF
+            $article->translate()->setTitle($this->faker->randomElement(self::$articleTitles));
+            $article->translate()->setContent(<<<EOF
 Spicy **jalapeno bacon** ipsum dolor amet veniam shank in dolore. Ham hock nisi landjaeger cow,
 lorem proident [beef ribs](https://baconipsum.com/) aute enim veniam ut cillum pork chuck picanha. Dolore reprehenderit
 labore minim pork belly spare ribs cupim short loin in. Elit exercitation eiusmod dolore cow
