diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 1b6badf4..a60aae35 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -55,6 +55,7 @@ class ArticleAdminController extends BaseController
         $form->handleRequest($request);
         if ($form->isSubmitted() && $form->isValid()) {
             $em->persist($article);
+            $article->mergeNewTranslations();
             $em->flush();
 
             $message = $translator->trans('flash.success.article.created');
