diff --git a/src/Entity/Article.php b/src/Entity/Article.php
index 4f55e850..58bac92f 100644
--- a/src/Entity/Article.php
+++ b/src/Entity/Article.php
@@ -84,6 +84,11 @@ class Article
             ->getValue($this->translate(), $method);
     }
 
+    public function translateWithoutFallback()
+    {
+        return $this->translate(null, false);
+    }
+
     public function getId()
     {
         return $this->id;
diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index 80a7fa18..ca1f0d14 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -34,12 +34,12 @@ class ArticleFormType extends AbstractType
 
         $builder
             ->add('title', TextType::class, [
-                'property_path' => 'translate.title',
+                'property_path' => 'translateWithoutFallback.title',
                 'label' => 'form.article.title',
                 'help' => 'form.article.title.help'
             ])
             ->add('content', TextareaType::class, [
-                'property_path' => 'translate.content',
+                'property_path' => 'translateWithoutFallback.content',
                 'label' => 'form.article.content',
                 'rows' => 15
             ])
