diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 5a81ba86..35b5a860 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -142,7 +142,7 @@ class ArticleAdminController extends BaseController
             ResponseHeaderBag::DISPOSITION_ATTACHMENT,
             $reference->getOriginalFilename()
         );
-        dd($disposition);
+        $response->headers->set('Content-Disposition', $disposition);
 
         return $response;
     }
