diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 7dfffa0a..47b437d9 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -9,11 +9,13 @@ use App\Repository\ArticleRepository;
 use App\Service\UploaderHelper;
 use Doctrine\ORM\EntityManagerInterface;
 use Gedmo\Sluggable\Util\Urlizer;
+use League\Flysystem\FilesystemInterface;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\File\UploadedFile;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\StreamedResponse;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\Validator\Constraints\File;
 use Symfony\Component\Validator\Constraints\NotBlank;
@@ -149,11 +151,19 @@ class ArticleAdminController extends BaseController
     /**
      * @Route("/admin/article/references/{id}/download", name="admin_article_download_reference", methods={"GET"})
      */
-    public function downloadArticleReference(ArticleReference $reference)
+    public function downloadArticleReference(ArticleReference $reference, FilesystemInterface $privateUploadsFilesystem)
     {
         $article = $reference->getArticle();
         $this->denyAccessUnlessGranted('MANAGE', $article);
-        dd($reference);
+
+        $response = new StreamedResponse(function() use ($reference, $privateUploadsFilesystem) {
+            $outputStream = fopen('php://output', 'wb');
+            $fileStream = $privateUploadsFilesystem->readStream($reference->getFilePath());
+
+            stream_copy_to_stream($fileStream, $outputStream);
+        });
+
+        return $response;
     }
 
     /**
diff --git a/src/Entity/ArticleReference.php b/src/Entity/ArticleReference.php
index c7c13f31..0e5dd3eb 100644
--- a/src/Entity/ArticleReference.php
+++ b/src/Entity/ArticleReference.php
@@ -2,6 +2,7 @@
 
 namespace App\Entity;
 
+use App\Service\UploaderHelper;
 use Doctrine\ORM\Mapping as ORM;
 
 /**
@@ -87,4 +88,9 @@ class ArticleReference
 
         return $this;
     }
+
+    public function getFilePath(): string
+    {
+        return UploaderHelper::ARTICLE_REFERENCE.'/'.$this->getFilename();
+    }
 }
