diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index 561c1fc..5a693ff 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -72,4 +72,12 @@ class ArticleReferenceAdminController extends BaseController
             'id' => $article->getId(),
         ]);
     }
+
+    /**
+     * @Route("/admin/article/references/{id}/download", name="admin_article_download_reference", methods={"GET"})
+     */
+    public function downloadArticleReference(ArticleReference $reference)
+    {
+        dd($reference);
+    }
 }
diff --git a/templates/article_admin/edit.html.twig b/templates/article_admin/edit.html.twig
index 557ccf7..83bf5d8 100644
--- a/templates/article_admin/edit.html.twig
+++ b/templates/article_admin/edit.html.twig
@@ -19,6 +19,12 @@
                 {% for reference in article.articleReferences %}
                     <li class="list-group-item d-flex justify-content-between align-items-center">
                         {{ reference.originalFilename }}
+
+                        <span>
+                            <a href="{{ path('admin_article_download_reference', {
+                                id: reference.id
+                            }) }}"><span class="fa fa-download"></span></a>
+                        </span>
                     </li>
                 {% endfor %}
             </ul>
