diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index dfbec902..9c5d1145 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -171,7 +171,7 @@ class ArticleAdminController extends BaseController
             ResponseHeaderBag::DISPOSITION_ATTACHMENT,
             $reference->getOriginalFilename()
         );
-        dd($disposition);
+        $response->headers->set('Content-Disposition', $disposition);
 
         return $response;
     }
