diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index c2a9227e..d6d46519 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -87,6 +87,7 @@ class ArticleReferenceAdminController extends BaseController
 
             stream_copy_to_stream($fileStream, $outputStream);
         });
+        $response->headers->set('Content-Type', $reference->getMimeType());
 
         return $response;
     }
