diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index 5a693ff..4ab42e6 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -78,6 +78,8 @@ class ArticleReferenceAdminController extends BaseController
      */
     public function downloadArticleReference(ArticleReference $reference)
     {
+        $article = $reference->getArticle();
+        $this->denyAccessUnlessGranted('MANAGE', $article);
         dd($reference);
     }
 }
