diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index 412ef32b..b363052a 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -88,6 +88,7 @@ class ArticleReferenceAdminController extends BaseController
 
             stream_copy_to_stream($fileStream, $outputStream);
         });
+        $response->headers->set('Content-Type', $reference->getMimeType());
 
         return $response;
     }
