diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index b8f778c4..dfbec902 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -16,6 +16,7 @@ use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\File\UploadedFile;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\ResponseHeaderBag;
 use Symfony\Component\HttpFoundation\StreamedResponse;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\Validator\Constraints\File;
@@ -166,6 +167,11 @@ class ArticleAdminController extends BaseController
 
             stream_copy_to_stream($fileStream, $outputStream);
         });
+        $disposition = $response->headers->makeDisposition(
+            ResponseHeaderBag::DISPOSITION_ATTACHMENT,
+            $reference->getOriginalFilename()
+        );
+        dd($disposition);
 
         return $response;
     }
