diff --git a/templates/article_admin/edit.html.twig b/templates/article_admin/edit.html.twig
index 1bff3fa02..557ccf796 100644
--- a/templates/article_admin/edit.html.twig
+++ b/templates/article_admin/edit.html.twig
@@ -15,6 +15,16 @@
         <div class="col-sm-4">
             <h3>References</h3>
 
+            <ul class="list-group small">
+                {% for reference in article.articleReferences %}
+                    <li class="list-group-item d-flex justify-content-between align-items-center">
+                        {{ reference.originalFilename }}
+                    </li>
+                {% endfor %}
+            </ul>
+
+            <hr>
+
             <form action="{{ path('admin_article_add_reference', {
                 id: article.id
             }) }}" method="POST" enctype="multipart/form-data">
