diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index ee16519c..973db55a 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -92,7 +92,7 @@ class ArticleReferenceAdminController extends BaseController
             ResponseHeaderBag::DISPOSITION_ATTACHMENT,
             $reference->getOriginalFilename()
         );
-        dd($disposition);
+        $response->headers->set('Content-Disposition', $disposition);
 
         return $response;
     }
