diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 80204ef8..7e32a874 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -167,7 +167,7 @@ class ArticleAdminController extends BaseController
             ResponseHeaderBag::DISPOSITION_ATTACHMENT,
             $reference->getOriginalFilename()
         );
-        dd($disposition);
+        $response->headers->set('Content-Disposition', $disposition);
 
         return $response;
     }
