diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index 0ea97f65..f7500295 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -71,4 +71,12 @@ class ArticleReferenceAdminController extends BaseController
             'id' => $article->getId(),
         ]);
     }
+
+    /**
+     * @Route("/admin/article/references/{id}/download", name="admin_article_download_reference", methods={"GET"})
+     */
+    public function downloadArticleReference(ArticleReference $reference)
+    {
+        dd($reference);
+    }
 }
diff --git a/templates/article_admin/edit.html.twig b/templates/article_admin/edit.html.twig
index 46b3b4f5..cd3a2eea 100644
--- a/templates/article_admin/edit.html.twig
+++ b/templates/article_admin/edit.html.twig
@@ -17,7 +17,9 @@
 
             <div class="list-group">
                 {% for reference in article.articleReferences %}
-                    <a href="#" class="list-group-item list-group-item-action">
+                    <a href="{{ path('admin_article_download_reference', {
+                        id: reference.id
+                    }) }}" class="list-group-item list-group-item-action">
                         {{ reference.originalFilename }}
                     </a>
                 {% endfor %}
