diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index a6fcc19e..c2a9227e 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -9,6 +9,7 @@ use Doctrine\ORM\EntityManagerInterface;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Symfony\Component\HttpFoundation\File\UploadedFile;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\StreamedResponse;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\Validator\Constraints\File;
 use Symfony\Component\Validator\Constraints\NotBlank;
@@ -75,10 +76,18 @@ class ArticleReferenceAdminController extends BaseController
     /**
      * @Route("/admin/article/references/{id}/download", name="admin_article_download_reference", methods={"GET"})
      */
-    public function downloadArticleReference(ArticleReference $reference)
+    public function downloadArticleReference(ArticleReference $reference, UploaderHelper $uploaderHelper)
     {
         $article = $reference->getArticle();
         $this->denyAccessUnlessGranted('MANAGE', $article);
-        dd($reference);
+
+        $response = new StreamedResponse(function() use ($reference, $uploaderHelper) {
+            $outputStream = fopen('php://output', 'wb');
+            $fileStream = $uploaderHelper->readStream($reference->getFilePath(), false);
+
+            stream_copy_to_stream($fileStream, $outputStream);
+        });
+
+        return $response;
     }
 }
diff --git a/src/Entity/ArticleReference.php b/src/Entity/ArticleReference.php
index c7c13f31..0e5dd3eb 100644
--- a/src/Entity/ArticleReference.php
+++ b/src/Entity/ArticleReference.php
@@ -2,6 +2,7 @@
 
 namespace App\Entity;
 
+use App\Service\UploaderHelper;
 use Doctrine\ORM\Mapping as ORM;
 
 /**
@@ -87,4 +88,9 @@ class ArticleReference
 
         return $this;
     }
+
+    public function getFilePath(): string
+    {
+        return UploaderHelper::ARTICLE_REFERENCE.'/'.$this->getFilename();
+    }
 }
diff --git a/src/Service/UploaderHelper.php b/src/Service/UploaderHelper.php
index ba93cf0d..8c8cdf71 100644
--- a/src/Service/UploaderHelper.php
+++ b/src/Service/UploaderHelper.php
@@ -65,6 +65,22 @@ class UploaderHelper
             ->getBasePath().$this->publicAssetBaseUrl.'/'.$path;
     }
 
+    /**
+     * @return resource
+     */
+    public function readStream(string $path, bool $isPublic)
+    {
+        $filesystem = $isPublic ? $this->filesystem : $this->privateFilesystem;
+
+        $resource = $filesystem->readStream($path);
+
+        if ($resource === false) {
+            throw new \Exception(sprintf('Error opening stream for "%s"', $path));
+        }
+
+        return $resource;
+    }
+
     private function uploadFile(File $file, string $directory, bool $isPublic)
     {
         if ($file instanceof UploadedFile) {
