diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index 8fc9a85..e3a6e82 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -94,7 +94,7 @@ class ArticleReferenceAdminController extends BaseController
             HeaderUtils::DISPOSITION_ATTACHMENT,
             $reference->getOriginalFilename()
         );
-        dd($disposition);
+        $response->headers->set('Content-Disposition', $disposition);
 
         return $response;
     }
