diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index e578138e..b066a1a3 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -65,7 +65,11 @@ class ArticleReferenceAdminController extends BaseController
         return $this->json(
             $articleReference,
             201,
-            [],
+            [
+                'Location' => $this->generateUrl('admin_article_download_reference', [
+                    'id' => $articleReference->getId(),
+                ])
+            ],
             [
                 'groups' => ['main']
             ]
