diff --git a/public/js/admin_article_form.js b/public/js/admin_article_form.js
index dc1d1f4d..610def49 100644
--- a/public/js/admin_article_form.js
+++ b/public/js/admin_article_form.js
@@ -74,6 +74,10 @@ function initializeDropzone()
     var dropzone = new Dropzone(inputElement, {
         paramName: 'reference',
         init: function() {
+            this.on('success', function(file, data) {
+                console.log(file, data);
+            });
+
             this.on('error', function(file, data) {
                 if (data.detail) {
                     this.emit('error', file, data.detail);
