diff --git a/public/js/admin_article_form.js b/public/js/admin_article_form.js
index 96f2dc7..9af5088 100644
--- a/public/js/admin_article_form.js
+++ b/public/js/admin_article_form.js
@@ -3,7 +3,7 @@ Dropzone.autoDiscover = false;
 $(document).ready(function() {
     var referenceList = new ReferenceList($('.js-reference-list'));
 
-    initializeDropzone();
+    initializeDropzone(referenceList);
 
     var $locationSelect = $('.js-article-form-location');
     var $specificLocationTarget = $('.js-specific-location-target');
@@ -47,6 +47,11 @@ class ReferenceList
         })
     }
 
+    addReference(reference) {
+        this.references.push(reference);
+        this.render();
+    }
+
     render() {
         const itemsHtml = this.references.map(reference => {
             return `
@@ -64,7 +69,10 @@ class ReferenceList
     }
 }
 
-function initializeDropzone() {
+/**
+ * @param {ReferenceList} referenceList
+ */
+function initializeDropzone(referenceList) {
     var formElement = document.querySelector('.js-reference-dropzone');
     if (!formElement) {
         return;
@@ -74,7 +82,7 @@ function initializeDropzone() {
         paramName: 'reference',
         init: function() {
             this.on('success', function(file, data) {
-                console.log(file, data);
+                referenceList.addReference(data);
             });
 
             this.on('error', function(file, data) {
