diff --git a/public/js/admin_article_form.js b/public/js/admin_article_form.js
index 1a8b27df..8ada9bf8 100644
--- a/public/js/admin_article_form.js
+++ b/public/js/admin_article_form.js
@@ -38,6 +38,13 @@ function initializeDropzone()
     console.log(inputElement);
 
     var dropzone = new Dropzone(inputElement, {
-        paramName: 'reference'
+        paramName: 'reference',
+        init: function() {
+            this.on('error', function(file, data) {
+                if (data.detail) {
+                    this.emit('error', file, data.detail);
+                }
+            });
+        }
     });
 }
