diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 862e0762..894c864e 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -140,7 +140,11 @@ class ArticleAdminController extends BaseController
         return $this->json(
             $articleReference,
             201,
-            [],
+            [
+                'Location' => $this->generateUrl('admin_article_download_reference', [
+                    'id' => $articleReference->getId(),
+                ])
+            ],
             [
                 'groups' => ['main']
             ]
