diff --git a/public/js/admin_article_form.js b/public/js/admin_article_form.js
index 610def49..771850c6 100644
--- a/public/js/admin_article_form.js
+++ b/public/js/admin_article_form.js
@@ -3,7 +3,7 @@ Dropzone.autoDiscover = false;
 $(document).ready(function() {
     var referenceList = new ReferenceList(document.querySelector('.js-reference-list'));
 
-    initializeDropzone();
+    initializeDropzone(referenceList);
 
     var $locationSelect = $('.js-article-form-location');
     var $specificLocationTarget = $('.js-specific-location-target');
@@ -47,6 +47,11 @@ class ReferenceList
         })
     }
 
+    addReference(reference) {
+        this.references.push(reference);
+        this.render();
+    }
+
     render() {
         const itemsHtml = this.references.map(reference => {
             return `
@@ -64,7 +69,10 @@ class ReferenceList
     }
 }
 
-function initializeDropzone()
+/**
+ * @param {ReferenceList} referenceList
+ */
+function initializeDropzone(referenceList)
 {
     var inputElement = document.querySelector('.js-reference-dropzone');
     if (!inputElement) {
@@ -75,7 +83,7 @@ function initializeDropzone()
         paramName: 'reference',
         init: function() {
             this.on('success', function(file, data) {
-                console.log(file, data);
+                referenceList.addReference(data);
             });
 
             this.on('error', function(file, data) {
