diff --git a/public/js/admin_article_form.js b/public/js/admin_article_form.js
index bec4c243..143d959e 100644
--- a/public/js/admin_article_form.js
+++ b/public/js/admin_article_form.js
@@ -1,4 +1,8 @@
+Dropzone.autoDiscover = false;
+
 $(document).ready(function() {
+    initializeDropzone();
+
     var $locationSelect = $('.js-article-form-location');
     var $specificLocationTarget = $('.js-specific-location-target');
 
@@ -24,3 +28,14 @@ $(document).ready(function() {
         });
     });
 });
+
+function initializeDropzone() {
+    var formElement = document.querySelector('.js-reference-dropzone');
+    if (!formElement) {
+        return;
+    }
+
+    var dropzone = new Dropzone(formElement, {
+        paramName: 'reference'
+    });
+}
diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index c010682d..e3a6e826 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -27,7 +27,6 @@ class ArticleReferenceAdminController extends BaseController
     {
         /** @var UploadedFile $uploadedFile */
         $uploadedFile = $request->files->get('reference');
-        dump($uploadedFile);
 
         $violations = $validator->validate(
             $uploadedFile,
diff --git a/templates/article_admin/edit.html.twig b/templates/article_admin/edit.html.twig
index 9c2f8021..a3de83b4 100644
--- a/templates/article_admin/edit.html.twig
+++ b/templates/article_admin/edit.html.twig
@@ -33,7 +33,7 @@
 
             <form action="{{ path('admin_article_add_reference', {
                 id: article.id
-            }) }}" method="POST" enctype="multipart/form-data" class="dropzone">
+            }) }}" method="POST" enctype="multipart/form-data" class="dropzone js-reference-dropzone">
             </form>
         </div>
     </div>
