diff --git a/public/js/admin_article_form.js b/public/js/admin_article_form.js
index 143d959e2..a589c8ce2 100644
--- a/public/js/admin_article_form.js
+++ b/public/js/admin_article_form.js
@@ -36,6 +36,13 @@ function initializeDropzone() {
     }
 
     var dropzone = new Dropzone(formElement, {
-        paramName: 'reference'
+        paramName: 'reference',
+        init: function() {
+            this.on('error', function(file, data) {
+                if (data.detail) {
+                    this.emit('error', file, data.detail);
+                }
+            });
+        }
     });
 }
