diff --git a/src/Repository/UserRepository.php b/src/Repository/UserRepository.php
index 1c40dec..17fbe7a 100644
--- a/src/Repository/UserRepository.php
+++ b/src/Repository/UserRepository.php
@@ -57,9 +57,13 @@ class UserRepository extends ServiceEntityRepository implements PasswordUpgrader
             ->getResult();
     }
 
+    /**
+     * @param User $user
+     */
     public function upgradePassword(UserInterface $user, string $newEncodedPassword): void
     {
-
+        $user->setPassword($newEncodedPassword);
+        $this->getEntityManager()->flush($user);
     }
 
 //    /**
