diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index ae5ffab4..8b8b8013 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -26,7 +26,8 @@ class ArticleFormType extends AbstractType
                 'class' => User::class,
                 'choice_label' => function(User $user) {
                     return sprintf('(%d) %s', $user->getId(), $user->getEmail());
-                }
+                },
+                'placeholder' => 'Choose an author'
             ])
         ;
     }
