diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index e050eb59..427e66b2 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -24,6 +24,7 @@ class ArticleFormType extends AbstractType
             ])
             ->add('author', EntityType::class, [
                 'class' => User::class,
+                'choice_label' => 'email',
             ])
         ;
     }
