diff --git a/.env.dist b/.env.dist
index 428a79fd..ceef5fd0 100644
--- a/.env.dist
+++ b/.env.dist
@@ -2,6 +2,10 @@
 # Copy this file to .env file for development, create environment variables when deploying to production
 # https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
 
+### CUSTOM VARS
+SLACK_WEBHOOK_ENDPOINT=https://hooks.slack.com/...
+### END CUSTOM VARS
+
 ###> symfony/framework-bundle ###
 APP_ENV=dev
 APP_SECRET=5ea3114a349591bd131296e00f21c20a
diff --git a/config/packages/nexy_slack.yaml b/config/packages/nexy_slack.yaml
index 5b4bedb8..abc371c1 100644
--- a/config/packages/nexy_slack.yaml
+++ b/config/packages/nexy_slack.yaml
@@ -1,2 +1,2 @@
 nexy_slack:
-    endpoint: 'https://hooks.slack.com/services/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'
+    endpoint: '%env(SLACK_WEBHOOK_ENDPOINT)%'
