diff --git a/templates/article/show.html.twig b/templates/article/show.html.twig
index 386baa63..155abb1e 100644
--- a/templates/article/show.html.twig
+++ b/templates/article/show.html.twig
@@ -15,7 +15,7 @@
                             <span class="show-article-title ">{{ title }}</span>
                             <br>
                             <span class="align-left article-details"><img class="article-author-img rounded-circle" src="{{ asset('images/alien-profile.png') }}"> Mike Ferengi </span>
-                            <span class="pl-2 article-details"> 3 hours ago</span>
+                            <span class="pl-2 article-details"> 4 hours ago</span>
                             <span class="pl-2 article-details">
                                 <span class="js-like-article-count">5</span>
                                 <a href="{{ path('article_toggle_heart', {slug: slug}) }}" class="fa fa-heart-o like-article js-like-article"></a>
