diff --git a/src/Migrations/Version20180418130337.php b/src/Migrations/Version20180418130337.php
index 38be73f2..4c3c7803 100644
--- a/src/Migrations/Version20180418130337.php
+++ b/src/Migrations/Version20180418130337.php
@@ -15,7 +15,8 @@ class Version20180418130337 extends AbstractMigration
         // this up() migration is auto-generated, please modify it to your needs
         $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
 
-        $this->addSql('ALTER TABLE article ADD created_at DATETIME NOT NULL, ADD updated_at DATETIME NOT NULL');
+        $this->addSql('ALTER TABLE article ADD created_at DATETIME DEFAULT NULL, ADD updated_at DATETIME DEFAULT NULL');
+        $this->addSql('UPDATE article SET created_at = NOW(), updated_at = NOW()');
     }
 
     public function down(Schema $schema)
