diff --git a/config/packages/stof_doctrine_extensions.yaml b/config/packages/stof_doctrine_extensions.yaml
index e7ac8f4a..a89cf08f 100644
--- a/config/packages/stof_doctrine_extensions.yaml
+++ b/config/packages/stof_doctrine_extensions.yaml
@@ -5,3 +5,4 @@ stof_doctrine_extensions:
     orm:
         default:
             sluggable: true
+            timestampable: true
diff --git a/src/Entity/Article.php b/src/Entity/Article.php
index 9f74c328..4a80d778 100644
--- a/src/Entity/Article.php
+++ b/src/Entity/Article.php
@@ -55,11 +55,13 @@ class Article
 
     /**
      * @ORM\Column(type="datetime")
+     * @Gedmo\Timestampable(on="create")
      */
     private $createdAt;
 
     /**
      * @ORM\Column(type="datetime")
+     * @Gedmo\Timestampable(on="update")
      */
     private $updatedAt;
 
