diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index 4d677d02..a13d78af 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -45,7 +45,10 @@ class ArticleController extends AbstractController
         }
 
         $comments = $article->getComments();
-        dump($comments);die;
+        foreach ($comments as $comment) {
+            dump($comment);
+        }
+        die;
 
         $comments = [
             'I ate a normal rock once. It did NOT taste like bacon!',
