diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 701b9cb5f..89ba5cbbb 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -76,9 +76,11 @@ class ArticleAdminController extends BaseController
         /** @var UploadedFile $uploadedFile */
         $uploadedFile = $request->files->get('image');
         $destination = $this->getParameter('kernel.project_dir').'/public/uploads';
+        $newFilename = uniqid().'-'.$uploadedFile->getClientOriginalName();
+
         dd($uploadedFile->move(
             $destination,
-            $uploadedFile->getClientOriginalName()
+            $newFilename
         ));
     }
 
