diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 7600cc4b..6e3bef44 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -76,7 +76,9 @@ class ArticleAdminController extends BaseController
         /** @var UploadedFile $uploadedFile */
         $uploadedFile = $request->files->get('image');
         $destination = $this->getParameter('kernel.project_dir').'/public/uploads';
-        $newFilename = uniqid().'-'.$uploadedFile->getClientOriginalName();
+
+        $originalFilename = pathinfo($uploadedFile->getClientOriginalName(), PATHINFO_FILENAME);
+        $newFilename = $originalFilename.'-'.uniqid().'.'.$uploadedFile->guessExtension();
 
         dd($uploadedFile->move(
             $destination,
