diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index dd1d784b..5c31db06 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -4,6 +4,7 @@ namespace App\Controller;
 
 use App\Entity\Article;
 use App\Form\ArticleFormType;
+use App\Repository\ArticleRepository;
 use Doctrine\ORM\EntityManagerInterface;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
@@ -48,4 +49,16 @@ class ArticleAdminController extends AbstractController
     {
         dd($article);
     }
+
+    /**
+     * @Route("/admin/article")
+     */
+    public function list(ArticleRepository $articleRepo)
+    {
+        $articles = $articleRepo->findAll();
+
+        return $this->render('article_admin/list.html.twig', [
+            'articles' => $articles,
+        ]);
+    }
 }
diff --git a/src/Entity/Article.php b/src/Entity/Article.php
index 29137ec0..62ef8a23 100644
--- a/src/Entity/Article.php
+++ b/src/Entity/Article.php
@@ -125,6 +125,11 @@ class Article
         return $this->publishedAt;
     }
 
+    public function isPublished(): bool
+    {
+        return $this->publishedAt !== null;
+    }
+
     public function setPublishedAt(?\DateTimeInterface $publishedAt): self
     {
         $this->publishedAt = $publishedAt;
diff --git a/templates/article_admin/list.html.twig b/templates/article_admin/list.html.twig
new file mode 100644
index 00000000..9d9113a0
--- /dev/null
+++ b/templates/article_admin/list.html.twig
@@ -0,0 +1,37 @@
+{% extends 'content_base.html.twig' %}
+
+{% block content_body %}
+    <a href="{{ path('admin_article_new') }}" class="btn btn-primary pull-right">
+        Create <span class="fa fa-plus-circle"></span>
+    </a>
+    <h1>All Articles</h1>
+
+    <table class="table">
+        <thead>
+            <tr>
+                <th>Title</th>
+                <th>Author</th>
+                <th>Published?</th>
+                <th>&nbsp;</th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for article in articles %}
+                <tr>
+                    <td>{{ article.title }}</td>
+                    <td>{{ article.author.email }}</td>
+                    <td>
+                        <span class="fa fa-{{ article.isPublished ? 'check' : 'times' }}"></span>
+                    </td>
+                    <td>
+                        <a href="{{ path('admin_article_edit', {
+                            id: article.id
+                        }) }}">
+                            <span class="fa fa-pencil"></span>
+                        </a>
+                    </td>
+                </tr>
+            {% endfor %}
+        </tbody>
+    </table>
+{% endblock %}
