diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 446e1a6f..abeb944c 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -7,6 +7,7 @@ use App\Form\ArticleFormType;
 use Doctrine\ORM\EntityManagerInterface;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
@@ -16,10 +17,15 @@ class ArticleAdminController extends AbstractController
      * @Route("/admin/article/new", name="admin_article_new")
      * @IsGranted("ROLE_ADMIN_ARTICLE")
      */
-    public function new(EntityManagerInterface $em)
+    public function new(EntityManagerInterface $em, Request $request)
     {
         $form = $this->createForm(ArticleFormType::class);
 
+        $form->handleRequest($request);
+        if ($form->isSubmitted() && $form->isValid()) {
+            dd($form->getData());
+        }
+
         return $this->render('article_admin/new.html.twig', [
             'articleForm' => $form->createView()
         ]);
