diff --git a/src/DataFixtures/CommentFixture.php b/src/DataFixtures/CommentFixture.php
index 08e1f70..e51ad46 100644
--- a/src/DataFixtures/CommentFixture.php
+++ b/src/DataFixtures/CommentFixture.php
@@ -16,7 +16,7 @@ class CommentFixture extends BaseFixture
             );
 
             $comment->setAuthorName($this->faker->name);
-            $comment->setArticle($this->getReference(Article::class.'_0'));
+            $comment->setArticle($this->getReference(Article::class.'_'.$this->faker->numberBetween(0, 9)));
         });
 
         $manager->flush();
