diff --git a/src/DataFixtures/CommentFixture.php b/src/DataFixtures/CommentFixture.php
index e527eb74..cf8ab397 100644
--- a/src/DataFixtures/CommentFixture.php
+++ b/src/DataFixtures/CommentFixture.php
@@ -2,14 +2,21 @@
 
 namespace App\DataFixtures;
 
+use App\Entity\Comment;
 use Doctrine\Common\Persistence\ObjectManager;
 
 class CommentFixture extends BaseFixture
 {
     protected function loadData(ObjectManager $manager)
     {
-        // $product = new Product();
-        // $manager->persist($product);
+        $this->createMany(Comment::class, 100, function(Comment $comment) {
+            $comment->setContent(
+                $this->faker->boolean ? $this->faker->paragraph : $this->faker->sentences(2, true)
+            );
+
+            $comment->setAuthorName($this->faker->name);
+            $comment->setCreatedAt($this->faker->dateTimeBetween('-1 months', '-1 seconds'));
+        });
 
         $manager->flush();
     }
