diff --git a/src/DataFixtures/A0CommentFixture.php b/src/DataFixtures/CommentFixture.php
similarity index 91%
rename from src/DataFixtures/A0CommentFixture.php
rename to src/DataFixtures/CommentFixture.php
index 1296fb76..13d15ef5 100644
--- a/src/DataFixtures/A0CommentFixture.php
+++ b/src/DataFixtures/CommentFixture.php
@@ -7,7 +7,7 @@ use App\Entity\Comment;
 use Doctrine\Common\DataFixtures\DependentFixtureInterface;
 use Doctrine\Common\Persistence\ObjectManager;
 
-class A0CommentFixture extends BaseFixture implements DependentFixtureInterface
+class CommentFixture extends BaseFixture implements DependentFixtureInterface
 {
     protected function loadData(ObjectManager $manager)
     {
