diff --git a/src/DataFixtures/CommentFixture.php b/src/DataFixtures/CommentFixture.php
index 75e01dc6..633d98c6 100644
--- a/src/DataFixtures/CommentFixture.php
+++ b/src/DataFixtures/CommentFixture.php
@@ -17,7 +17,7 @@ class CommentFixture extends BaseFixture
 
             $comment->setAuthorName($this->faker->name);
             $comment->setCreatedAt($this->faker->dateTimeBetween('-1 months', '-1 seconds'));
-            $comment->setArticle($this->getReference(Article::class.'_0'));
+            $comment->setArticle($this->getReference(Article::class.'_'.$this->faker->numberBetween(0, 9)));
         });
 
         $manager->flush();
