diff --git a/src/DataFixtures/ArticleFixture.php b/src/DataFixtures/ArticleFixture.php
index e750029..6513c1e 100644
--- a/src/DataFixtures/ArticleFixture.php
+++ b/src/DataFixtures/ArticleFixture.php
@@ -3,12 +3,11 @@
 namespace App\DataFixtures;
 
 use App\Entity\Article;
-use Doctrine\Bundle\FixturesBundle\Fixture;
 use Doctrine\Common\Persistence\ObjectManager;
 
-class ArticleFixture extends Fixture
+class ArticleFixture extends BaseFixture
 {
-    public function load(ObjectManager $manager)
+    public function loadData(ObjectManager $manager)
     {
         $article = new Article();
         $article->setTitle('Why Asteroids Taste Like Bacon')
diff --git a/src/DataFixtures/BaseFixture.php b/src/DataFixtures/BaseFixture.php
new file mode 100644
index 0000000..c3216f5
--- /dev/null
+++ b/src/DataFixtures/BaseFixture.php
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\DataFixtures;
+
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\Persistence\ObjectManager;
+
+abstract class BaseFixture extends Fixture
+{
+    /** @var ObjectManager */
+    private $manager;
+
+    abstract protected function loadData(ObjectManager $manager);
+
+    public function load(ObjectManager $manager)
+    {
+        $this->manager = $manager;
+
+        $this->loadData($manager);
+    }
+}
