diff --git a/src/DataFixtures/CommentFixture.php b/src/DataFixtures/CommentFixture.php
index cf8ab397..75e01dc6 100644
--- a/src/DataFixtures/CommentFixture.php
+++ b/src/DataFixtures/CommentFixture.php
@@ -2,6 +2,7 @@
 
 namespace App\DataFixtures;
 
+use App\Entity\Article;
 use App\Entity\Comment;
 use Doctrine\Common\Persistence\ObjectManager;
 
@@ -16,6 +17,7 @@ class CommentFixture extends BaseFixture
 
             $comment->setAuthorName($this->faker->name);
             $comment->setCreatedAt($this->faker->dateTimeBetween('-1 months', '-1 seconds'));
+            $comment->setArticle($this->getReference(Article::class.'_0'));
         });
 
         $manager->flush();
