diff --git a/src/DataFixtures/CommentFixture.php b/src/DataFixtures/CommentFixture.php
index a5d20150..e527eb74 100644
--- a/src/DataFixtures/CommentFixture.php
+++ b/src/DataFixtures/CommentFixture.php
@@ -2,12 +2,11 @@
 
 namespace App\DataFixtures;
 
-use Doctrine\Bundle\FixturesBundle\Fixture;
 use Doctrine\Common\Persistence\ObjectManager;
 
-class CommentFixture extends Fixture
+class CommentFixture extends BaseFixture
 {
-    public function load(ObjectManager $manager)
+    protected function loadData(ObjectManager $manager)
     {
         // $product = new Product();
         // $manager->persist($product);
