diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index e31ddad0..941a70b7 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -15,41 +15,7 @@ class ArticleAdminController extends AbstractController
      */
     public function new(EntityManagerInterface $em)
     {
-        $article = new Article();
-        $article->setTitle('Why Asteroids Taste Like Bacon')
-            ->setSlug('why-asteroids-taste-like-bacon-'.rand(100, 999))
-            ->setContent(<<<EOF
-Spicy **jalapeno bacon** ipsum dolor amet veniam shank in dolore. Ham hock nisi landjaeger cow,
-lorem proident [beef ribs](https://baconipsum.com/) aute enim veniam ut cillum pork chuck picanha. Dolore reprehenderit
-labore minim pork belly spare ribs cupim short loin in. Elit exercitation eiusmod dolore cow
-**turkey** shank eu pork belly meatball non cupim.
-
-Laboris beef ribs fatback fugiat eiusmod jowl kielbasa alcatra dolore velit ea ball tip. Pariatur
-laboris sunt venison, et laborum dolore minim non meatball. Shankle eu flank aliqua shoulder,
-capicola biltong frankfurter boudin cupim officia. Exercitation fugiat consectetur ham. Adipisicing
-picanha shank et filet mignon pork belly ut ullamco. Irure velit turducken ground round doner incididunt
-occaecat lorem meatball prosciutto quis strip steak.
-
-Meatball adipisicing ribeye bacon strip steak eu. Consectetur ham hock pork hamburger enim strip steak
-mollit quis officia meatloaf tri-tip swine. Cow ut reprehenderit, buffalo incididunt in filet mignon
-strip steak pork belly aliquip capicola officia. Labore deserunt esse chicken lorem shoulder tail consectetur
-cow est ribeye adipisicing. Pig hamburger pork belly enim. Do porchetta minim capicola irure pancetta chuck
-fugiat.
-EOF
-        );
-
-        // publish most articles
-        if (rand(1, 10) > 2) {
-            $article->setPublishedAt(new \DateTime(sprintf('-%d days', rand(1, 100))));
-        }
-
-        $article->setAuthor('Mike Ferengi')
-            ->setHeartCount(rand(5, 100))
-            ->setImageFilename('asteroid.jpeg')
-        ;
-
-        $em->persist($article);
-        $em->flush();
+        die('todo');
 
         return new Response(sprintf(
             'Hiya! New Article id: #%d slug: %s',
diff --git a/src/DataFixtures/ArticleFixtures.php b/src/DataFixtures/ArticleFixtures.php
index 09b28bf3..4938ca0f 100644
--- a/src/DataFixtures/ArticleFixtures.php
+++ b/src/DataFixtures/ArticleFixtures.php
@@ -2,6 +2,7 @@
 
 namespace App\DataFixtures;
 
+use App\Entity\Article;
 use Doctrine\Bundle\FixturesBundle\Fixture;
 use Doctrine\Common\Persistence\ObjectManager;
 
@@ -9,8 +10,40 @@ class ArticleFixtures extends Fixture
 {
     public function load(ObjectManager $manager)
     {
-        // $product = new Product();
-        // $manager->persist($product);
+        $article = new Article();
+        $article->setTitle('Why Asteroids Taste Like Bacon')
+            ->setSlug('why-asteroids-taste-like-bacon-'.rand(100, 999))
+            ->setContent(<<<EOF
+Spicy **jalapeno bacon** ipsum dolor amet veniam shank in dolore. Ham hock nisi landjaeger cow,
+lorem proident [beef ribs](https://baconipsum.com/) aute enim veniam ut cillum pork chuck picanha. Dolore reprehenderit
+labore minim pork belly spare ribs cupim short loin in. Elit exercitation eiusmod dolore cow
+**turkey** shank eu pork belly meatball non cupim.
+
+Laboris beef ribs fatback fugiat eiusmod jowl kielbasa alcatra dolore velit ea ball tip. Pariatur
+laboris sunt venison, et laborum dolore minim non meatball. Shankle eu flank aliqua shoulder,
+capicola biltong frankfurter boudin cupim officia. Exercitation fugiat consectetur ham. Adipisicing
+picanha shank et filet mignon pork belly ut ullamco. Irure velit turducken ground round doner incididunt
+occaecat lorem meatball prosciutto quis strip steak.
+
+Meatball adipisicing ribeye bacon strip steak eu. Consectetur ham hock pork hamburger enim strip steak
+mollit quis officia meatloaf tri-tip swine. Cow ut reprehenderit, buffalo incididunt in filet mignon
+strip steak pork belly aliquip capicola officia. Labore deserunt esse chicken lorem shoulder tail consectetur
+cow est ribeye adipisicing. Pig hamburger pork belly enim. Do porchetta minim capicola irure pancetta chuck
+fugiat.
+EOF
+        );
+
+        // publish most articles
+        if (rand(1, 10) > 2) {
+            $article->setPublishedAt(new \DateTime(sprintf('-%d days', rand(1, 100))));
+        }
+
+        $article->setAuthor('Mike Ferengi')
+            ->setHeartCount(rand(5, 100))
+            ->setImageFilename('asteroid.jpeg')
+        ;
+
+        $manager->persist($article);
 
         $manager->flush();
     }
