diff --git a/src/DataFixtures/CommentFixture.php b/src/DataFixtures/A0CommentFixture.php
similarity index 75%
rename from src/DataFixtures/CommentFixture.php
rename to src/DataFixtures/A0CommentFixture.php
index 4a4dfd32..1296fb76 100644
--- a/src/DataFixtures/CommentFixture.php
+++ b/src/DataFixtures/A0CommentFixture.php
@@ -4,9 +4,10 @@ namespace App\DataFixtures;
 
 use App\Entity\Article;
 use App\Entity\Comment;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
 use Doctrine\Common\Persistence\ObjectManager;
 
-class CommentFixture extends BaseFixture
+class A0CommentFixture extends BaseFixture implements DependentFixtureInterface
 {
     protected function loadData(ObjectManager $manager)
     {
@@ -22,4 +23,9 @@ class CommentFixture extends BaseFixture
 
         $manager->flush();
     }
+
+    public function getDependencies()
+    {
+        return [ArticleFixtures::class];
+    }
 }
