diff --git a/src/DataFixtures/ArticleFixtures.php b/src/DataFixtures/ArticleFixtures.php
index a001b2a2..44fc0f91 100644
--- a/src/DataFixtures/ArticleFixtures.php
+++ b/src/DataFixtures/ArticleFixtures.php
@@ -33,12 +33,12 @@ EOF
             );
 
             // publish most articles
-            if (rand(1, 10) > 2) {
-                $article->setPublishedAt(new \DateTime(sprintf('-%d days', rand(1, 100))));
+            if ($this->faker->boolean(70)) {
+                $article->setPublishedAt($this->faker->dateTimeBetween('-100 days', '-1 days'));
             }
 
             $article->setAuthor('Mike Ferengi')
-                ->setHeartCount(rand(5, 100))
+                ->setHeartCount($this->faker->numberBetween(5, 100))
                 ->setImageFilename('asteroid.jpeg')
             ;
         });
diff --git a/src/DataFixtures/BaseFixture.php b/src/DataFixtures/BaseFixture.php
index 31b93e23..a5e12fc9 100644
--- a/src/DataFixtures/BaseFixture.php
+++ b/src/DataFixtures/BaseFixture.php
@@ -4,17 +4,23 @@ namespace App\DataFixtures;
 
 use Doctrine\Bundle\FixturesBundle\Fixture;
 use Doctrine\Common\Persistence\ObjectManager;
+use Faker\Factory;
+use Faker\Generator;
 
 abstract class BaseFixture extends Fixture
 {
     /** @var ObjectManager */
     private $manager;
 
+    /** @var Generator */
+    protected $faker;
+
     abstract protected function loadData(ObjectManager $manager);
 
     public function load(ObjectManager $manager)
     {
         $this->manager = $manager;
+        $this->faker = Factory::create();
 
         $this->loadData($manager);
     }
