diff --git a/src/DataFixtures/ArticleFixtures.php b/src/DataFixtures/ArticleFixtures.php
index e71a1054..a001b2a2 100644
--- a/src/DataFixtures/ArticleFixtures.php
+++ b/src/DataFixtures/ArticleFixtures.php
@@ -9,10 +9,10 @@ class ArticleFixtures extends BaseFixture
 {
     public function loadData(ObjectManager $manager)
     {
-        $article = new Article();
-        $article->setTitle('Why Asteroids Taste Like Bacon')
-            ->setSlug('why-asteroids-taste-like-bacon-'.rand(100, 999))
-            ->setContent(<<<EOF
+        $this->createMany(Article::class, 10, function(Article $article, $count) {
+            $article->setTitle('Why Asteroids Taste Like Bacon')
+                ->setSlug('why-asteroids-taste-like-bacon-'.$count)
+                ->setContent(<<<EOF
 Spicy **jalapeno bacon** ipsum dolor amet veniam shank in dolore. Ham hock nisi landjaeger cow,
 lorem proident [beef ribs](https://baconipsum.com/) aute enim veniam ut cillum pork chuck picanha. Dolore reprehenderit
 labore minim pork belly spare ribs cupim short loin in. Elit exercitation eiusmod dolore cow
@@ -30,19 +30,18 @@ strip steak pork belly aliquip capicola officia. Labore deserunt esse chicken lo
 cow est ribeye adipisicing. Pig hamburger pork belly enim. Do porchetta minim capicola irure pancetta chuck
 fugiat.
 EOF
-        );
-
-        // publish most articles
-        if (rand(1, 10) > 2) {
-            $article->setPublishedAt(new \DateTime(sprintf('-%d days', rand(1, 100))));
-        }
-
-        $article->setAuthor('Mike Ferengi')
-            ->setHeartCount(rand(5, 100))
-            ->setImageFilename('asteroid.jpeg')
-        ;
-
-        $manager->persist($article);
+            );
+
+            // publish most articles
+            if (rand(1, 10) > 2) {
+                $article->setPublishedAt(new \DateTime(sprintf('-%d days', rand(1, 100))));
+            }
+
+            $article->setAuthor('Mike Ferengi')
+                ->setHeartCount(rand(5, 100))
+                ->setImageFilename('asteroid.jpeg')
+            ;
+        });
 
         $manager->flush();
     }
diff --git a/src/DataFixtures/BaseFixture.php b/src/DataFixtures/BaseFixture.php
index c3216f59..31b93e23 100644
--- a/src/DataFixtures/BaseFixture.php
+++ b/src/DataFixtures/BaseFixture.php
@@ -18,4 +18,16 @@ abstract class BaseFixture extends Fixture
 
         $this->loadData($manager);
     }
+
+    protected function createMany(string $className, int $count, callable $factory)
+    {
+        for ($i = 0; $i < $count; $i++) {
+            $entity = new $className();
+            $factory($entity, $i);
+
+            $this->manager->persist($entity);
+            // store for usage later as App\Entity\ClassName_#COUNT#
+            $this->addReference($className . '_' . $i, $entity);
+        }
+    }
 }
