diff --git a/templates/article/homepage.html.twig b/templates/article/homepage.html.twig
index e65f35fc..f5f7d42b 100644
--- a/templates/article/homepage.html.twig
+++ b/templates/article/homepage.html.twig
@@ -21,7 +21,7 @@
                 {% for article in articles %}
                 <div class="article-container my-1">
                     <a href="{{ path('article_show', {slug: article.slug}) }}">
-                        <img class="article-img" src="{{ uploaded_asset(article.imagePath)|imagine_filter('squared_thumbnail_small') }}">
+                        <img class="article-img" src="{{ article.imagePath|imagine_filter('squared_thumbnail_small') }}">
                         <div class="article-title d-inline-block pl-3 align-middle">
                             <span>{{ article.title }}</span>
                             <small>({{ article.nonDeletedComments|length }} comments)</small>
diff --git a/templates/article/show.html.twig b/templates/article/show.html.twig
index 5615115f..6bc658bd 100644
--- a/templates/article/show.html.twig
+++ b/templates/article/show.html.twig
@@ -5,7 +5,7 @@
 {% block content_body %}
     <div class="row">
         <div class="col-sm-12">
-            <img class="show-article-img" src="{{ uploaded_asset(article.imagePath)|imagine_filter('squared_thumbnail_medium') }}">
+            <img class="show-article-img" src="{{ article.imagePath|imagine_filter('squared_thumbnail_medium') }}">
             <div class="show-article-title-container d-inline-block pl-3 align-middle">
                 <span class="show-article-title ">{{ article.title }}</span>
                 <br>
diff --git a/templates/article_admin/_form.html.twig b/templates/article_admin/_form.html.twig
index 2772198b..94dbc097 100644
--- a/templates/article_admin/_form.html.twig
+++ b/templates/article_admin/_form.html.twig
@@ -13,7 +13,7 @@
         </div>
         <div class="col-sm-3">
             {% if articleForm.vars.data.imageFilename %}
-                <img src="{{ uploaded_asset(articleForm.vars.data.imagePath) | imagine_filter('squared_thumbnail_small') }}" height="100">
+                <img src="{{ articleForm.vars.data.imagePath | imagine_filter('squared_thumbnail_small') }}" height="100">
             {% endif %}
         </div>
     </div>
