diff --git a/src/Service/UploaderHelper.php b/src/Service/UploaderHelper.php
index a62d1fde3..d2954d47a 100644
--- a/src/Service/UploaderHelper.php
+++ b/src/Service/UploaderHelper.php
@@ -3,7 +3,9 @@
 namespace App\Service;
 
 use Gedmo\Sluggable\Util\Urlizer;
+use League\Flysystem\FileNotFoundException;
 use League\Flysystem\FilesystemInterface;
+use Psr\Log\LoggerInterface;
 use Symfony\Component\HttpFoundation\File\File;
 use Symfony\Component\Asset\Context\RequestStackContext;
 use Symfony\Component\HttpFoundation\File\UploadedFile;
@@ -16,10 +18,13 @@ class UploaderHelper
 
     private $requestStackContext;
 
-    public function __construct(FilesystemInterface $publicUploadsFilesystem, RequestStackContext $requestStackContext)
+    private $logger;
+
+    public function __construct(FilesystemInterface $publicUploadsFilesystem, RequestStackContext $requestStackContext, LoggerInterface $logger)
     {
         $this->filesystem = $publicUploadsFilesystem;
         $this->requestStackContext = $requestStackContext;
+        $this->logger = $logger;
     }
 
     public function uploadArticleImage(File $file, ?string $existingFilename): string
@@ -41,7 +46,11 @@ class UploaderHelper
         }
 
         if ($existingFilename) {
-            $this->filesystem->delete(self::ARTICLE_IMAGE.'/'.$existingFilename);
+            try {
+                $this->filesystem->delete(self::ARTICLE_IMAGE.'/'.$existingFilename);
+            } catch (FileNotFoundException $e) {
+                $this->logger->alert(sprintf('Old uploaded file "%s" was missing when trying to delete', $existingFilename));
+            }
         }
 
         return $newFilename;
