diff --git a/src/Service/UploaderHelper.php b/src/Service/UploaderHelper.php
index 0a45463e..abefd20b 100644
--- a/src/Service/UploaderHelper.php
+++ b/src/Service/UploaderHelper.php
@@ -31,10 +31,14 @@ class UploaderHelper
         }
         $newFilename = Urlizer::urlize(pathinfo($originalFilename, PATHINFO_FILENAME)).'-'.uniqid().'.'.$file->guessExtension();
 
-        $this->filesystem->write(
+        $stream = fopen($file->getPathname(), 'r');
+        $this->filesystem->writeStream(
             self::ARTICLE_IMAGE . '/' . $newFilename,
-            file_get_contents($file->getPathname())
+            $stream
         );
+        if (is_resource($stream)) {
+            fclose($stream);
+        }
 
         return $newFilename;
     }
