diff --git a/src/Service/UploaderHelper.php b/src/Service/UploaderHelper.php
index d2954d4..33cc20c 100644
--- a/src/Service/UploaderHelper.php
+++ b/src/Service/UploaderHelper.php
@@ -37,17 +37,26 @@ class UploaderHelper
         $newFilename = Urlizer::urlize(pathinfo($originalFilename, PATHINFO_FILENAME)).'-'.uniqid().'.'.$file->guessExtension();
 
         $stream = fopen($file->getPathname(), 'r');
-        $this->filesystem->writeStream(
+        $result = $this->filesystem->writeStream(
             self::ARTICLE_IMAGE.'/'.$newFilename,
             $stream
         );
+
+        if ($result === false) {
+            throw new \Exception(sprintf('Could not write uploaded file "%s"', $newFilename));
+        }
+
         if (is_resource($stream)) {
             fclose($stream);
         }
 
         if ($existingFilename) {
             try {
-                $this->filesystem->delete(self::ARTICLE_IMAGE.'/'.$existingFilename);
+                $result = $this->filesystem->delete(self::ARTICLE_IMAGE.'/'.$existingFilename);
+
+                if ($result === false) {
+                    throw new \Exception(sprintf('Could not delete old uploaded file "%s"', $existingFilename));
+                }
             } catch (FileNotFoundException $e) {
                 $this->logger->alert(sprintf('Old uploaded file "%s" was missing when trying to delete', $existingFilename));
             }
