diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 7e8f7cf6..5fea0f82 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -34,7 +34,7 @@ class ArticleAdminController extends BaseController
             $uploadedFile = $form['imageFile']->getData();
 
             if ($uploadedFile) {
-                $newFilename = $uploaderHelper->uploadArticleImage($uploadedFile);
+                $newFilename = $uploaderHelper->uploadArticleImage($uploadedFile, $article->getImageFilename());
                 $article->setImageFilename($newFilename);
             }
 
@@ -66,7 +66,7 @@ class ArticleAdminController extends BaseController
             /** @var UploadedFile $uploadedFile */
             $uploadedFile = $form['imageFile']->getData();
             if ($uploadedFile) {
-                $newFilename = $uploaderHelper->uploadArticleImage($uploadedFile);
+                $newFilename = $uploaderHelper->uploadArticleImage($uploadedFile, $article->getImageFilename());
                 $article->setImageFilename($newFilename);
             }
 
diff --git a/src/DataFixtures/ArticleFixtures.php b/src/DataFixtures/ArticleFixtures.php
index e34b11cf..d298ef4d 100644
--- a/src/DataFixtures/ArticleFixtures.php
+++ b/src/DataFixtures/ArticleFixtures.php
@@ -96,6 +96,6 @@ EOF
         $fs->copy(__DIR__.'/images/'.$randomImage, $targetPath, true);
 
         return $this->uploaderHelper
-            ->uploadArticleImage(new File($targetPath));
+            ->uploadArticleImage(new File($targetPath), null);
     }
 }
diff --git a/src/Service/UploaderHelper.php b/src/Service/UploaderHelper.php
index fdeff25c..915c7567 100644
--- a/src/Service/UploaderHelper.php
+++ b/src/Service/UploaderHelper.php
@@ -22,7 +22,7 @@ class UploaderHelper
         $this->requestStackContext = $requestStackContext;
     }
 
-    public function uploadArticleImage(File $file): string
+    public function uploadArticleImage(File $file, ?string $existingFilename): string
     {
         if ($file instanceof UploadedFile) {
             $originalFilename = $file->getClientOriginalName();
@@ -40,6 +40,10 @@ class UploaderHelper
             fclose($stream);
         }
 
+        if ($existingFilename) {
+            $this->filesystem->delete(self::ARTICLE_IMAGE . '/' . $existingFilename);
+        }
+
         return $newFilename;
     }
 
