diff --git a/config/services.yaml b/config/services.yaml
index 6f7700c6f..578d7df11 100644
--- a/config/services.yaml
+++ b/config/services.yaml
@@ -20,7 +20,7 @@ services:
         bind:
             $markdownLogger: '@monolog.logger.markdown'
             $isDebug: '%kernel.debug%'
-            $uploadsPath: '%kernel.project_dir%/public/uploads'
+            $publicUploadsFilesystem: '@oneup_flysystem.public_uploads_filesystem_filesystem'
 
     # custom aliases for autowiring
     Nexy\Slack\Client: '@nexy_slack.client'
diff --git a/src/Service/UploaderHelper.php b/src/Service/UploaderHelper.php
index d9e7962a1..6331c5498 100644
--- a/src/Service/UploaderHelper.php
+++ b/src/Service/UploaderHelper.php
@@ -16,9 +16,9 @@ class UploaderHelper
 
     private $requestStackContext;
 
-    public function __construct(FilesystemInterface $filesystem, RequestStackContext $requestStackContext)
+    public function __construct(FilesystemInterface $publicUploadsFilesystem, RequestStackContext $requestStackContext)
     {
-        $this->filesystem = $filesystem;
+        $this->filesystem = $publicUploadsFilesystem;
         $this->requestStackContext = $requestStackContext;
     }
 
