diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 48d7d0ab..e099090b 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -56,16 +56,19 @@ class ArticleAdminController extends BaseController
         if ($form->isSubmitted() && $form->isValid()) {
             /** @var UploadedFile $uploadedFile */
             $uploadedFile = $form['imageFile']->getData();
-            $destination = $this->getParameter('kernel.project_dir').'/public/uploads/article_images';
 
-            $originalFilename = pathinfo($uploadedFile->getClientOriginalName(), PATHINFO_FILENAME);
-            $newFilename = Urlizer::urlize($originalFilename) . '-' . uniqid('', true) . '.' . $uploadedFile->guessExtension();
+            if ($uploadedFile) {
+                $destination = $this->getParameter('kernel.project_dir') . '/public/uploads/article_images';
 
-            $uploadedFile->move(
-                $destination,
-                $newFilename
-            );
-            $article->setImageFilename($newFilename);
+                $originalFilename = pathinfo($uploadedFile->getClientOriginalName(), PATHINFO_FILENAME);
+                $newFilename = Urlizer::urlize($originalFilename) . '-' . uniqid('', true) . '.' . $uploadedFile->guessExtension();
+
+                $uploadedFile->move(
+                    $destination,
+                    $newFilename
+                );
+                $article->setImageFilename($newFilename);
+            }
 
             $em->persist($article);
             $em->flush();
diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index 2297b7e0..a9335388 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -52,7 +52,8 @@ class ArticleFormType extends AbstractType
                 'required' => false,
             ])
             ->add('imageFile', FileType::class, [
-                'mapped' => false
+                'mapped' => false,
+                'required' => false,
             ])
         ;
 
