diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index d9b56fbb..d87cfd16 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -27,7 +27,9 @@ class ArticleFormType extends AbstractType
             ->add('title', TextType::class, [
                 'help' => 'Choose something catchy!'
             ])
-            ->add('content')
+            ->add('content', null, [
+                'rows' => 15
+            ])
             ->add('publishedAt', null, [
                 'widget' => 'single_text'
             ])
