diff --git a/src/Form/TypeExtension/TextareaSizeExtension.php b/src/Form/TypeExtension/TextareaSizeExtension.php
new file mode 100644
index 00000000..e3a89f4b
--- /dev/null
+++ b/src/Form/TypeExtension/TextareaSizeExtension.php
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Form\TypeExtension;
+
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormTypeExtensionInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class TextareaSizeExtension implements FormTypeExtensionInterface
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        // TODO: Implement buildForm() method.
+    }
+
+    public function buildView(FormView $view, FormInterface $form, array $options)
+    {
+        // TODO: Implement buildView() method.
+    }
+
+    public function finishView(FormView $view, FormInterface $form, array $options)
+    {
+        // TODO: Implement finishView() method.
+    }
+
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        // TODO: Implement configureOptions() method.
+    }
+
+    public function getExtendedType()
+    {
+        // TODO: Implement getExtendedType() method.
+    }
+}
