diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index a8c62ca..f84e8bc 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -33,6 +33,7 @@ class ArticleFormType extends AbstractType
 
         $builder
             ->add('title', TextType::class, [
+                'label' => 'form.article.title',
                 'help' => 'Choose something catchy!'
             ])
             ->add('content', null, [
diff --git a/templates/article_admin/_form.html.twig b/templates/article_admin/_form.html.twig
index 781e079..45aadd9 100644
--- a/templates/article_admin/_form.html.twig
+++ b/templates/article_admin/_form.html.twig
@@ -1,7 +1,5 @@
 {{ form_start(articleForm) }}
-    {{ form_row(articleForm.title, {
-        label: 'form.article.title'
-    }) }}
+    {{ form_row(articleForm.title) }}
     {{ form_row(articleForm.author) }}
     {{ form_row(articleForm.location, {
         attr: {
