diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 6bde616..a1378ea 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -54,6 +54,8 @@ class ArticleAdminController extends BaseController
 
         $form->handleRequest($request);
         if ($form->isSubmitted() && $form->isValid()) {
+            dd($form['imageFile']->getData());
+
             $em->persist($article);
             $em->flush();
 
diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index 4b969cf..2297b7e 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -51,7 +51,9 @@ class ArticleFormType extends AbstractType
                 ],
                 'required' => false,
             ])
-            ->add('imageFilename', FileType::class)
+            ->add('imageFile', FileType::class, [
+                'mapped' => false
+            ])
         ;
 
         if ($options['include_published_at']) {
diff --git a/templates/article_admin/_form.html.twig b/templates/article_admin/_form.html.twig
index d38d576..397f259 100644
--- a/templates/article_admin/_form.html.twig
+++ b/templates/article_admin/_form.html.twig
@@ -2,6 +2,7 @@
     {{ form_row(articleForm.title, {
         label: 'Article title'
     }) }}
+    {{ form_row(articleForm.imageFile) }}
     {{ form_row(articleForm.author) }}
     {{ form_row(articleForm.location, {
         attr: {
diff --git a/templates/article_admin/edit.html.twig b/templates/article_admin/edit.html.twig
index a55c9a3..342703c 100644
--- a/templates/article_admin/edit.html.twig
+++ b/templates/article_admin/edit.html.twig
@@ -3,14 +3,6 @@
 {% block content_body %}
     <h1>Edit the Article! 🚀</h1>
 
-    <form method="POST" action="{{ path('upload_test') }}" enctype="multipart/form-data">
-        <input type="file" name="image">
-
-        <button type="submit" class="btn btn-primary">Upload!</button>
-    </form>
-
-    <hr>
-
     {{ include('article_admin/_form.html.twig', {
         button_text: 'Update!'
     }) }}
