diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index 19d91620..2c930097 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -24,8 +24,10 @@ class ArticleFormType extends AbstractType
 
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        /** @var Article|null $article */
         $article = $options['data'] ?? null;
         $isEdit = $article && $article->getId();
+        $location = $article ? $article->getLocation() : null;
 
         $builder
             ->add('title', TextType::class, [
@@ -46,14 +48,15 @@ class ArticleFormType extends AbstractType
                 ],
                 'required' => false,
             ])
-            ->add('specificLocationName', ChoiceType::class, [
+        ;
+
+        if ($location) {
+            $builder->add('specificLocationName', ChoiceType::class, [
                 'placeholder' => 'Where exactly?',
-                'choices' => [
-                    'TODO' => 'TODO'
-                ],
+                'choices' => $this->getLocationNameChoices($location),
                 'required' => false,
-            ])
-        ;
+            ]);
+        }
 
         if ($options['include_published_at']) {
             $builder->add('publishedAt', null, [
@@ -69,4 +72,36 @@ class ArticleFormType extends AbstractType
             'include_published_at' => false,
         ]);
     }
+
+    private function getLocationNameChoices(string $location)
+    {
+        $planets = [
+            'Mercury',
+            'Venus',
+            'Earth',
+            'Mars',
+            'Jupiter',
+            'Saturn',
+            'Uranus',
+            'Neptune',
+        ];
+
+        $stars = [
+            'Polaris',
+            'Sirius',
+            'Alpha Centauari A',
+            'Alpha Centauari B',
+            'Betelgeuse',
+            'Rigel',
+            'Other'
+        ];
+
+        $locationNameChoices = [
+            'solar_system' => array_combine($planets, $planets),
+            'star' => array_combine($stars, $stars),
+            'interstellar_space' => null,
+        ];
+
+        return $locationNameChoices[$location];
+    }
 }
diff --git a/templates/article_admin/_form.html.twig b/templates/article_admin/_form.html.twig
index 706c6a2a..33b85c0e 100644
--- a/templates/article_admin/_form.html.twig
+++ b/templates/article_admin/_form.html.twig
@@ -4,7 +4,9 @@
     }) }}
     {{ form_row(articleForm.author) }}
     {{ form_row(articleForm.location) }}
-    {{ form_row(articleForm.specificLocationName) }}
+    {%  if articleForm.specificLocationName is defined %}
+        {{ form_row(articleForm.specificLocationName) }}
+    {% endif %}
     {{ form_row(articleForm.content) }}
     {% if articleForm.publishedAt is defined %}
         {{ form_row(articleForm.publishedAt) }}
