diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index 56a25c8e..5fcec8fc 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -30,7 +30,6 @@ class ArticleFormType extends AbstractType
         /** @var Article|null $article */
         $article = $options['data'] ?? null;
         $isEdit = $article && $article->getId();
-        $location = $article ? $article->getLocation() : null;
 
         $builder
             ->add('title', TextType::class, [
@@ -53,20 +52,28 @@ class ArticleFormType extends AbstractType
             ])
         ;
 
-        if ($location) {
-            $builder->add('specificLocationName', ChoiceType::class, [
-                'placeholder' => 'Where exactly?',
-                'choices' => $this->getLocationNameChoices($location),
-                'required' => false,
-            ]);
-        }
-
         if ($options['include_published_at']) {
             $builder->add('publishedAt', null, [
                 'widget' => 'single_text',
             ]);
         }
 
+        $builder->addEventListener(
+            FormEvents::PRE_SET_DATA,
+            function (FormEvent $event) {
+                /** @var Article|null $data */
+                $data = $event->getData();
+                if (!$data) {
+                    return;
+                }
+
+                $this->setupSpecificLocationNameField(
+                    $event->getForm(),
+                    $data->getLocation()
+                );
+            }
+        );
+
         $builder->get('location')->addEventListener(
             FormEvents::POST_SUBMIT,
             function(FormEvent $event) {
