diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index e8f6d8c1..db3f826c 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -38,6 +38,7 @@ class ArticleFormType extends AbstractType
                 'disabled' => $isEdit
             ])
             ->add('location', ChoiceType::class, [
+                'placeholder' => 'Choose a location',
                 'choices' => [
                     'The Solar System' => 'solar_system',
                     'Near a star' => 'star',
diff --git a/templates/article_admin/_form.html.twig b/templates/article_admin/_form.html.twig
index eb9984fb..23872311 100644
--- a/templates/article_admin/_form.html.twig
+++ b/templates/article_admin/_form.html.twig
@@ -3,6 +3,7 @@
         label: 'Article title'
     }) }}
     {{ form_row(articleForm.author) }}
+    {{ form_row(articleForm.location) }}
     {{ form_row(articleForm.content) }}
     {% if articleForm.publishedAt is defined %}
         {{ form_row(articleForm.publishedAt) }}
