diff --git a/tests/SecurityControllerTest.php b/tests/SecurityControllerTest.php
new file mode 100644
index 000000000..2e43a58c1
--- /dev/null
+++ b/tests/SecurityControllerTest.php
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Tests;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class SecurityControllerTest extends WebTestCase
+{
+    public function testSomething()
+    {
+        $client = static::createClient();
+        $crawler = $client->request('GET', '/');
+
+        $this->assertResponseIsSuccessful();
+        $this->assertSelectorTextContains('h1', 'Hello World');
+    }
+}
