diff --git a/tests/Controller/SecurityControllerTest.php b/tests/Controller/SecurityControllerTest.php
index 9a6148c5..8c8fd710 100644
--- a/tests/Controller/SecurityControllerTest.php
+++ b/tests/Controller/SecurityControllerTest.php
@@ -22,5 +22,11 @@ class SecurityControllerTest extends WebTestCase
         $client->submit($form);
 
         $this->assertResponseIsSuccessful();
+
+        /* Symfony 4.4:
+        $this->assertEmailCount(1);
+        $email = $this->getMailerMessage(0);
+        $this->assertEmailHeaderSame($email, 'To', 'fabien@symfony.com');
+        */
     }
 }
