diff --git a/tests/SecurityControllerTest.php b/tests/Controller/SecurityControllerTest.php
similarity index 91%
rename from tests/SecurityControllerTest.php
rename to tests/Controller/SecurityControllerTest.php
index 2e43a58..99534f7 100644
--- a/tests/SecurityControllerTest.php
+++ b/tests/Controller/SecurityControllerTest.php
@@ -1,6 +1,6 @@
 <?php
 
-namespace App\Tests;
+namespace App\Tests\Controller;
 
 use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
 
