diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index 3b308989..dd21420f 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -94,8 +94,22 @@ class ArticleReferenceAdminController extends BaseController
      * @Route("/admin/article/{id}/references/reorder", methods="POST", name="admin_article_reorder_references")
      * @IsGranted("MANAGE", subject="article")
      */
-    public function reorderArticleReferences(Article $article)
+    public function reorderArticleReferences(Article $article, Request $request, EntityManagerInterface $entityManager)
     {
+        $orderedIds = json_decode($request->getContent(), true);
+
+        if ($orderedIds === null) {
+            return $this->json(['message' => 'Invalid body'], 400);
+        }
+
+        // from (position)=>(id) to (id)=>(position)
+        $orderedIds = array_flip($orderedIds);
+        foreach ($article->getArticleReferences() as $reference) {
+            $reference->setPosition($orderedIds[$reference->getId()]);
+        }
+
+        $entityManager->flush();
+
         return $this->json(
             $article->getArticleReferences(),
             200,
