diff --git a/public/js/admin_article_form.js b/public/js/admin_article_form.js
index c7b9d34..882f026 100644
--- a/public/js/admin_article_form.js
+++ b/public/js/admin_article_form.js
@@ -39,6 +39,9 @@ class ReferenceList
         this.sortable = Sortable.create(this.$element[0], {
             handle: '.drag-handle',
             animation: 150,
+            onEnd: () => {
+                console.log(this.sortable.toArray());
+            }
         });
         this.references = [];
         this.render();
