diff --git a/public/js/admin_article_form.js b/public/js/admin_article_form.js
index 3110ed48..6efd6428 100644
--- a/public/js/admin_article_form.js
+++ b/public/js/admin_article_form.js
@@ -43,6 +43,10 @@ class ReferenceList
             this.handleReferenceDelete(event);
         });
 
+        this.$element.on('blur', '.js-edit-filename', (event) => {
+            this.handleReferenceEditFilename(event);
+        });
+
         $.ajax({
             url: this.$element.data('url')
         }).then(data => {
@@ -72,11 +76,26 @@ class ReferenceList
         });
     }
 
+    handleReferenceEditFilename(event) {
+        const $li = $(event.currentTarget).closest('.list-group-item');
+        const id = $li.data('id');
+        const reference = this.references.find(reference => {
+            return reference.id === id;
+        });
+        reference.originalFilename = $(event.currentTarget).val();
+
+        $.ajax({
+            url: '/admin/article/references/'+id,
+            method: 'PUT',
+            data: reference
+        });
+    }
+
     render() {
         const itemsHtml = this.references.map(reference => {
             return `
 <li class="list-group-item d-flex justify-content-between align-items-center" data-id="${reference.id}">
-    ${reference.originalFilename}
+    <input type="text" value="${reference.originalFilename}" class="form-control js-edit-filename" style="width: auto;">
 
     <span>
         <a href="/admin/article/references/${reference.id}/download" class="btn btn-link btn-sm"><span class="fa fa-download" style="vertical-align: middle"></span></a>
