diff --git a/public/js/admin_article_form.js b/public/js/admin_article_form.js
index 6efd6428..24978c5b 100644
--- a/public/js/admin_article_form.js
+++ b/public/js/admin_article_form.js
@@ -87,7 +87,7 @@ class ReferenceList
         $.ajax({
             url: '/admin/article/references/'+id,
             method: 'PUT',
-            data: reference
+            data: JSON.stringify(reference)
         });
     }
 
