diff --git a/assets/js/app.js b/assets/js/app.js
index eecd0c14..e8f5cf69 100644
--- a/assets/js/app.js
+++ b/assets/js/app.js
@@ -11,6 +11,6 @@ require('../css/app.css');
 // Need jQuery? Install it with "yarn add jquery", then uncomment to require it.
 // const $ = require('jquery');
 
-const getNiceMessage = require('./get_nice_message');
+import getNiceMessage from './get_nice_message';
 
 console.log(getNiceMessage(5));
diff --git a/assets/js/get_nice_message.js b/assets/js/get_nice_message.js
index 367348af..e215acd7 100644
--- a/assets/js/get_nice_message.js
+++ b/assets/js/get_nice_message.js
@@ -1,3 +1,3 @@
-module.exports = function(exclamationCount) {
+export default function(exclamationCount) {
     return 'Hello Webpack Encore! Edit me in assets/js/app.js'+'!'.repeat(exclamationCount);
 };
