diff --git a/assets/js/app.js b/assets/js/app.js
index 0648e871..2cb1da50 100644
--- a/assets/js/app.js
+++ b/assets/js/app.js
@@ -11,4 +11,4 @@ require('../css/app.css');
 // Need jQuery? Install it with "yarn add jquery", then uncomment to require it.
 // const $ = require('jquery');
 
-console.log('Hello Webpack Encore! Edit me in assets/js/app.js');
+console.log('Hello Webpack Encore! Edit me in assets/js/app.js!!!');
