diff --git a/tests/Service/MailerTest.php b/tests/Service/MailerTest.php
index d4b7a7488..4657f2a7c 100644
--- a/tests/Service/MailerTest.php
+++ b/tests/Service/MailerTest.php
@@ -7,12 +7,13 @@ use App\Entity\User;
 use App\Service\Mailer;
 use Knp\Snappy\Pdf;
 use PHPUnit\Framework\TestCase;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
 use Symfony\Component\Mailer\MailerInterface;
 use Symfony\Component\Mime\NamedAddress;
 use Symfony\WebpackEncoreBundle\Asset\EntrypointLookupInterface;
 use Twig\Environment;
 
-class MailerTest extends TestCase
+class MailerTest extends KernelTestCase
 {
     public function testSendWelcomeMessage()
     {
@@ -42,12 +43,12 @@ class MailerTest extends TestCase
 
     public function testIntegrationSendAuthorWeeklyReportMessage()
     {
+        self::bootKernel();
         $symfonyMailer = $this->createMock(MailerInterface::class);
         $symfonyMailer->expects($this->once())
             ->method('send');
-
-        $pdf = $this->createMock(Pdf::class);
-        $twig = $this->createMock(Environment::class);
+        $pdf = self::$container->get(Pdf::class);
+        $twig = self::$container->get(Environment::class);
         $entrypointLookup = $this->createMock(EntrypointLookupInterface::class);
 
         $user = new User();
