diff --git a/tests/Service/MailerTest.php b/tests/Service/MailerTest.php
index 3fa67b27a..d4b7a7488 100644
--- a/tests/Service/MailerTest.php
+++ b/tests/Service/MailerTest.php
@@ -53,8 +53,10 @@ class MailerTest extends TestCase
         $user = new User();
         $user->setFirstName('Victor');
         $user->setEmail('victor@symfonycasts.com');
+        $article = new Article();
+        $article->setTitle('Black Holes: Ultimate Party Pooper');
 
         $mailer = new Mailer($symfonyMailer, $twig, $pdf, $entrypointLookup);
-        $email = $mailer->sendWelcomeMessage($user);
+        $email = $mailer->sendAuthorWeeklyReportMessage($user, [$article]);
     }
 }
