diff --git a/templates/base.html.twig b/templates/base.html.twig
index 284d53b6..faa497c4 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -39,9 +39,9 @@
                               <img class="nav-profile-img rounded-circle" src="{{ asset('images/astronaut-profile.png') }}">
                             </a>
                             <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
-                                <a class="dropdown-item" href="#">Profile</a>
+                                <a class="dropdown-item" href="{{ path('app_account') }}">Profile</a>
                                 <a class="dropdown-item" href="#">Create Post</a>
-                                <a class="dropdown-item" href="#">Logout</a>
+                                <a class="dropdown-item" href="{{ path('app_logout') }}">Logout</a>
                             </div>
                         </li>
                     {% else %}
