diff --git a/templates/base.html.twig b/templates/base.html.twig
index f76b5e2e..284d53b6 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -33,21 +33,22 @@
                     <button class="btn btn-info my-2 my-sm-0" type="submit">Search</button>
                 </form>
                 <ul class="navbar-nav ml-auto">
-                    <li class="nav-item">
-                        <a style="color: #fff;" class="nav-link" href="{{ path('app_login') }}">Login</a>
-                    </li>
-                    {#
-                    <li class="nav-item dropdown" style="margin-right: 75px;">
-                        <a class="nav-link dropdown-toggle" href="http://example.com" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                          <img class="nav-profile-img rounded-circle" src="{{ asset('images/astronaut-profile.png') }}">
-                        </a>
-                        <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
-                            <a class="dropdown-item" href="#">Profile</a>
-                            <a class="dropdown-item" href="#">Create Post</a>
-                            <a class="dropdown-item" href="#">Logout</a>
-                        </div>
-                    </li>
-                    #}
+                    {% if is_granted('ROLE_USER') %}
+                        <li class="nav-item dropdown" style="margin-right: 75px;">
+                            <a class="nav-link dropdown-toggle" href="http://example.com" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                              <img class="nav-profile-img rounded-circle" src="{{ asset('images/astronaut-profile.png') }}">
+                            </a>
+                            <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                                <a class="dropdown-item" href="#">Profile</a>
+                                <a class="dropdown-item" href="#">Create Post</a>
+                                <a class="dropdown-item" href="#">Logout</a>
+                            </div>
+                        </li>
+                    {% else %}
+                        <li class="nav-item">
+                            <a style="color: #fff;" class="nav-link" href="{{ path('app_login') }}">Login</a>
+                        </li>
+                    {% endif %}
                 </ul>
             </div>
         </nav>
