diff --git a/public/js/article_show.js b/public/js/article_show.js
index 20ebf90a..ee86b6fa 100644
--- a/public/js/article_show.js
+++ b/public/js/article_show.js
@@ -5,6 +5,11 @@ $(document).ready(function() {
         var $link = $(e.currentTarget);
         $link.toggleClass('fa-heart-o').toggleClass('fa-heart');
 
-        $('.js-like-article-count').html('TEST');
+        $.ajax({
+            method: 'POST',
+            url: $link.attr('href')
+        }).done(function(data) {
+            $('.js-like-article-count').html(data.hearts);
+        })
     });
 });
diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index 5d36fcae..4b6f99e5 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -30,6 +30,7 @@ class ArticleController extends AbstractController
 
         return $this->render('article/show.html.twig', [
             'title' => ucwords(str_replace('-', ' ', $slug)),
+            'slug' => $slug,
             'comments' => $comments,
         ]);
     }
diff --git a/templates/article/show.html.twig b/templates/article/show.html.twig
index b1c24d24..346fe33c 100644
--- a/templates/article/show.html.twig
+++ b/templates/article/show.html.twig
@@ -18,7 +18,7 @@
                             <span class="pl-2 article-details"> 3 hours ago</span>
                             <span class="pl-2 article-details">
                                 <span class="js-like-article-count">5</span>
-                                <a href="#" class="fa fa-heart-o like-article js-like-article"></a>
+                                <a href="{{ path('article_toggle_heart', {slug: slug}) }}" class="fa fa-heart-o like-article js-like-article"></a>
                             </span>
                         </div>
                     </div>
