diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index 5fc6a95d..5d36fcae 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -35,7 +35,7 @@ class ArticleController extends AbstractController
     }
 
     /**
-     * @Route("/news/{slug}/heart", name="article_toggle_heart")
+     * @Route("/news/{slug}/heart", name="article_toggle_heart", methods={"POST"})
      */
     public function toggleArticleHeart($slug)
     {
