diff --git a/templates/comment_admin/index.html.twig b/templates/comment_admin/index.html.twig
index d0a6334d..7c8b7a60 100644
--- a/templates/comment_admin/index.html.twig
+++ b/templates/comment_admin/index.html.twig
@@ -9,6 +9,22 @@
         <div class="col-sm-12">
             <h1>Manage Comments</h1>
 
+            <form>
+                <div class="input-group mb-3">
+                    <input type="text"
+                           name="q"
+                           class="form-control"
+                           placeholder="Search..."
+                    >
+                    <div class="input-group-append">
+                        <button type="submit"
+                                class="btn btn-outline-secondary">
+                            <span class="fa fa-search"></span>
+                        </button>
+                    </div>
+                </div>
+            </form>
+
             <table class="table table-striped">
                 <thead>
                 <tr>
