diff --git a/src/Repository/CommentRepository.php b/src/Repository/CommentRepository.php
index 82bc2170..3e1bebf0 100644
--- a/src/Repository/CommentRepository.php
+++ b/src/Repository/CommentRepository.php
@@ -38,7 +38,7 @@ class CommentRepository extends ServiceEntityRepository
             ->innerJoin('c.article', 'a');
 
         if ($term) {
-            $qb->andWhere('c.content LIKE :term OR c.authorName LIKE :term')
+            $qb->andWhere('c.content LIKE :term OR c.authorName LIKE :term OR a.title LIKE :term')
                 ->setParameter('term', '%' . $term . '%')
             ;
         }
