diff --git a/src/Repository/CommentRepository.php b/src/Repository/CommentRepository.php
index a5b274e8..2c443341 100644
--- a/src/Repository/CommentRepository.php
+++ b/src/Repository/CommentRepository.php
@@ -29,7 +29,7 @@ class CommentRepository extends ServiceEntityRepository
             ->innerJoin('c.article', 'a');
 
         if ($term) {
-            $qb->andWhere('c.content LIKE :term OR c.authorName LIKE :term')
+            $qb->andWhere('c.content LIKE :term OR c.authorName LIKE :term OR a.title LIKE :term')
                 ->setParameter('term', '%' . $term . '%')
             ;
         }
