diff --git a/.gitignore b/.gitignore
index e9a59860..206e71ea 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,5 @@
+/public/translations
+
 ###> symfony/framework-bundle ###
 /.env.local
 /.env.local.php
diff --git a/config/packages/bazinga_js_translation.yaml b/config/packages/bazinga_js_translation.yaml
index 2f7f2ce1..63bcae70 100644
--- a/config/packages/bazinga_js_translation.yaml
+++ b/config/packages/bazinga_js_translation.yaml
@@ -1,2 +1,5 @@
 bazinga_js_translation:
     locale_fallback: '%locale%'
+    active_locales: '%app_locales_array%'
+    active_domains:
+        - javascript
diff --git a/templates/article/homepage.html.twig b/templates/article/homepage.html.twig
index 8a7bd644..96ab8eb6 100644
--- a/templates/article/homepage.html.twig
+++ b/templates/article/homepage.html.twig
@@ -71,9 +71,16 @@
 
     <script src="https://cdn.jsdelivr.net/npm/sweetalert2@8"></script>
     <script src="{{ asset('bundles/bazingajstranslation/js/translator.min.js') }}"></script>
-    <script src="{{ url('bazinga_jstranslation_js', {
-        domain: 'javascript',
-        locales: app_locales_array|join(',')
-    }) }}"></script>
+    {% if app.environment == 'prod' %}
+        <script src="{{ asset('translations/config.js') }}"></script>
+        {% for locale in app_locales_array %}
+            <script src="{{ asset('translations/javascript/'~locale~'.js') }}"></script>
+        {% endfor %}
+    {% else %}
+        <script src="{{ url('bazinga_jstranslation_js', {
+            domain: 'javascript',
+            locales: app_locales_array|join(',')
+        }) }}"></script>
+    {% endif %}
     <script src="{{ asset('js/purchase.js') }}"></script>
 {% endblock %}
