diff --git a/public/js/admin_article_form.js b/public/js/admin_article_form.js
index a589c8ce..1c61c5fa 100644
--- a/public/js/admin_article_form.js
+++ b/public/js/admin_article_form.js
@@ -1,6 +1,8 @@
 Dropzone.autoDiscover = false;
 
 $(document).ready(function() {
+    var referenceList = new ReferenceList($('.js-reference-list'));
+
     initializeDropzone();
 
     var $locationSelect = $('.js-article-form-location');
@@ -29,6 +31,39 @@ $(document).ready(function() {
     });
 });
 
+// todo - use Webpack Encore so ES6 syntax is transpiled to ES5
+class ReferenceList
+{
+    constructor($element) {
+        this.$element = $element;
+        this.references = [];
+        this.render();
+
+        $.ajax({
+            url: this.$element.data('url')
+        }).then(data => {
+            this.references = data;
+            this.render();
+        })
+    }
+
+    render() {
+        const itemsHtml = this.references.map(reference => {
+            return `
+<li class="list-group-item d-flex justify-content-between align-items-center">
+    ${reference.originalFilename}
+
+    <span>
+        <a href="/admin/article/references/${reference.id}/download"><span class="fa fa-download"></span></a>
+    </span>
+</li>
+`
+        });
+
+        this.$element.html(itemsHtml.join(''));
+    }
+}
+
 function initializeDropzone() {
     var formElement = document.querySelector('.js-reference-dropzone');
     if (!formElement) {
diff --git a/templates/article_admin/edit.html.twig b/templates/article_admin/edit.html.twig
index a3de83b4..b693a08a 100644
--- a/templates/article_admin/edit.html.twig
+++ b/templates/article_admin/edit.html.twig
@@ -15,19 +15,7 @@
         <div class="col-sm-4">
             <h3>References</h3>
 
-            <ul class="list-group small">
-                {% for reference in article.articleReferences %}
-                    <li class="list-group-item d-flex justify-content-between align-items-center">
-                        {{ reference.originalFilename }}
-
-                        <span>
-                            <a href="{{ path('admin_article_download_reference', {
-                                id: reference.id
-                            }) }}"><span class="fa fa-download"></span></a>
-                        </span>
-                    </li>
-                {% endfor %}
-            </ul>
+            <ul class="list-group small js-reference-list" data-url="{{ path('admin_article_list_references', {id: article.id}) }}"></ul>
 
             <hr>
 
