diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index afb73dd1..d97f426b 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -78,7 +78,14 @@ class ArticleReferenceAdminController extends BaseController
      */
     public function getArticleReferences(Article $article)
     {
-        return $this->json($article->getArticleReferences());
+        return $this->json(
+            $article->getArticleReferences(),
+            200,
+            [],
+            [
+                'groups' => ['main']
+            ]
+        );
     }
 
     /**
