diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 20a1af54..b6811f0d 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -157,7 +157,14 @@ class ArticleAdminController extends BaseController
      */
     public function getArticleReferences(Article $article)
     {
-        return $this->json($article->getArticleReferences());
+        return $this->json(
+            $article->getArticleReferences(),
+            200,
+            [],
+            [
+                'groups' => ['main']
+            ]
+        );
     }
 
     /**
