diff --git a/src/Controller/WeatherController.php b/src/Controller/WeatherController.php
index 29e7983..0e1effd 100644
--- a/src/Controller/WeatherController.php
+++ b/src/Controller/WeatherController.php
@@ -11,9 +11,8 @@ class WeatherController extends AbstractController
     /**
      * @Route({"en": "/weather", "es": "/clima"}, name="app_weather")
      */
-    public function index(Request $request)
+    public function index()
     {
-        dump($request->getLocale());
         return $this->render('weather/index.html.twig');
     }
 }
diff --git a/templates/weather/index.html.twig b/templates/weather/index.html.twig
index ff6af2d..ac46c32 100644
--- a/templates/weather/index.html.twig
+++ b/templates/weather/index.html.twig
@@ -3,7 +3,6 @@
 {% block title %}Weather in Space!{% endblock %}
 
 {% block content_body %}
-    {{ dump(app.request.locale) }}
     <div class="row">
         <div class="col-sm-12">
             <h1 class="text-center">{{ 'page.weather.header'|trans }}</h1>
