diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index a0563bd1..9c638912 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -17,7 +17,7 @@ class ArticleController extends AbstractController
     }
 
     /**
-     * @Route("/news/{slug}")
+     * @Route("/news/{slug}", name="article_show")
      */
     public function show($slug)
     {
diff --git a/templates/article/homepage.html.twig b/templates/article/homepage.html.twig
index 9c87cb37..5624534f 100644
--- a/templates/article/homepage.html.twig
+++ b/templates/article/homepage.html.twig
@@ -19,7 +19,7 @@
                 <!-- Supporting Articles -->
 
                 <div class="article-container my-1">
-                    <a href="#">
+                    <a href="{{ path('article_show') }}">
                         <img class="article-img" src="{{ asset('images/asteroid.jpeg') }}">
                         <div class="article-title d-inline-block pl-3 align-middle">
                             <span>Why do Asteroids Taste Like Bacon?</span>
