diff --git a/templates/article_admin/_form.html.twig b/templates/article_admin/_form.html.twig
index 45aadd94..450e2ead 100644
--- a/templates/article_admin/_form.html.twig
+++ b/templates/article_admin/_form.html.twig
@@ -1,3 +1,15 @@
+<div>
+    {% for locale in app_locales_array %}
+        <a href="{{ path(app.request.get('_route', 'app_homepage'), app.request.get('_route_params', [])|merge({_translatable_locale: locale})) }}">
+            {% if locale == app.session.get('_translatable_locale') %}
+                <strong>{{ locale|upper }}</strong>
+            {% else %}
+                {{ locale|upper }}
+            {% endif %}
+        </a>
+    {% endfor %}
+</div>
+
 {{ form_start(articleForm) }}
     {{ form_row(articleForm.title) }}
     {{ form_row(articleForm.author) }}
