diff --git a/public/css/login.css b/public/css/login.css
new file mode 100644
index 00000000..8ac8f6fe
--- /dev/null
+++ b/public/css/login.css
@@ -0,0 +1,39 @@
+body {
+    background-color: #fff;
+}
+
+.form-signin {
+    width: 100%;
+    max-width: 330px;
+    padding: 15px;
+    margin: auto;
+    margin-top: 50px;
+}
+
+.form-signin .checkbox {
+    font-weight: 400;
+}
+
+.form-signin .form-control {
+    position: relative;
+    box-sizing: border-box;
+    height: auto;
+    padding: 10px;
+    font-size: 16px;
+}
+
+.form-signin .form-control:focus {
+    z-index: 2;
+}
+
+.form-signin input[type="email"] {
+    margin-bottom: -1px;
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 0;
+}
+
+.form-signin input[type="password"] {
+    margin-bottom: 10px;
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+}
\ No newline at end of file
diff --git a/templates/security/login.html.twig b/templates/security/login.html.twig
index 9cb70745..77ecfde2 100644
--- a/templates/security/login.html.twig
+++ b/templates/security/login.html.twig
@@ -2,26 +2,30 @@
 
 {% block title %}Login!{% endblock %}
 
-{% block body %}
-    <h1>Login to the SpaceBar!</h1>
-
-    {% if error %}
-        <div>{{ error.messageKey|trans(error.messageData, 'security') }}</div>
-    {% endif %}
-
-    <form method="post">
-        <label for="username">Username:</label>
-        <input type="text" id="username" name="_username" value="{{ last_username }}" />
+{% block stylesheets %}
+    {{ parent() }}
 
-        <label for="password">Password:</label>
-        <input type="password" id="password" name="_password" />
+    <link rel="stylesheet" href="{{ asset('css/login.css') }}">
+{% endblock %}
 
-        {#
-            If you want to control the URL the user
-            is redirected to on success (more details below)
-            <input type="hidden" name="_target_path" value="/account" />
-        #}
+{% block body %}
+    <form class="form-signin" method="post">
+        {% if error %}
+            <div>{{ error.messageKey|trans(error.messageData, 'security') }}</div>
+        {% endif %}
 
-        <button type="submit">login</button>
+        <h1 class="h3 mb-3 font-weight-normal">Please sign in</h1>
+        <label for="inputEmail" class="sr-only">Email address</label>
+        <input type="email" name="email" id="inputEmail" class="form-control" placeholder="Email address" required autofocus>
+        <label for="inputPassword" class="sr-only">Password</label>
+        <input type="password" name="password" id="inputPassword" class="form-control" placeholder="Password" required>
+        <div class="checkbox mb-3">
+            <label>
+                <input type="checkbox" value="remember-me"> Remember me
+            </label>
+        </div>
+        <button class="btn btn-lg btn-primary btn-block" type="submit">
+            Sign in
+        </button>
     </form>
 {% endblock %}
