diff --git a/templates/security/login.html.twig b/templates/security/login.html.twig
index 77ecfde2..43481485 100644
--- a/templates/security/login.html.twig
+++ b/templates/security/login.html.twig
@@ -11,7 +11,7 @@
 {% block body %}
     <form class="form-signin" method="post">
         {% if error %}
-            <div>{{ error.messageKey|trans(error.messageData, 'security') }}</div>
+            <div class="alert alert-danger">{{ error.messageKey|trans(error.messageData, 'security') }}</div>
         {% endif %}
 
         <h1 class="h3 mb-3 font-weight-normal">Please sign in</h1>
