diff --git a/templates/security/login.html.twig b/templates/security/login.html.twig
index 011c6edd..9cb70745 100644
--- a/templates/security/login.html.twig
+++ b/templates/security/login.html.twig
@@ -9,7 +9,7 @@
         <div>{{ error.messageKey|trans(error.messageData, 'security') }}</div>
     {% endif %}
 
-    <form action="{{ path('login') }}" method="post">
+    <form method="post">
         <label for="username">Username:</label>
         <input type="text" id="username" name="_username" value="{{ last_username }}" />
 
