diff --git a/src/DataFixtures/TagFixture.php b/src/DataFixtures/TagFixture.php
new file mode 100644
index 00000000..3c6b0d76
--- /dev/null
+++ b/src/DataFixtures/TagFixture.php
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\DataFixtures;
+
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\Persistence\ObjectManager;
+
+class TagFixture extends Fixture
+{
+    public function load(ObjectManager $manager)
+    {
+        // $product = new Product();
+        // $manager->persist($product);
+
+        $manager->flush();
+    }
+}
