diff --git a/templates/article/show.html.twig b/templates/article/show.html.twig
index 2384cb73..f70db230 100644
--- a/templates/article/show.html.twig
+++ b/templates/article/show.html.twig
@@ -17,6 +17,11 @@
                     <span class="js-like-article-count">{{ article.heartCount }}</span>
                     <a href="{{ path('article_toggle_heart', {slug: article.slug}) }}" class="fa fa-heart-o like-article js-like-article"></a>
                 </span>
+                <span class="pl-2 article-details">
+                    {% for tag in article.tags %}
+                        <span class="badge badge-secondary">{{ tag.name }}</span>
+                    {% endfor %}
+                </span>
             </div>
         </div>
     </div>
