diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index c7df7e85..d3ba2276 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -31,7 +31,7 @@ class ArticleController extends AbstractController
     public function homepage(EntityManagerInterface $em)
     {
         $repository = $em->getRepository(Article::class);
-        $articles = $repository->findBy([], ['publishedAt' => 'DESC']);
+        $articles = $repository->findAllPublishedOrderedByNewest();
 
         return $this->render('article/homepage.html.twig', [
             'articles' => $articles,
diff --git a/src/Repository/ArticleRepository.php b/src/Repository/ArticleRepository.php
index e52228c8..143e4887 100644
--- a/src/Repository/ArticleRepository.php
+++ b/src/Repository/ArticleRepository.php
@@ -25,10 +25,8 @@ class ArticleRepository extends ServiceEntityRepository
     public function findAllPublishedOrderedByNewest()
     {
         return $this->createQueryBuilder('a')
-            ->andWhere('a.exampleField = :val')
-            ->setParameter('val', $value)
-            ->orderBy('a.id', 'ASC')
-            ->setMaxResults(10)
+            ->andWhere('a.publishedAt IS NOT NULL')
+            ->orderBy('a.publishedAt', 'DESC')
             ->getQuery()
             ->getResult()
         ;
