diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index 50cb59e3..c7df7e85 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -31,7 +31,7 @@ class ArticleController extends AbstractController
     public function homepage(EntityManagerInterface $em)
     {
         $repository = $em->getRepository(Article::class);
-        $articles = $repository->findAll();
+        $articles = $repository->findBy([], ['publishedAt' => 'DESC']);
 
         return $this->render('article/homepage.html.twig', [
             'articles' => $articles,
