diff --git a/assets/js/article_show.js b/assets/js/article_show.js
index 248b9d6..52d31d2 100644
--- a/assets/js/article_show.js
+++ b/assets/js/article_show.js
@@ -1,3 +1,4 @@
+import '../css/article_show.scss';
 import $ from 'jquery';
 
 $(document).ready(function() {
diff --git a/templates/article/show.html.twig b/templates/article/show.html.twig
index d8e2322..cef0d0b 100644
--- a/templates/article/show.html.twig
+++ b/templates/article/show.html.twig
@@ -83,3 +83,9 @@
 
     {{ encore_entry_script_tags('article_show') }}
 {% endblock %}
+
+{% block stylesheets %}
+    {{ parent() }}
+
+    {{ encore_entry_link_tags('article_show') }}
+{% endblock %}
