diff --git a/src/Repository/CommentRepository.php b/src/Repository/CommentRepository.php
index 3e1bebf0..75849817 100644
--- a/src/Repository/CommentRepository.php
+++ b/src/Repository/CommentRepository.php
@@ -35,7 +35,8 @@ class CommentRepository extends ServiceEntityRepository
     public function findAllWithSearch(?string $term)
     {
         $qb = $this->createQueryBuilder('c')
-            ->innerJoin('c.article', 'a');
+            ->innerJoin('c.article', 'a')
+            ->addSelect('a');
 
         if ($term) {
             $qb->andWhere('c.content LIKE :term OR c.authorName LIKE :term OR a.title LIKE :term')
