diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index d87cfd16..daa7d1a0 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -23,6 +23,9 @@ class ArticleFormType extends AbstractType
 
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $user = $options['data'] ?? null;
+        dd($user);
+
         $builder
             ->add('title', TextType::class, [
                 'help' => 'Choose something catchy!'
