diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index 2e2a567e..5dcf0f74 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -23,7 +23,8 @@ class ArticleFormType extends AbstractType
 
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        dd($options);
+        $article = $options['data'] ?? null;
+        dd($article);
         $builder
             ->add('title', TextType::class, [
                 'help' => 'Choose something catchy!'
