diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 7f1630a1..09036a04 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -46,7 +46,9 @@ class ArticleAdminController extends AbstractController
      */
     public function edit(Article $article, Request $request, EntityManagerInterface $em)
     {
-        $form = $this->createForm(ArticleFormType::class, $article);
+        $form = $this->createForm(ArticleFormType::class, $article, [
+            'include_published_at' => true
+        ]);
 
         $form->handleRequest($request);
         if ($form->isSubmitted() && $form->isValid()) {
diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index c3a4b745..953edf49 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -33,19 +33,23 @@ class ArticleFormType extends AbstractType
             ->add('content', null, [
                 'rows' => 15
             ])
-            ->add('publishedAt', null, [
-                'widget' => 'single_text'
-            ])
             ->add('author', UserSelectTextType::class, [
                 'disabled' => $isEdit
             ])
         ;
+
+        if ($options['include_published_at']) {
+            $builder->add('publishedAt', null, [
+                'widget' => 'single_text',
+            ]);
+        }
     }
 
     public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
-            'data_class' => Article::class
+            'data_class' => Article::class,
+            'include_published_at' => false,
         ]);
     }
 }
diff --git a/templates/article_admin/_form.html.twig b/templates/article_admin/_form.html.twig
index df490b22..eb9984fb 100644
--- a/templates/article_admin/_form.html.twig
+++ b/templates/article_admin/_form.html.twig
@@ -4,7 +4,9 @@
     }) }}
     {{ form_row(articleForm.author) }}
     {{ form_row(articleForm.content) }}
-    {{ form_row(articleForm.publishedAt) }}
+    {% if articleForm.publishedAt is defined %}
+        {{ form_row(articleForm.publishedAt) }}
+    {% endif %}
 
     <button type="submit" class="btn btn-primary">{{ button_text }}</button>
 {{ form_end(articleForm) }}
\ No newline at end of file
