diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index daa7d1a0..57a5bc05 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -24,7 +24,7 @@ class ArticleFormType extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $user = $options['data'] ?? null;
-        dd($user);
+        $isEdit = $user && $user->getId();
 
         $builder
             ->add('title', TextType::class, [
@@ -36,7 +36,9 @@ class ArticleFormType extends AbstractType
             ->add('publishedAt', null, [
                 'widget' => 'single_text'
             ])
-            ->add('author', UserSelectTextType::class)
+            ->add('author', UserSelectTextType::class, [
+                'disabled' => $isEdit
+            ])
         ;
     }
 
