diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index d87cfd16..2e2a567e 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -23,6 +23,7 @@ class ArticleFormType extends AbstractType
 
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        dd($options);
         $builder
             ->add('title', TextType::class, [
                 'help' => 'Choose something catchy!'
