diff --git a/src/Entity/Article.php b/src/Entity/Article.php
index 00862f47..f33da598 100644
--- a/src/Entity/Article.php
+++ b/src/Entity/Article.php
@@ -182,7 +182,7 @@ class Article
     {
         if (!$this->comments->contains($comment)) {
             $this->comments[] = $comment;
-            $comment->setArticle($this);
+            //$comment->setArticle($this);
         }
 
         return $this;
