diff --git a/src/DataFixtures/ArticleFixtures.php b/src/DataFixtures/ArticleFixtures.php
index 6fa5546f..8452b2ed 100644
--- a/src/DataFixtures/ArticleFixtures.php
+++ b/src/DataFixtures/ArticleFixtures.php
@@ -62,14 +62,17 @@ EOF
             $comment1 = new Comment();
             $comment1->setAuthorName('Mike Ferengi');
             $comment1->setContent('I ate a normal rock once. It did NOT taste like bacon!');
-            $comment1->setArticle($article);
+            //$comment1->setArticle($article);
             $manager->persist($comment1);
 
             $comment2 = new Comment();
             $comment2->setAuthorName('Mike Ferengi');
             $comment2->setContent('Woohoo! I\'m going on an all-asteroid diet!');
-            $comment2->setArticle($article);
+            //$comment2->setArticle($article);
             $manager->persist($comment2);
+
+            $article->addComment($comment1);
+            $article->addComment($comment2);
         });
 
         $manager->flush();
