diff --git a/config/routes.yaml b/config/routes.yaml
index c3283aa2..07824453 100644
--- a/config/routes.yaml
+++ b/config/routes.yaml
@@ -1,3 +1,3 @@
-#index:
-#    path: /
-#    controller: App\Controller\DefaultController::index
+index:
+    path: /
+    controller: App\Controller\ArticleController::homepage
diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
new file mode 100644
index 00000000..42ca0d12
--- /dev/null
+++ b/src/Controller/ArticleController.php
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Component\HttpFoundation\Response;
+
+class ArticleController
+{
+    public function homepage()
+    {
+        return new Response('OMG! My first page already! WOOO!');
+    }
+}
