diff --git a/config/routes.yaml b/config/routes.yaml
index 07824453..35ff087e 100644
--- a/config/routes.yaml
+++ b/config/routes.yaml
@@ -1,3 +1,3 @@
-index:
-    path: /
-    controller: App\Controller\ArticleController::homepage
+#index:
+#    path: /
+#    controller: App\Controller\ArticleController::homepage
diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index 42ca0d12..8d7f0632 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -2,10 +2,14 @@
 
 namespace App\Controller;
 
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Symfony\Component\HttpFoundation\Response;
 
 class ArticleController
 {
+    /**
+     * @Route("/")
+     */
     public function homepage()
     {
         return new Response('OMG! My first page already! WOOO!');
