diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index 8d7f0632..ed19173a 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -14,4 +14,12 @@ class ArticleController
     {
         return new Response('OMG! My first page already! WOOO!');
     }
+
+    /**
+     * @Route("/news/why-asteroids-taste-like-bacon")
+     */
+    public function show()
+    {
+        return new Response('Future page to show one space article!');
+    }
 }
