diff --git a/src/Controller/CommentAdminController.php b/src/Controller/CommentAdminController.php
index de6290cc..8cea2a77 100644
--- a/src/Controller/CommentAdminController.php
+++ b/src/Controller/CommentAdminController.php
@@ -3,6 +3,7 @@
 namespace App\Controller;
 
 use App\Repository\CommentRepository;
+use Knp\Component\Pager\PaginatorInterface;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
@@ -12,13 +13,20 @@ class CommentAdminController extends Controller
     /**
      * @Route("/admin/comment", name="comment_admin")
      */
-    public function index(CommentRepository $repository, Request $request)
+    public function index(CommentRepository $repository, Request $request, PaginatorInterface $paginator)
     {
         $q = $request->query->get('q');
-        $comments = $repository->findAllWithSearch($q);
+
+        $queryBuilder = $repository->getWithSearchQueryBuilder($q);
+
+        $pagination = $paginator->paginate(
+            $queryBuilder, /* query NOT result */
+            $request->query->getInt('page', 1)/*page number*/,
+            10/*limit per page*/
+        );
 
         return $this->render('comment_admin/index.html.twig', [
-            'comments' => $comments,
+            'pagination' => $pagination,
         ]);
     }
 }
diff --git a/src/Repository/CommentRepository.php b/src/Repository/CommentRepository.php
index 75849817..cea2a74a 100644
--- a/src/Repository/CommentRepository.php
+++ b/src/Repository/CommentRepository.php
@@ -5,6 +5,7 @@ namespace App\Repository;
 use App\Entity\Comment;
 use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
 use Doctrine\Common\Collections\Criteria;
+use Doctrine\ORM\QueryBuilder;
 use Symfony\Bridge\Doctrine\RegistryInterface;
 
 /**
@@ -30,9 +31,8 @@ class CommentRepository extends ServiceEntityRepository
 
     /**
      * @param string|null $term
-     * @return Comment[]
      */
-    public function findAllWithSearch(?string $term)
+    public function getWithSearchQueryBuilder(?string $term): QueryBuilder
     {
         $qb = $this->createQueryBuilder('c')
             ->innerJoin('c.article', 'a')
@@ -46,8 +46,6 @@ class CommentRepository extends ServiceEntityRepository
 
         return $qb
             ->orderBy('c.createdAt', 'DESC')
-            ->getQuery()
-            ->getResult()
         ;
     }
 
diff --git a/templates/comment_admin/index.html.twig b/templates/comment_admin/index.html.twig
index 83d56ccf..8b13f9eb 100644
--- a/templates/comment_admin/index.html.twig
+++ b/templates/comment_admin/index.html.twig
@@ -7,7 +7,7 @@
 {% block content_body %}
     <div class="row">
         <div class="col-sm-12">
-            <h1>Manage Comments</h1>
+            <h1>Manage Comments ({{ pagination.getTotalItemCount }})</h1>
 
             <form>
                 <div class="input-group mb-3">
@@ -36,7 +36,7 @@
                 </tr>
                 </thead>
                 <tbody>
-                {% for comment in comments %}
+                {% for comment in pagination %}
                     <tr>
                         <td>
                             <a href="{{ path('article_show', {'slug': comment.article.slug}) }}">
@@ -62,6 +62,8 @@
                 {% endfor %}
                 </tbody>
             </table>
+
+            {{ knp_pagination_render(pagination) }}
         </div>
     </div>
 {% endblock %}
