diff --git a/src/Service/MarkdownHelper.php b/src/Service/MarkdownHelper.php
index 1b9079c1..b48ba902 100644
--- a/src/Service/MarkdownHelper.php
+++ b/src/Service/MarkdownHelper.php
@@ -11,12 +11,14 @@ class MarkdownHelper
     private $cache;
     private $markdown;
     private $logger;
+    private $isDebug;
 
-    public function __construct(AdapterInterface $cache, MarkdownInterface $markdown, LoggerInterface $markdownLogger)
+    public function __construct(AdapterInterface $cache, MarkdownInterface $markdown, LoggerInterface $markdownLogger, bool $isDebug)
     {
         $this->cache = $cache;
         $this->markdown = $markdown;
         $this->logger = $markdownLogger;
+        $this->isDebug = $isDebug;
     }
 
     public function parse(string $source): string
@@ -25,7 +27,11 @@ class MarkdownHelper
             $this->logger->info('They are talking about bacon again!');
         }
 
-        dump($this->cache);die;
+        // skip caching entirely in debug
+        if ($this->isDebug) {
+            return $this->markdown->transform($source);
+        }
+
         $item = $this->cache->getItem('markdown_'.md5($source));
         if (!$item->isHit()) {
             $item->set($this->markdown->transform($source));
