diff --git a/src/Entity/User.php b/src/Entity/User.php
index 7eac7b0a..d054821e 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -88,7 +88,7 @@ class User implements UserInterface
      */
     public function getPassword()
     {
-        // not needed for apps that do not check user passwords
+        return $this->password;
     }
 
     /**
@@ -96,7 +96,7 @@ class User implements UserInterface
      */
     public function getSalt()
     {
-        // not needed for apps that do not check user passwords
+        // not needed when using bcrypt or argon
     }
 
     /**
