diff --git a/templates/email/author-weekly-report-pdf.html.twig b/templates/email/author-weekly-report-pdf.html.twig
index 3e0fa571..6a0c57b3 100644
--- a/templates/email/author-weekly-report-pdf.html.twig
+++ b/templates/email/author-weekly-report-pdf.html.twig
@@ -6,8 +6,18 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Document</title>
+
+    {{ encore_entry_link_tags('app') }}
 </head>
 <body>
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-12">
+                <h1>Weekly report {{ 'now'|date('Y-m-d') }}</h1>
 
+                {{ include('email/_report-table.html.twig') }}
+            </div>
+        </div>
+    </div>
 </body>
 </html>
