diff --git a/templates/email/author-weekly-report-pdf.html.twig b/templates/email/author-weekly-report-pdf.html.twig
new file mode 100644
index 000000000..3e0fa571c
--- /dev/null
+++ b/templates/email/author-weekly-report-pdf.html.twig
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+
+</body>
+</html>
