diff --git a/src/Command/AuthorWeeklyReportSendCommand.php b/src/Command/AuthorWeeklyReportSendCommand.php
index 0e1307c9d..c8aec34f2 100644
--- a/src/Command/AuthorWeeklyReportSendCommand.php
+++ b/src/Command/AuthorWeeklyReportSendCommand.php
@@ -75,7 +75,8 @@ class AuthorWeeklyReportSendCommand extends Command
                 ->context([
                     'author' => $author,
                     'articles' => $articles,
-                ]);
+                ])
+                ->attach($pdf, sprintf('weekly-report-%s.pdf', date('Y-m-d')));
             $this->mailer->send($email);
         }
         $io->progressFinish();
