diff --git a/templates/email/author-weekly-report-pdf.html.twig b/templates/email/author-weekly-report-pdf.html.twig
index 6a0c57b38..b1c8c2e8a 100644
--- a/templates/email/author-weekly-report-pdf.html.twig
+++ b/templates/email/author-weekly-report-pdf.html.twig
@@ -7,7 +7,9 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Document</title>
 
-    {{ encore_entry_link_tags('app') }}
+    {% for path in encore_entry_css_files('app') %}
+        <link rel="stylesheet" href="{{ absolute_url(path) }}">
+    {% endfor %}
 </head>
 <body>
     <div class="container">
