diff --git a/templates/email/author-weekly-report-pdf.html.twig b/templates/email/author-weekly-report-pdf.html.twig
new file mode 100644
index 00000000..a44238b7
--- /dev/null
+++ b/templates/email/author-weekly-report-pdf.html.twig
@@ -0,0 +1,21 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+
+    {{ encore_entry_link_tags('app') }}
+</head>
+<body>
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-12">
+                {{ include('email/_report-table.html.twig') }}
+            </div>
+        </div>
+    </div>
+</body>
+</html>
