diff --git a/config/packages/php_translation.yaml b/config/packages/php_translation.yaml
index d3446dc2..7246b410 100644
--- a/config/packages/php_translation.yaml
+++ b/config/packages/php_translation.yaml
@@ -9,3 +9,4 @@ translation:
             output_dir: "%kernel.project_dir%/translations"
             excluded_names: ["*TestCase.php", "*Test.php"]
             excluded_dirs: [cache, data, logs]
+            blacklist_domains: [app]
diff --git a/translations/messages+intl-icu.en.yaml b/translations/app+intl-icu.en.yaml
similarity index 100%
rename from translations/messages+intl-icu.en.yaml
rename to translations/app+intl-icu.en.yaml
diff --git a/translations/messages+intl-icu.es.yaml b/translations/app+intl-icu.es.yaml
similarity index 100%
rename from translations/messages+intl-icu.es.yaml
rename to translations/app+intl-icu.es.yaml
diff --git a/translations/messages+intl-icu.uk.yaml b/translations/app+intl-icu.uk.yaml
similarity index 100%
rename from translations/messages+intl-icu.uk.yaml
rename to translations/app+intl-icu.uk.yaml
