diff --git a/templates/base.html.twig b/templates/base.html.twig
index d24549a1..6e540d9f 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -96,7 +96,9 @@
 
         <footer class="footer">
             <div class="container text-center">
-                <span class="text-muted">Made with <i class="fa fa-heart" style="color: red;"></i> by the guys and gals at <a href="https://symfonycasts.com">SymfonyCasts</a></span>
+                <span class="text-muted">
+                    {% trans %}footer.html{% endtrans %}
+                </span>
             </div>
         </footer>
 
diff --git a/translations/messages.en.yaml b/translations/messages.en.yaml
index 723479bc..c2487005 100644
--- a/translations/messages.en.yaml
+++ b/translations/messages.en.yaml
@@ -13,3 +13,4 @@ page.weather.space.content: There's no way to deny it. It is going to be very co
 language.en: English
 language.es: Spanish
 language.uk: Ukrainian
+footer.html: 'Made with <i class="fa fa-heart" style="color: red;"></i> by the guys and gals at <a href="https://symfonycasts.com">SymfonyCasts</a>'
