diff --git a/templates/base.html.twig b/templates/base.html.twig
index d8599f00..e1c552d5 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -97,7 +97,11 @@
         <footer class="footer">
             <div class="container text-center">
                 <span class="text-muted">
-                    {% trans %}footer.html{% endtrans %}
+                    {% trans with {
+                        '%icon%': '<i class="fa fa-heart" style="color: red;"></i>',
+                        '%link_start%': '<a href="https://symfonycasts.com">',
+                        '%link_end%': '</a>',
+                    } %}footer.html{% endtrans %}
                 </span>
             </div>
         </footer>
diff --git a/translations/messages.en.yaml b/translations/messages.en.yaml
index c2487005..38ccfcbb 100644
--- a/translations/messages.en.yaml
+++ b/translations/messages.en.yaml
@@ -13,4 +13,4 @@ page.weather.space.content: There's no way to deny it. It is going to be very co
 language.en: English
 language.es: Spanish
 language.uk: Ukrainian
-footer.html: 'Made with <i class="fa fa-heart" style="color: red;"></i> by the guys and gals at <a href="https://symfonycasts.com">SymfonyCasts</a>'
+footer.html: 'Made with %icon% by the guys and gals at %link_start%SymfonyCasts%link_end%'
