diff --git a/templates/article/homepage.html.twig b/templates/article/homepage.html.twig
index 22b5bf9..cae24a3 100644
--- a/templates/article/homepage.html.twig
+++ b/templates/article/homepage.html.twig
@@ -49,7 +49,7 @@
                     <p>{{ 'ice_cream_promo.title.html'|trans|raw }}</p>
                     {% set spaceIceCreamPrice = 9.99 %}
                     <button class="btn btn-info js-purchase-btn">
-                        Buy Now for ${{ spaceIceCreamPrice|number_format(2) }}!
+                        {{ 'ice_cream_promo.button'|trans({'%price%': '$'~spaceIceCreamPrice|number_format(2)}) }}
                     </button>
                 </div>
 
diff --git a/translations/messages.en.yaml b/translations/messages.en.yaml
index cef7e40..5da4c10 100644
--- a/translations/messages.en.yaml
+++ b/translations/messages.en.yaml
@@ -1,5 +1,6 @@
 trending_quotes: 'Trending Quotes'
 ice_cream_promo.title.html: '<span class="advertisement-text">New:</span> Space Ice Cream!'
+ice_cream_promo.button: Buy Now for %price%!
 search.placeholder: Search
 search.button: Search
 page.weather.header: Universal Forecast
