diff --git a/public/css/styles.css b/public/css/styles.css
index fa8a899f..ffdebfe3 100644
--- a/public/css/styles.css
+++ b/public/css/styles.css
@@ -170,6 +170,11 @@ html {height:100%}
     background-color: #fff;
 }
 
+.show-article-container.show-article-container-border-green {
+    border-top: 3px solid green;
+    border-radius: 3px;
+}
+
 .show-article-img {
     width: 250px;
     height: auto;
@@ -203,6 +208,10 @@ html {height:100%}
     font-size: 1.5em;
 }
 
+.comment-container {
+    max-width: 600px;
+}
+
 .comment-img {
     width: 50px;
     height: auto;
@@ -221,6 +230,9 @@ html {height:100%}
     .comment-form {
         min-width: 260px;
     }
+    .comment-container {
+        max-width: 280px;
+    }
 }
 
 
diff --git a/templates/base.html.twig b/templates/base.html.twig
index 442c02d8..0e8c3535 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -7,7 +7,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
         {% block stylesheets %}
-            <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
+            <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
             <link rel="stylesheet" href="{{ asset('css/font-awesome.css') }}">
             <link rel="stylesheet" href="{{ asset('css/styles.css') }}">
         {% endblock %}
