diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index fc31cb6d..52c4f62f 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -26,7 +26,6 @@ class ArticleController extends AbstractController
             'Woohoo! I\'m going on an all-asteroid diet!',
             'I like bacon too! Buy some from my site! bakinsomebacon.com',
         ];
-        dump($slug, $this);
 
         return $this->render('article/show.html.twig', [
             'title' => ucwords(str_replace('-', ' ', $slug)),
diff --git a/templates/article/show.html.twig b/templates/article/show.html.twig
index c48b53b2..51be37b6 100644
--- a/templates/article/show.html.twig
+++ b/templates/article/show.html.twig
@@ -3,6 +3,7 @@
 {% block title %}Read: {{ title }}{% endblock %}
 
 {% block body %}
+{{ dump() }}
 <h1>{{ title }}</h1>
 
 <div>
