diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index 335e15fc..9699ef17 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -36,7 +36,7 @@ class ArticleController extends AbstractController
     /**
      * @Route("/news/{slug}", name="article_show")
      */
-    public function show($slug, MarkdownHelper $markdownHelper, SlackClient $slack, EntityManagerInterface $em)
+    public function show($slug, SlackClient $slack, EntityManagerInterface $em)
     {
         if ($slug === 'khaaaaaan') {
             $slack->sendMessage('Kahn', 'Ah, Kirk, my old friend...');
@@ -49,40 +49,15 @@ class ArticleController extends AbstractController
             throw $this->createNotFoundException(sprintf('No article for slug "%s"', $slug));
         }
 
-        dump($article);die;
-
         $comments = [
             'I ate a normal rock once. It did NOT taste like bacon!',
             'Woohoo! I\'m going on an all-asteroid diet!',
             'I like bacon too! Buy some from my site! bakinsomebacon.com',
         ];
 
-        $articleContent = <<<EOF
-Spicy **jalapeno bacon** ipsum dolor amet veniam shank in dolore. Ham hock nisi landjaeger cow,
-lorem proident [beef ribs](https://baconipsum.com/) aute enim veniam ut cillum pork chuck picanha. Dolore reprehenderit
-labore minim pork belly spare ribs cupim short loin in. Elit exercitation eiusmod dolore cow
-**turkey** shank eu pork belly meatball non cupim.
-
-Laboris beef ribs fatback fugiat eiusmod jowl kielbasa alcatra dolore velit ea ball tip. Pariatur
-laboris sunt venison, et laborum dolore minim non meatball. Shankle eu flank aliqua shoulder,
-capicola biltong frankfurter boudin cupim officia. Exercitation fugiat consectetur ham. Adipisicing
-picanha shank et filet mignon pork belly ut ullamco. Irure velit turducken ground round doner incididunt
-occaecat lorem meatball prosciutto quis strip steak.
-
-Meatball adipisicing ribeye bacon strip steak eu. Consectetur ham hock pork hamburger enim strip steak
-mollit quis officia meatloaf tri-tip swine. Cow ut reprehenderit, buffalo incididunt in filet mignon
-strip steak pork belly aliquip capicola officia. Labore deserunt esse chicken lorem shoulder tail consectetur
-cow est ribeye adipisicing. Pig hamburger pork belly enim. Do porchetta minim capicola irure pancetta chuck
-fugiat.
-EOF;
-
-        $articleContent = $markdownHelper->parse($articleContent);
-
         return $this->render('article/show.html.twig', [
-            'title' => ucwords(str_replace('-', ' ', $slug)),
-            'slug' => $slug,
+            'article' => $article,
             'comments' => $comments,
-            'articleContent' => $articleContent,
         ]);
     }
 
diff --git a/templates/article/show.html.twig b/templates/article/show.html.twig
index 155abb1e..66663311 100644
--- a/templates/article/show.html.twig
+++ b/templates/article/show.html.twig
@@ -1,6 +1,6 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}Read: {{ title }}{% endblock %}
+{% block title %}Read: {{ article.title }}{% endblock %}
 
 {% block body %}
 
@@ -12,13 +12,13 @@
                     <div class="col-sm-12">
                         <img class="show-article-img" src="{{ asset('images/asteroid.jpeg') }}">
                         <div class="show-article-title-container d-inline-block pl-3 align-middle">
-                            <span class="show-article-title ">{{ title }}</span>
+                            <span class="show-article-title ">{{ article.title }}</span>
                             <br>
                             <span class="align-left article-details"><img class="article-author-img rounded-circle" src="{{ asset('images/alien-profile.png') }}"> Mike Ferengi </span>
                             <span class="pl-2 article-details"> 4 hours ago</span>
                             <span class="pl-2 article-details">
                                 <span class="js-like-article-count">5</span>
-                                <a href="{{ path('article_toggle_heart', {slug: slug}) }}" class="fa fa-heart-o like-article js-like-article"></a>
+                                <a href="{{ path('article_toggle_heart', {slug: article.slug}) }}" class="fa fa-heart-o like-article js-like-article"></a>
                             </span>
                         </div>
                     </div>
@@ -26,7 +26,7 @@
                 <div class="row">
                     <div class="col-sm-12">
                         <div class="article-text">
-                            {{ articleContent|raw }}
+                            {{ article.content|markdown }}
                         </div>
                     </div>
                 </div>
