diff --git a/src/Form/UserRegistrationFormType.php b/src/Form/UserRegistrationFormType.php
new file mode 100644
index 00000000..8a949167
--- /dev/null
+++ b/src/Form/UserRegistrationFormType.php
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\User;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class UserRegistrationFormType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('email')
+            ->add('roles')
+            ->add('firstName')
+            ->add('password')
+            ->add('twitterUsername')
+        ;
+    }
+
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            'data_class' => User::class,
+        ]);
+    }
+}
