diff --git a/src/Controller/SecurityController.php b/src/Controller/SecurityController.php
index 40877daa..fdf81e82 100644
--- a/src/Controller/SecurityController.php
+++ b/src/Controller/SecurityController.php
@@ -48,12 +48,11 @@ class SecurityController extends AbstractController
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
-            dd($form['plainPassword']->getData());
             /** @var User $user */
             $user = $form->getData();
             $user->setPassword($passwordEncoder->encodePassword(
                 $user,
-                $user->getPassword()
+                $form['plainPassword']->getData()
             ));
 
             $em = $this->getDoctrine()->getManager();
