diff --git a/src/Controller/SecurityController.php b/src/Controller/SecurityController.php
index 349ddaf3..ff02f24c 100644
--- a/src/Controller/SecurityController.php
+++ b/src/Controller/SecurityController.php
@@ -50,6 +50,12 @@ class SecurityController extends AbstractController
                 $user,
                 $request->request->get('password')
             ));
+
+            $em = $this->getDoctrine()->getManager();
+            $em->persist($user);
+            $em->flush();
+
+            return $this->redirectToRoute('app_account');
         }
 
         return $this->render('security/register.html.twig');
