diff --git a/src/Form/UserRegistrationFormType.php b/src/Form/UserRegistrationFormType.php
index d31b94b5..7caa105d 100644
--- a/src/Form/UserRegistrationFormType.php
+++ b/src/Form/UserRegistrationFormType.php
@@ -13,7 +13,9 @@ class UserRegistrationFormType extends AbstractType
     {
         $builder
             ->add('email')
-            ->add('password')
+            // don't use password: avoid EVER setting that on a
+            // field that might be persisted
+            ->add('plainPassword')
         ;
     }
 
