diff --git a/src/Form/UserRegistrationFormType.php b/src/Form/UserRegistrationFormType.php
index 56fdc643..bb65891b 100644
--- a/src/Form/UserRegistrationFormType.php
+++ b/src/Form/UserRegistrationFormType.php
@@ -4,6 +4,7 @@ namespace App\Form;
 
 use App\Entity\User;
 use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\PasswordType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
@@ -15,7 +16,7 @@ class UserRegistrationFormType extends AbstractType
             ->add('email')
             // don't use password: avoid EVER setting that on a
             // field that might be persisted
-            ->add('plainPassword', null, [
+            ->add('plainPassword', PasswordType::class, [
                 'mapped' => false
             ]);
         ;
