diff --git a/src/Entity/User.php b/src/Entity/User.php
index 11de028c..ed71c83e 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -24,7 +24,7 @@ class User implements UserInterface
     /**
      * @ORM\Column(type="string", length=180, unique=true)
      * @Groups("main")
-     * @Assert\NotBlank()
+     * @Assert\NotBlank(message="Please enter an email")
      * @Assert\Email()
      */
     private $email;
