diff --git a/config/packages/liip_imagine.yaml b/config/packages/liip_imagine.yaml
index 372e159e3..a86c1f52d 100644
--- a/config/packages/liip_imagine.yaml
+++ b/config/packages/liip_imagine.yaml
@@ -14,6 +14,13 @@ liip_imagine:
                     mode:          outbound
                     allow_upscale: true
 
+        squared_thumbnail_medium:
+            filters:
+                thumbnail:
+                    size:          [500, 500]
+                    mode:          outbound
+                    allow_upscale: true
+
 #
 #            # set your image quality defaults
 #            jpeg_quality:          85
diff --git a/templates/article/show.html.twig b/templates/article/show.html.twig
index 08a9a295a..5615115f5 100644
--- a/templates/article/show.html.twig
+++ b/templates/article/show.html.twig
@@ -5,7 +5,7 @@
 {% block content_body %}
     <div class="row">
         <div class="col-sm-12">
-            <img class="show-article-img" src="{{ uploaded_asset(article.imagePath) }}">
+            <img class="show-article-img" src="{{ uploaded_asset(article.imagePath)|imagine_filter('squared_thumbnail_medium') }}">
             <div class="show-article-title-container d-inline-block pl-3 align-middle">
                 <span class="show-article-title ">{{ article.title }}</span>
                 <br>
