diff --git a/src/Entity/Article.php b/src/Entity/Article.php
index 97752eec..eb36fa25 100644
--- a/src/Entity/Article.php
+++ b/src/Entity/Article.php
@@ -3,6 +3,7 @@
 namespace App\Entity;
 
 use App\Repository\CommentRepository;
+use App\Service\UploaderHelper;
 use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\Common\Collections\Collection;
 use Doctrine\Common\Collections\Criteria;
@@ -184,7 +185,7 @@ class Article
 
     public function getImagePath()
     {
-        return 'uploads/article_images/'.$this->getImageFilename();
+        return 'uploads/'.UploaderHelper::ARTICLE_IMAGE.'/'.$this->getImageFilename();
     }
 
     /**
diff --git a/src/Service/UploaderHelper.php b/src/Service/UploaderHelper.php
index 660bd3dd..d245323f 100644
--- a/src/Service/UploaderHelper.php
+++ b/src/Service/UploaderHelper.php
@@ -7,6 +7,8 @@ use Symfony\Component\HttpFoundation\File\UploadedFile;
 
 class UploaderHelper
 {
+    const ARTICLE_IMAGE = 'article_image';
+
     private $uploadsPath;
 
     public function __construct(string $uploadsPath)
@@ -16,7 +18,7 @@ class UploaderHelper
 
     public function uploadArticleImage(UploadedFile $uploadedFile): string
     {
-        $destination = $this->uploadsPath.'/article_image';
+        $destination = $this->uploadsPath.'/'.self::ARTICLE_IMAGE;
 
         $originalFilename = pathinfo($uploadedFile->getClientOriginalName(), PATHINFO_FILENAME);
         $newFilename = Urlizer::urlize($originalFilename).'-'.uniqid().'.'.$uploadedFile->guessExtension();
