diff --git a/templates/article_admin/_form.html.twig b/templates/article_admin/_form.html.twig
index fe51885..ec0bcc7 100644
--- a/templates/article_admin/_form.html.twig
+++ b/templates/article_admin/_form.html.twig
@@ -12,7 +12,7 @@
             }) }}
         </div>
         <div class="col-sm-3">
-            {% if articleForm.vars.data.imageFilename %}
+            {% if articleForm.vars.data.imageFilename|default %}
                 <img src="{{ uploaded_asset(articleForm.vars.data.imagePath)|imagine_filter('squared_thumbnail_small') }}" height="100">
             {% endif %}
         </div>
@@ -36,4 +36,4 @@
     {% endif %}
 
     <button type="submit" class="btn btn-primary">{{ button_text }}</button>
-{{ form_end(articleForm) }}
\ No newline at end of file
+{{ form_end(articleForm) }}
