diff --git a/src/DataFixtures/UserFixture.php b/src/DataFixtures/UserFixture.php
index a2836e6b..a6f0f584 100644
--- a/src/DataFixtures/UserFixture.php
+++ b/src/DataFixtures/UserFixture.php
@@ -30,6 +30,20 @@ class UserFixture extends BaseFixture
             return $user;
         });
 
+        $this->createMany(3, 'admin_users', function($i) {
+            $user = new User();
+            $user->setEmail(sprintf('admin%d@thespacebar.com', $i));
+            $user->setFirstName($this->faker->firstName);
+            $user->setRoles(['ROLE_ADMIN']);
+
+            $user->setPassword($this->passwordEncoder->encodePassword(
+                $user,
+                'engage'
+            ));
+
+            return $user;
+        });
+
         $manager->flush();
     }
 }
