diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 5da7588e..455e672d 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -10,7 +10,7 @@ use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
 /**
- * @IsGranted("ROLE_ADMIN")
+ * @IsGranted("ROLE_ADMIN_ARTICLE")
  */
 class ArticleAdminController extends AbstractController
 {
diff --git a/src/Controller/CommentAdminController.php b/src/Controller/CommentAdminController.php
index 026e68d1..ef6b7299 100644
--- a/src/Controller/CommentAdminController.php
+++ b/src/Controller/CommentAdminController.php
@@ -10,7 +10,7 @@ use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
 /**
- * @IsGranted("ROLE_ADMIN")
+ * @IsGranted("ROLE_ADMIN_COMMENT")
  */
 class CommentAdminController extends Controller
 {
diff --git a/templates/base.html.twig b/templates/base.html.twig
index 1b48f0c9..5a777016 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -40,7 +40,7 @@
                             </a>
                             <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
                                 <a class="dropdown-item" href="{{ path('app_account') }}">Profile</a>
-                                {% if is_granted('ROLE_ADMIN') %}
+                                {% if is_granted('ROLE_ADMIN_ARTICLE') %}
                                     <a class="dropdown-item" href="{{ path('admin_article_new') }}">Create Post</a>
                                 {% endif %}
                                 <a class="dropdown-item" href="{{ path('app_logout') }}">Logout</a>
