diff --git a/src/Controller/AccountController.php b/src/Controller/AccountController.php
index 3829472f..817a1c00 100644
--- a/src/Controller/AccountController.php
+++ b/src/Controller/AccountController.php
@@ -2,9 +2,13 @@
 
 namespace App\Controller;
 
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\Routing\Annotation\Route;
 
+/**
+ * @IsGranted("ROLE_USER")
+ */
 class AccountController extends AbstractController
 {
     /**
