diff --git a/src/Controller/WeatherController.php b/src/Controller/WeatherController.php
index f657c63..6a051aa 100644
--- a/src/Controller/WeatherController.php
+++ b/src/Controller/WeatherController.php
@@ -9,7 +9,11 @@ use Symfony\Component\Routing\Annotation\Route;
 class WeatherController extends AbstractController
 {
     /**
-     * @Route("/{_locale}/weather", name="app_weather", requirements={"_locale": "%app_locales%"})
+     * @Route({
+     *     "en": "/en/weather",
+     *     "es": "/es/clima",
+     *     "uk": "/uk/pogoda"
+     * }, name="app_weather", requirements={"_locale": "%app_locales%"})
      */
     public function index()
     {
