diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index 2778e13..e7b2a3f 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -24,7 +24,7 @@ class ArticleController extends AbstractController
     }
 
     /**
-     * @Route("/{_locale}", name="app_homepage")
+     * @Route("/{_locale}", name="app_homepage", requirements={"_locale": "en|es|uk"})
      */
     public function homepage(ArticleRepository $repository)
     {
