diff --git a/config/services.yaml b/config/services.yaml
index 4b2f7050..d2231ada 100644
--- a/config/services.yaml
+++ b/config/services.yaml
@@ -6,8 +6,7 @@
 parameters:
     cache_adapter: cache.adapter.apcu
     locale: 'en'
-    uploads_dir_name: 'uploads'
-    uploads_base_url: '%env(SITE_BASE_URL)%/%uploads_dir_name%'
+    uploads_base_url: 'https://s3.amazonaws.com/%env(AWS_S3_BUCKET_NAME)%'
 
 services:
     # default configuration for services in *this* file
diff --git a/src/Service/UploaderHelper.php b/src/Service/UploaderHelper.php
index db310aab..a742fe68 100644
--- a/src/Service/UploaderHelper.php
+++ b/src/Service/UploaderHelper.php
@@ -60,6 +60,12 @@ class UploaderHelper
 
     public function getPublicPath(string $path): string
     {
+        $fullPath = $this->publicAssetBaseUrl.'/'.$path;
+        // if it's already absolute, just return
+        if (strpos($fullPath, '://') !== false) {
+            return $fullPath;
+        }
+
         // needed if you deploy under a subdirectory
         return $this->requestStackContext
             ->getBasePath().$this->publicAssetBaseUrl.'/'.$path;
