diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index 73126131..8aa65dd2 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -129,9 +129,16 @@ class ArticleReferenceAdminController extends BaseController
         $article = $reference->getArticle();
         $this->denyAccessUnlessGranted('MANAGE', $article);
 
+        $disposition = HeaderUtils::makeDisposition(
+            ResponseHeaderBag::DISPOSITION_ATTACHMENT,
+            $reference->getOriginalFilename()
+        );
+
         $command = $s3Client->getCommand('GetObject', [
             'Bucket' => $s3BucketName,
-            'Key' => $reference->getFilePath()
+            'Key' => $reference->getFilePath(),
+            'ResponseContentType' => $reference->getMimeType(),
+            'ResponseContentDisposition' => $disposition,
         ]);
         $request = $s3Client->createPresignedRequest($command, '+30 minutes');
 
