diff --git a/src/Controller/ArticleReferenceAdminController.php b/src/Controller/ArticleReferenceAdminController.php
index 230ad59f..63d7995e 100644
--- a/src/Controller/ArticleReferenceAdminController.php
+++ b/src/Controller/ArticleReferenceAdminController.php
@@ -130,7 +130,7 @@ class ArticleReferenceAdminController extends BaseController
 
         $response = new StreamedResponse(function() use ($reference, $uploaderHelper) {
             $outputStream = fopen('php://output', 'wb');
-            $fileStream = $uploaderHelper->readStream($reference->getFilePath(), false);
+            $fileStream = $uploaderHelper->readStream($reference->getFilePath());
 
             stream_copy_to_stream($fileStream, $outputStream);
         });
@@ -155,7 +155,7 @@ class ArticleReferenceAdminController extends BaseController
         $entityManager->remove($reference);
         $entityManager->flush();
 
-        $uploaderHelper->deleteFile($reference->getFilePath(), false);
+        $uploaderHelper->deleteFile($reference->getFilePath());
 
         return new Response(null, 204);
     }
diff --git a/src/Service/UploaderHelper.php b/src/Service/UploaderHelper.php
index f639a014..f7a120d3 100644
--- a/src/Service/UploaderHelper.php
+++ b/src/Service/UploaderHelper.php
@@ -73,11 +73,9 @@ class UploaderHelper
     /**
      * @return resource
      */
-    public function readStream(string $path, bool $isPublic)
+    public function readStream(string $path)
     {
-        $filesystem = $isPublic ? $this->filesystem : $this->privateFilesystem;
-
-        $resource = $filesystem->readStream($path);
+        $resource = $this->filesystem->readStream($path);
 
         if ($resource === false) {
             throw new \Exception(sprintf('Error opening stream for "%s"', $path));
@@ -86,11 +84,9 @@ class UploaderHelper
         return $resource;
     }
 
-    public function deleteFile(string $path, bool $isPublic)
+    public function deleteFile(string $path)
     {
-        $filesystem = $isPublic ? $this->filesystem : $this->privateFilesystem;
-
-        $result = $filesystem->delete($path);
+        $result = $this->filesystem->delete($path);
 
         if ($result === false) {
             throw new \Exception(sprintf('Error deleting "%s"', $path));
