diff --git a/assets/css/app.css b/assets/css/app.scss
similarity index 100%
rename from assets/css/app.css
rename to assets/css/app.scss
diff --git a/assets/js/app.js b/assets/js/app.js
index 7f467b9..5a3c2d2 100644
--- a/assets/js/app.js
+++ b/assets/js/app.js
@@ -6,7 +6,7 @@
  */
 
 // any CSS you require will output into a single css file (app.css in this case)
-import '../css/app.css';
+import '../css/app.scss';
 
 import $ from 'jquery';
 import 'bootstrap'; // adds functions to jQuery
