diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
new file mode 100644
index 00000000..d7ffa83e
--- /dev/null
+++ b/src/Controller/ArticleAdminController.php
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ArticleAdminController extends AbstractController
+{
+    /**
+     * @Route("/admin/article/new")
+     */
+    public function new()
+    {
+        return new Response('space rocks... include comets, asteroids & meteoroids');
+    }
+}
