diff --git a/src/Controller/AdminController.php b/src/Controller/AdminController.php
new file mode 100644
index 0000000..fbdeec3
--- /dev/null
+++ b/src/Controller/AdminController.php
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class AdminController extends AbstractController
+{
+    /**
+     * @Route("/admin/article/new")
+     */
+    public function newAction()
+    {
+        return new Response('boring... but WORKING route!');
+    }
+}
