diff --git a/src/DataFixtures/ArticleFixtures.php b/src/DataFixtures/ArticleFixtures.php
index a4f0a3de..9ff194cd 100644
--- a/src/DataFixtures/ArticleFixtures.php
+++ b/src/DataFixtures/ArticleFixtures.php
@@ -62,6 +62,7 @@ EOF
             $comment1 = new Comment();
             $comment1->setAuthorName('Mike Ferengi');
             $comment1->setContent('I ate a normal rock once. It did NOT taste like bacon!');
+            $comment1->setArticle($article);
             $manager->persist($comment1);
         });
 
