diff --git a/src/DataFixtures/ArticleFixture.php b/src/DataFixtures/ArticleFixture.php
index 674366d..bb93fc8 100644
--- a/src/DataFixtures/ArticleFixture.php
+++ b/src/DataFixtures/ArticleFixture.php
@@ -64,6 +64,12 @@ EOF
             $comment1->setContent('I ate a normal rock once. It did NOT taste like bacon!');
             $comment1->setArticle($article);
             $manager->persist($comment1);
+
+            $comment2 = new Comment();
+            $comment2->setAuthorName('Mike Ferengi');
+            $comment2->setContent('Woohoo! I\'m going on an all-asteroid diet!');
+            $comment2->setArticle($article);
+            $manager->persist($comment2);
         });
 
         $manager->flush();
