diff --git a/config/secrets/dev/dev.MAILER_DSN.97e321.php b/config/secrets/dev/dev.MAILER_DSN.97e321.php
new file mode 100644
index 000000000..a7da03cb4
--- /dev/null
+++ b/config/secrets/dev/dev.MAILER_DSN.97e321.php
@@ -0,0 +1,3 @@
+<?php // dev.MAILER_DSN.97e321 on Thu, 09 Jan 2020 19:18:02 +0000
+
+return "ih\xB4\x0B\xE2\x7B\xECHkMP\x80\x5B\x0Es\x11s\x5D\x24\xA9\xAC\xD9CS\x81\x8A\x16\xD5\xB6\x1Dg\x7DMB\xF1\x5D\x2F\x40\x88\xD4\xFEF\x88\x20\x3BA\xB9\xD7\xC2\xC1M\x06\xE9\x5D\x10\x92\xA4\x3B\xA4";
diff --git a/config/secrets/dev/dev.decrypt.private.php b/config/secrets/dev/dev.decrypt.private.php
new file mode 100644
index 000000000..7c9252c7c
--- /dev/null
+++ b/config/secrets/dev/dev.decrypt.private.php
@@ -0,0 +1,3 @@
+<?php // dev.decrypt.private on Thu, 09 Jan 2020 19:18:02 +0000
+
+return "\xB6\x27V\x97\x17v\x08\x5E\xC4pC\xC9\x07\x02S\xDCEU8\x88\xD1\x2C\xFDN0\x1B\xB5\x10\x07\x10\xDE\x99\xF7\xFDY\x1Ebh\xD3\x0C\x2A\xDF\xB6\x5E\xEC\x00\x12\x8C\x284\xB5\x2C\xE7\xCD\xBC3\xFE\x9B\xD1\x1C\x28\xBE\xB7\x1B";
diff --git a/config/secrets/dev/dev.encrypt.public.php b/config/secrets/dev/dev.encrypt.public.php
new file mode 100644
index 000000000..5ff0bf499
--- /dev/null
+++ b/config/secrets/dev/dev.encrypt.public.php
@@ -0,0 +1,3 @@
+<?php // dev.encrypt.public on Thu, 09 Jan 2020 19:18:02 +0000
+
+return "\xF7\xFDY\x1Ebh\xD3\x0C\x2A\xDF\xB6\x5E\xEC\x00\x12\x8C\x284\xB5\x2C\xE7\xCD\xBC3\xFE\x9B\xD1\x1C\x28\xBE\xB7\x1B";
diff --git a/config/secrets/dev/dev.list.php b/config/secrets/dev/dev.list.php
new file mode 100644
index 000000000..d3c920aef
--- /dev/null
+++ b/config/secrets/dev/dev.list.php
@@ -0,0 +1,5 @@
+<?php
+
+return array (
+  'MAILER_DSN' => NULL,
+);
