diff --git a/config/services.yaml b/config/services.yaml
index 913a6f9..0a90f9c 100644
--- a/config/services.yaml
+++ b/config/services.yaml
@@ -23,6 +23,7 @@ services:
             $uploadsFilesystem: '@oneup_flysystem.uploads_filesystem_filesystem'
             $uploadedAssetsBaseUrl: '%uploads_base_url%'
             string $publicDir: '%kernel.project_dir%/public'
+            $mailerDsn: '%env(MAILER_DSN)%'
 
     # custom aliases for autowiring
     Nexy\Slack\Client: '@nexy_slack.client'
diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index c265b81..9c2fc18 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -26,8 +26,9 @@ class ArticleController extends AbstractController
     /**
      * @Route("/", name="app_homepage")
      */
-    public function homepage(ArticleRepository $repository)
+    public function homepage(ArticleRepository $repository, $mailerDsn)
     {
+        dump($mailerDsn);die;
         $articles = $repository->findAllPublishedOrderedByNewest();
 
         return $this->render('article/homepage.html.twig', [
