diff --git a/config/services.yaml b/config/services.yaml
index ce0336ba6..312f7af42 100644
--- a/config/services.yaml
+++ b/config/services.yaml
@@ -22,7 +22,6 @@ services:
             $uploadsFilesystem: '@oneup_flysystem.uploads_filesystem_filesystem'
             $uploadedAssetsBaseUrl: '%uploads_base_url%'
             string $publicDir: '%kernel.project_dir%/public'
-            $mailerDsn: '%env(MAILER_DSN)%'
 
     # custom aliases for autowiring
     Nexy\Slack\Client: '@nexy_slack.client'
diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index 9c2fc18c4..c265b81ea 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -26,9 +26,8 @@ class ArticleController extends AbstractController
     /**
      * @Route("/", name="app_homepage")
      */
-    public function homepage(ArticleRepository $repository, $mailerDsn)
+    public function homepage(ArticleRepository $repository)
     {
-        dump($mailerDsn);die;
         $articles = $repository->findAllPublishedOrderedByNewest();
 
         return $this->render('article/homepage.html.twig', [
