diff --git a/config/secrets/prod/prod.MAILER_DSN.97e321.php b/config/secrets/prod/prod.MAILER_DSN.97e321.php
new file mode 100644
index 000000000..e529cb112
--- /dev/null
+++ b/config/secrets/prod/prod.MAILER_DSN.97e321.php
@@ -0,0 +1,3 @@
+<?php // prod.MAILER_DSN.97e321 on Thu, 09 Jan 2020 19:19:08 +0000
+
+return "\x2F\x92\x11_\x14\x23N\xA1\xF9KV\xFF\xCC\x3F\x13\xAAo\xB2\x81k\x21H\x3Bu\x99\x15A\xDC\xF9\x3FE8O\xC4\xEE\xC3\xDF\xAE\xD2\x81wO\x86\xFC\x0C\x94\x8D\x26\xE80q\x19N\xFD\xA9\xF1\x99\x04\xDB\xC3\xDCY1\xCEic\x5B\x16\x40\xD5p\x3B\xA9\x3A\x1E";
diff --git a/config/secrets/prod/prod.encrypt.public.php b/config/secrets/prod/prod.encrypt.public.php
new file mode 100644
index 000000000..77eafe544
--- /dev/null
+++ b/config/secrets/prod/prod.encrypt.public.php
@@ -0,0 +1,3 @@
+<?php // prod.encrypt.public on Thu, 09 Jan 2020 19:19:08 +0000
+
+return "\x3CiY\x80\xB0\xB3\x04\xAA\xC9pk\xDB\x98\xEA\xA5\x8E\x07\x3CF\x84\x16D\xA5\x29\xF8\x1C\xF31\xDD\xCC\xAE\x13";
diff --git a/config/secrets/prod/prod.list.php b/config/secrets/prod/prod.list.php
new file mode 100644
index 000000000..d3c920aef
--- /dev/null
+++ b/config/secrets/prod/prod.list.php
@@ -0,0 +1,5 @@
+<?php
+
+return array (
+  'MAILER_DSN' => NULL,
+);
