diff --git a/src/Service/UploaderHelper.php b/src/Service/UploaderHelper.php
new file mode 100644
index 00000000..674793d3
--- /dev/null
+++ b/src/Service/UploaderHelper.php
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Service;
+
+use Gedmo\Sluggable\Util\Urlizer;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+
+class UploaderHelper
+{
+    public function uploadArticleImage(UploadedFile $uploadedFile): string
+    {
+        $destination = $this->getParameter('kernel.project_dir') . '/public/uploads/article_images';
+
+        $originalFilename = pathinfo($uploadedFile->getClientOriginalName(), PATHINFO_FILENAME);
+        $newFilename = Urlizer::urlize($originalFilename) . '-' . uniqid('', true) . '.' . $uploadedFile->guessExtension();
+
+        $uploadedFile->move(
+            $destination,
+            $newFilename
+        );
+    }
+}
