diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index ef7e0830..ecd4203b 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -2,6 +2,7 @@
 
 namespace App\Controller;
 
+use App\Service\MarkdownHelper;
 use Michelf\MarkdownInterface;
 use Psr\Log\LoggerInterface;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
@@ -24,7 +25,7 @@ class ArticleController extends AbstractController
     /**
      * @Route("/news/{slug}", name="article_show")
      */
-    public function show($slug, MarkdownInterface $markdown, AdapterInterface $cache)
+    public function show($slug, MarkdownInterface $markdown, AdapterInterface $cache, MarkdownHelper $markdownHelper)
     {
         $comments = [
             'I ate a normal rock once. It did NOT taste like bacon!',
@@ -51,12 +52,7 @@ cow est ribeye adipisicing. Pig hamburger pork belly enim. Do porchetta minim ca
 fugiat.
 EOF;
 
-        $item = $cache->getItem('markdown_'.md5($articleContent));
-        if (!$item->isHit()) {
-            $item->set($markdown->transform($articleContent));
-            $cache->save($item);
-        }
-        $articleContent = $item->get();
+        $articleContent = $markdownHelper->parse($articleContent);
 
         return $this->render('article/show.html.twig', [
             'title' => ucwords(str_replace('-', ' ', $slug)),
diff --git a/src/Service/MarkdownHelper.php b/src/Service/MarkdownHelper.php
new file mode 100644
index 00000000..860eafff
--- /dev/null
+++ b/src/Service/MarkdownHelper.php
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Service;
+
+class MarkdownHelper
+{
+    public function parse(string $source): string
+    {
+        $item = $cache->getItem('markdown_'.md5($source));
+        if (!$item->isHit()) {
+            $item->set($markdown->transform($source));
+            $cache->save($item);
+        }
+
+        return $item->get();
+    }
+}
