diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index 7b27bfd..2d7de1d 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -43,7 +43,9 @@ class ArticleController extends AbstractController
     {
         // TODO - actually heart/unheart the article!
 
-        $logger->info('Article is being hearted!');
+        $logger->info('Article is being hearted!', [
+            'slug' => $slug
+        ]);
 
         return new JsonResponse(['hearts' => rand(5, 100)]);
     }
