diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index ecd4203b..dcf0a180 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -52,7 +52,11 @@ cow est ribeye adipisicing. Pig hamburger pork belly enim. Do porchetta minim ca
 fugiat.
 EOF;
 
-        $articleContent = $markdownHelper->parse($articleContent);
+        $articleContent = $markdownHelper->parse(
+            $articleContent,
+            $cache,
+            $markdown
+        );
 
         return $this->render('article/show.html.twig', [
             'title' => ucwords(str_replace('-', ' ', $slug)),
diff --git a/src/Service/MarkdownHelper.php b/src/Service/MarkdownHelper.php
index 860eafff..2d5e017d 100644
--- a/src/Service/MarkdownHelper.php
+++ b/src/Service/MarkdownHelper.php
@@ -2,9 +2,12 @@
 
 namespace App\Service;
 
+use Michelf\MarkdownInterface;
+use Symfony\Component\Cache\Adapter\AdapterInterface;
+
 class MarkdownHelper
 {
-    public function parse(string $source): string
+    public function parse(string $source, AdapterInterface $cache, MarkdownInterface $markdown): string
     {
         $item = $cache->getItem('markdown_'.md5($source));
         if (!$item->isHit()) {
