diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index f27abc1b..cd54acbb 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -22,7 +22,7 @@ class ArticleController extends AbstractController
     /**
      * @Route("/news/{slug}", name="article_show")
      */
-    public function show($slug, Environment $twigEnvironment)
+    public function show($slug)
     {
         $comments = [
             'I ate a normal rock once. It did NOT taste like bacon!',
@@ -30,13 +30,11 @@ class ArticleController extends AbstractController
             'I like bacon too! Buy some from my site! bakinsomebacon.com',
         ];
 
-        $html = $twigEnvironment->render('article/show.html.twig', [
+        return $this->render('article/show.html.twig', [
             'title' => ucwords(str_replace('-', ' ', $slug)),
             'slug' => $slug,
             'comments' => $comments,
         ]);
-
-        return new Response($html);
     }
 
     /**
