diff --git a/webpack.config.js b/webpack.config.js
index 84014087..10eb54b9 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -30,8 +30,8 @@ Encore
 
     // will require an extra script tag for runtime.js
     // but, you probably want this, unless you're building a single-page app
-    //.enableSingleRuntimeChunk()
-    .disableSingleRuntimeChunk()
+    .enableSingleRuntimeChunk()
+    //.disableSingleRuntimeChunk()
 
     /*
      * FEATURE CONFIG
