diff --git a/templates/base.html.twig b/templates/base.html.twig
index 5a777016..70df7253 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -14,6 +14,12 @@
     </head>
 
     <body>
+        {% if is_granted('ROLE_PREVIOUS_ADMIN') %}
+            <div class="alert alert-warning" style="margin-bottom: 0;">
+                You are currently switched to this user.
+                <a href="{{ path('app_homepage', {'_switch_user': '_exit'}) }}">Exit Impersonation</a>
+            </div>
+        {% endif %}
         <nav class="navbar navbar-expand-lg navbar-dark navbar-bg mb-5">
             <a style="margin-left: 75px;" class="navbar-brand space-brand" href="{{ path('app_homepage') }}">The Space Bar</a>
             <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
