diff --git a/src/DataFixtures/UserFixture.php b/src/DataFixtures/UserFixture.php
index c9e4cee2..6f8974cd 100644
--- a/src/DataFixtures/UserFixture.php
+++ b/src/DataFixtures/UserFixture.php
@@ -22,6 +22,7 @@ class UserFixture extends BaseFixture
             $user = new User();
             $user->setEmail(sprintf('spacebar%d@example.com', $i));
             $user->setFirstName($this->faker->firstName);
+            $user->agreeToTerms();
 
             if ($this->faker->boolean) {
                 $user->setTwitterUsername($this->faker->userName);
@@ -45,6 +46,7 @@ class UserFixture extends BaseFixture
             $user->setEmail(sprintf('admin%d@thespacebar.com', $i));
             $user->setFirstName($this->faker->firstName);
             $user->setRoles(['ROLE_ADMIN']);
+            $user->agreeToTerms();
 
             $user->setPassword($this->passwordEncoder->encodePassword(
                 $user,
