diff --git a/src/Controller/SecurityController.php b/src/Controller/SecurityController.php
index fdf81e82..0a2418b0 100644
--- a/src/Controller/SecurityController.php
+++ b/src/Controller/SecurityController.php
@@ -54,6 +54,10 @@ class SecurityController extends AbstractController
                 $user,
                 $form['plainPassword']->getData()
             ));
+            // be absolutely sure they agree
+            if (true === $form['agreeTerms']->getData()) {
+                $user->agreeToTerms();
+            }
 
             $em = $this->getDoctrine()->getManager();
             $em->persist($user);
diff --git a/src/Entity/User.php b/src/Entity/User.php
index 60364b7d..101c0eb5 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -262,10 +262,8 @@ class User implements UserInterface
         return $this->agreedTermsAt;
     }
 
-    public function setAgreedTermsAt(\DateTimeInterface $agreedTermsAt): self
+    public function agreeToTerms()
     {
-        $this->agreedTermsAt = $agreedTermsAt;
-
-        return $this;
+        $this->agreedTermsAt = new \DateTime();
     }
 }
diff --git a/src/Form/UserRegistrationFormType.php b/src/Form/UserRegistrationFormType.php
index 98689183..670b54e9 100644
--- a/src/Form/UserRegistrationFormType.php
+++ b/src/Form/UserRegistrationFormType.php
@@ -8,6 +8,7 @@ use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
 use Symfony\Component\Form\Extension\Core\Type\PasswordType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\IsTrue;
 use Symfony\Component\Validator\Constraints\Length;
 use Symfony\Component\Validator\Constraints\NotBlank;
 
@@ -31,7 +32,14 @@ class UserRegistrationFormType extends AbstractType
                     ])
                 ]
             ])
-            ->add('agreeTerms', CheckboxType::class)
+            ->add('agreeTerms', CheckboxType::class, [
+                'mapped' => false,
+                'constraints' => [
+                    new IsTrue([
+                        'message' => 'I know, it\'s silly, but you must agree to our terms.'
+                    ])
+                ]
+            ])
         ;
     }
 
