diff --git a/src/Entity/User.php b/src/Entity/User.php
index dccf8f74..60364b7d 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -66,6 +66,11 @@ class User implements UserInterface
      */
     private $articles;
 
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $agreedTermsAt;
+
     public function __construct()
     {
         $this->apiTokens = new ArrayCollection();
@@ -251,4 +256,16 @@ class User implements UserInterface
     {
         return $this->getFirstName();
     }
+
+    public function getAgreedTermsAt(): ?\DateTimeInterface
+    {
+        return $this->agreedTermsAt;
+    }
+
+    public function setAgreedTermsAt(\DateTimeInterface $agreedTermsAt): self
+    {
+        $this->agreedTermsAt = $agreedTermsAt;
+
+        return $this;
+    }
 }
diff --git a/src/Form/UserRegistrationFormType.php b/src/Form/UserRegistrationFormType.php
index 449ea592..98689183 100644
--- a/src/Form/UserRegistrationFormType.php
+++ b/src/Form/UserRegistrationFormType.php
@@ -4,6 +4,7 @@ namespace App\Form;
 
 use App\Entity\User;
 use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
 use Symfony\Component\Form\Extension\Core\Type\PasswordType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
@@ -29,7 +30,8 @@ class UserRegistrationFormType extends AbstractType
                         'minMessage' => 'Come on, you can think of a password longer than that!'
                     ])
                 ]
-            ]);
+            ])
+            ->add('agreeTerms', CheckboxType::class)
         ;
     }
 
diff --git a/src/Migrations/Version20181016183947.php b/src/Migrations/Version20181016183947.php
new file mode 100644
index 00000000..aa8082a6
--- /dev/null
+++ b/src/Migrations/Version20181016183947.php
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20181016183947 extends AbstractMigration
+{
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE user ADD agreed_terms_at DATETIME NOT NULL');
+    }
+
+    public function down(Schema $schema) : void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE user DROP agreed_terms_at');
+    }
+}
