diff --git a/tests/Service/MailerTest.php b/tests/Service/MailerTest.php
index 87ed9247..cbebe14f 100644
--- a/tests/Service/MailerTest.php
+++ b/tests/Service/MailerTest.php
@@ -2,12 +2,30 @@
 
 namespace App\Tests\Service;
 
+use App\Entity\User;
+use App\Service\Mailer;
+use Knp\Snappy\Pdf;
 use PHPUnit\Framework\TestCase;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\WebpackEncoreBundle\Asset\EntrypointLookupInterface;
+use Twig\Environment;
 
-class MailerTest extends TestCase
+class MailerTest extends WebTestCase
 {
-    public function testSomething()
+    public function testSendingWelcomeMessage()
     {
-        $this->assertTrue(true);
+        self::bootKernel();
+        $symfonyMailer = self::$container->get(MailerInterface::class);
+        $pdf = self::$container->get(Pdf::class);
+        $twig = self::$container->get(Environment::class);
+        $entrypointLookup = self::$container->get(EntrypointLookupInterface::class);
+
+        $user = new User();
+        $user->setFirstName('Victor');
+        $user->setEmail('victor@symfonycasts.com');
+
+        $mailer = new Mailer($symfonyMailer, $twig, $pdf, $entrypointLookup);
+        $mailer->sendWelcomeMessage($user);
     }
 }
