diff --git a/src/Service/Mailer.php b/src/Service/Mailer.php
index b92b9db1c..8a4871c64 100644
--- a/src/Service/Mailer.php
+++ b/src/Service/Mailer.php
@@ -42,10 +42,10 @@ class Mailer
 
     public function sendAuthorWeeklyReportMessage(User $author, array $articles)
     {
-        $this->entrypointLookup->reset();
         $html = $this->twig->render('email/author-weekly-report-pdf.html.twig', [
             'articles' => $articles,
         ]);
+        $this->entrypointLookup->reset();
         $pdf = $this->pdf->getOutputFromHtml($html);
 
         $email = (new TemplatedEmail())
