diff --git a/templates/article/homepage.html.twig b/templates/article/homepage.html.twig
index e5309599..bb827617 100644
--- a/templates/article/homepage.html.twig
+++ b/templates/article/homepage.html.twig
@@ -54,7 +54,7 @@
                 </div>
 
                 <div class="quote-space pb-2 pt-2 px-5">
-                    <h3 class="text-center pb-3">Trending Quotes</h3>
+                    <h3 class="text-center pb-3">{{ 'Trending Quotes'|trans }}</h3>
                     <p><i class="fa fa-comment"></i> "Our two greatest problems are gravity and paperwork. We can lick gravity, but sometimes the paperwork is overwhelming." <br>— <a href="https://en.wikipedia.org/wiki/Wernher_von_Braun">Wernher von Braun, Rocket Engineer</a></p>
 
                     <p class="pt-4"><i class="fa fa-comment"></i> "Let's face it, space is a risky business. I always considered every launch a barely controlled explosion." <br>— <a href="https://en.wikipedia.org/wiki/Aaron_Cohen_(Deputy_NASA_administrator)">Aaron Cohen, NASA Administrator</a></p>
