diff --git a/src/Controller/SecurityController.php b/src/Controller/SecurityController.php
index 642ff88..1e6d896 100644
--- a/src/Controller/SecurityController.php
+++ b/src/Controller/SecurityController.php
@@ -6,6 +6,7 @@ use App\Entity\User;
 use App\Form\Model\UserRegistrationFormModel;
 use App\Form\UserRegistrationFormType;
 use App\Security\LoginFormAuthenticator;
+use Symfony\Bridge\Twig\Mime\TemplatedEmail;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Mailer\MailerInterface;
@@ -71,12 +72,11 @@ class SecurityController extends AbstractController
             $em->persist($user);
             $em->flush();
 
-            $email = (new Email())
+            $email = (new TemplatedEmail())
                 ->from('alienmailcarrier@example.com')
                 ->to($user->getEmail())
                 ->subject('Welcome to the Space Bar!')
-                ->text("Nice to meet you {$user->getFirstName()}! ❤️")
-                ->html("<h1>Nice to meet you {$user->getFirstName()}! ❤️</h1>");
+                ->htmlTemplate('email/welcome.html.twig');
 
             $mailer->send($email);
 
diff --git a/templates/email/welcome.html.twig b/templates/email/welcome.html.twig
new file mode 100644
index 0000000..6d363ce
--- /dev/null
+++ b/templates/email/welcome.html.twig
@@ -0,0 +1,97 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Welcome to the Space Bar!</title>
+    <style>
+        body {
+            margin: 0;
+            padding: 0;
+            background-color: #f3f3f3;
+            font-family: Helvetica, Arial, sans-serif;
+        }
+        h1 {
+            background-color: #264459;
+            color: #ffffff;
+            padding: 30px 0 50px 0;
+            font-weight: normal;
+        }
+        hr {
+            border: none;
+            border-top: 3px solid #264459;
+            margin: 20px;
+        }
+        .container {
+            background-color: #fefefe;
+            width: 580px;
+            margin: 0 auto;
+        }
+        .bottom {
+            background-color: #efefee;
+        }
+        .block {
+            margin: 0;
+            padding: 10px 20px 20px 20px;
+        }
+        .logo {
+            width: 100%;
+        }
+        .text-center {
+            text-align: center;
+        }
+        .btn {
+            display: inline-block;
+            padding: 10px 20px;
+            background-color: #264459;
+            color: #fefefe;
+            border: 1px solid #fff;
+            border-radius: 3px;
+            font-size: 20px;
+            font-weight: bold;
+            text-decoration: none;
+        }
+    </style>
+</head>
+<body>
+<div class="body">
+    <div class="container">
+        <div class="header text-center">
+            <a href="#homepage">
+                <img src="path/to/logo.png" class="logo" alt="SpaceBar Logo">
+            </a>
+        </div>
+        <div class="content">
+            <h1 class="text-center">Nice to meet you %name%!</h1>
+            <p class="block">
+                Welcome to <strong>the Space Bar</strong>, we can't wait to read what you have to write.
+                Get started on your first article and connect with the space bar community.
+            </p>
+            <p class="block text-center">
+                <a href="#new" class="btn">Get writing!</a>
+            </p>
+            <p class="block">
+                Check out our existing articles and share your thoughts in the comments!
+            </p>
+            <p class="block text-center">
+                <a href="#homepage" class="btn">Get reading!</a>
+            </p>
+            <p class="block">
+                We're so excited that you've decided to join us in our corner of the universe,
+                it's a friendly one with other creative and insightful writers just like you!
+                Need help from a friend? We're always just a message away.
+            </p>
+        </div>
+        <div class="footer">
+            <p class="block">
+                Cheers,<br><br>
+                Your friendly <em>Space Bar Team</em>
+            </p>
+            <p class="block bottom text-center">
+                Sent with ❤️ from the friendly folks at The Space Bar
+            </p>
+        </div>
+    </div>
+</div>
+</body>
+</html>
