diff --git a/src/Twig/AppExtension.php b/src/Twig/AppExtension.php
index 3049b492..c29342f1 100644
--- a/src/Twig/AppExtension.php
+++ b/src/Twig/AppExtension.php
@@ -11,19 +11,12 @@ class AppExtension extends AbstractExtension
     public function getFilters(): array
     {
         return [
-            new TwigFilter('filter_name', [$this, 'doSomething'], ['is_safe' => ['html']]),
+            new TwigFilter('cached_markdown', [$this, 'processMarkdown'], ['is_safe' => ['html']]),
         ];
     }
 
-    public function getFunctions(): array
+    public function processMarkdown($value)
     {
-        return [
-            new TwigFunction('function_name', [$this, 'doSomething']),
-        ];
-    }
-
-    public function doSomething($value)
-    {
-        // ...
+        return strtoupper($value);
     }
 }
diff --git a/templates/article/show.html.twig b/templates/article/show.html.twig
index 66663311..56c358cb 100644
--- a/templates/article/show.html.twig
+++ b/templates/article/show.html.twig
@@ -26,7 +26,7 @@
                 <div class="row">
                     <div class="col-sm-12">
                         <div class="article-text">
-                            {{ article.content|markdown }}
+                            {{ article.content|cached_markdown }}
                         </div>
                     </div>
                 </div>
