diff --git a/templates/article/homepage.html.twig b/templates/article/homepage.html.twig
index e05c2087..4eac2c24 100644
--- a/templates/article/homepage.html.twig
+++ b/templates/article/homepage.html.twig
@@ -33,7 +33,7 @@
                             {% endfor %}
                             <br>
                             <span class="align-left article-details"><img class="article-author-img rounded-circle" src="{{ asset('images/alien-profile.png') }}"> {{ article.author }} </span>
-                            <span class="pl-5 article-details float-right"> {{ article.publishedAt ? article.publishedAt|ago }}</span>
+                            <span class="ml-5 article-details float-right" title="{{ article.publishedAt ? article.publishedAt|localizeddate }}" data-toggle="tooltip"> {{ article.publishedAt ? article.publishedAt|ago }}</span>
                         </div>
                     </a>
                 </div>
