diff --git a/templates/article/show.html.twig b/templates/article/show.html.twig
index 56c358cb..e44bd775 100644
--- a/templates/article/show.html.twig
+++ b/templates/article/show.html.twig
@@ -15,7 +15,9 @@
                             <span class="show-article-title ">{{ article.title }}</span>
                             <br>
                             <span class="align-left article-details"><img class="article-author-img rounded-circle" src="{{ asset('images/alien-profile.png') }}"> Mike Ferengi </span>
-                            <span class="pl-2 article-details"> 4 hours ago</span>
+                            <span class="pl-2 article-details">
+                                {{ article.publishedAt ? article.publishedAt|date('Y-m-d') : 'unpublished' }}
+                            </span>
                             <span class="pl-2 article-details">
                                 <span class="js-like-article-count">5</span>
                                 <a href="{{ path('article_toggle_heart', {slug: article.slug}) }}" class="fa fa-heart-o like-article js-like-article"></a>
