diff --git a/src/Twig/AppExtension.php b/src/Twig/AppExtension.php
index ff8d9a88..05111e89 100644
--- a/src/Twig/AppExtension.php
+++ b/src/Twig/AppExtension.php
@@ -3,17 +3,19 @@
 namespace App\Twig;
 
 use App\Service\MarkdownHelper;
+use Psr\Container\ContainerInterface;
+use Symfony\Component\DependencyInjection\ServiceSubscriberInterface;
 use Twig\Extension\AbstractExtension;
 use Twig\TwigFilter;
 use Twig\TwigFunction;
 
-class AppExtension extends AbstractExtension
+class AppExtension extends AbstractExtension implements ServiceSubscriberInterface
 {
-    private $markdownHelper;
+    private $container;
 
-    public function __construct(MarkdownHelper $markdownHelper)
+    public function __construct(ContainerInterface $container)
     {
-        $this->markdownHelper = $markdownHelper;
+        $this->container = $container;
     }
 
     public function getFilters(): array
@@ -25,6 +27,15 @@ class AppExtension extends AbstractExtension
 
     public function processMarkdown($value)
     {
-        return $this->markdownHelper->parse($value);
+        return $this->container
+            ->get(MarkdownHelper::class)
+            ->parse($value);
+    }
+
+    public static function getSubscribedServices()
+    {
+        return [
+            MarkdownHelper::class,
+        ];
     }
 }
