diff --git a/src/Twig/AppExtension.php b/src/Twig/AppExtension.php
new file mode 100644
index 00000000..3049b492
--- /dev/null
+++ b/src/Twig/AppExtension.php
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+use Twig\TwigFunction;
+
+class AppExtension extends AbstractExtension
+{
+    public function getFilters(): array
+    {
+        return [
+            new TwigFilter('filter_name', [$this, 'doSomething'], ['is_safe' => ['html']]),
+        ];
+    }
+
+    public function getFunctions(): array
+    {
+        return [
+            new TwigFunction('function_name', [$this, 'doSomething']),
+        ];
+    }
+
+    public function doSomething($value)
+    {
+        // ...
+    }
+}
