diff --git a/src/Entity/Article.php b/src/Entity/Article.php
index 69e90ab3..45f472f2 100644
--- a/src/Entity/Article.php
+++ b/src/Entity/Article.php
@@ -140,4 +140,9 @@ class Article
 
         return $this;
     }
+
+    public function getImagePath()
+    {
+        return 'images/'.$this->getImageFilename();
+    }
 }
diff --git a/templates/article/homepage.html.twig b/templates/article/homepage.html.twig
index e8cddf2c..8d49b567 100644
--- a/templates/article/homepage.html.twig
+++ b/templates/article/homepage.html.twig
@@ -21,7 +21,7 @@
                 {% for article in articles %}
                 <div class="article-container my-1">
                     <a href="{{ path('article_show', {slug: article.slug}) }}">
-                        <img class="article-img" src="{{ asset('images/asteroid.jpeg') }}">
+                        <img class="article-img" src="{{ asset(article.imagePath) }}">
                         <div class="article-title d-inline-block pl-3 align-middle">
                             <span>{{ article.title }}</span>
                             <br>
diff --git a/templates/article/show.html.twig b/templates/article/show.html.twig
index a08bbb0e..9e20312a 100644
--- a/templates/article/show.html.twig
+++ b/templates/article/show.html.twig
@@ -10,7 +10,7 @@
             <div class="show-article-container p-3 mt-4">
                 <div class="row">
                     <div class="col-sm-12">
-                        <img class="show-article-img" src="{{ asset('images/asteroid.jpeg') }}">
+                        <img class="show-article-img" src="{{ asset(article.imagePath) }}">
                         <div class="show-article-title-container d-inline-block pl-3 align-middle">
                             <span class="show-article-title ">{{ article.title }}</span>
                             <br>
