diff --git a/src/Entity/User.php b/src/Entity/User.php
index 2b7dee68..d3eebbe8 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -143,4 +143,15 @@ class User implements UserInterface
 
         return $this;
     }
+
+    public function getAvatarUrl(int $size = null): string
+    {
+        $url = 'https://robohash.org/'.$this->getEmail();
+
+        if ($size) {
+            $url .= sprintf('?size=%dx%d', $size, $size);
+        }
+
+        return $url;
+    }
 }
diff --git a/templates/account/index.html.twig b/templates/account/index.html.twig
index 263ad4d7..21c63ab5 100644
--- a/templates/account/index.html.twig
+++ b/templates/account/index.html.twig
@@ -15,7 +15,7 @@
                 <div class="row spacepurplebg white">
                     <div class="col-md-2 no-pad">
                         <div class="user-image">
-                            <img src="https://robohash.org/{{ app.user.email }}" class="img-responsive thumbnail">
+                            <img src="{{ app.user.avatarUrl }}" class="img-responsive thumbnail">
                         </div>
                     </div>
                     <div class="col-md-10 no-pad">
diff --git a/templates/base.html.twig b/templates/base.html.twig
index 248604d8..1b48f0c9 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -36,7 +36,7 @@
                     {% if is_granted('ROLE_USER') %}
                         <li class="nav-item dropdown" style="margin-right: 75px;">
                             <a class="nav-link dropdown-toggle" href="http://example.com" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                              <img class="nav-profile-img rounded-circle" src="https://robohash.org/{{ app.user.email }}">
+                              <img class="nav-profile-img rounded-circle" src="{{ app.user.avatarUrl(100) }}">
                             </a>
                             <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
                                 <a class="dropdown-item" href="{{ path('app_account') }}">Profile</a>
