diff --git a/src/Controller/AccountController.php b/src/Controller/AccountController.php
index 42e56eda..b9026f91 100644
--- a/src/Controller/AccountController.php
+++ b/src/Controller/AccountController.php
@@ -4,13 +4,12 @@ namespace App\Controller;
 
 use Psr\Log\LoggerInterface;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
-use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\Routing\Annotation\Route;
 
 /**
  * @IsGranted("ROLE_USER")
  */
-class AccountController extends AbstractController
+class AccountController extends BaseController
 {
     /**
      * @Route("/account", name="app_account")
diff --git a/src/Controller/BaseController.php b/src/Controller/BaseController.php
new file mode 100644
index 00000000..21714f5e
--- /dev/null
+++ b/src/Controller/BaseController.php
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\User;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+abstract class BaseController extends AbstractController
+{
+    protected function getUser(): User
+    {
+        return parent::getUser();
+    }
+}
\ No newline at end of file
