diff --git a/src/DataFixtures/UserFixture.php b/src/DataFixtures/UserFixture.php
index cfb03741..9277a3da 100644
--- a/src/DataFixtures/UserFixture.php
+++ b/src/DataFixtures/UserFixture.php
@@ -2,14 +2,20 @@
 
 namespace App\DataFixtures;
 
+use App\Entity\User;
 use Doctrine\Common\Persistence\ObjectManager;
 
 class UserFixture extends BaseFixture
 {
     protected function loadData(ObjectManager $manager)
     {
-        // $product = new Product();
-        // $manager->persist($product);
+        $this->createMany(10, 'main_users', function($i) {
+            $user = new User();
+            $user->setEmail(sprintf('spacebar%d@example.com', $i));
+            $user->setFirstName($this->faker->firstName);
+
+            return $user;
+        });
 
         $manager->flush();
     }
