diff --git a/src/Controller/AccountController.php b/src/Controller/AccountController.php
index e191f776..42e56eda 100644
--- a/src/Controller/AccountController.php
+++ b/src/Controller/AccountController.php
@@ -2,6 +2,7 @@
 
 namespace App\Controller;
 
+use Psr\Log\LoggerInterface;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\Routing\Annotation\Route;
@@ -14,9 +15,9 @@ class AccountController extends AbstractController
     /**
      * @Route("/account", name="app_account")
      */
-    public function index()
+    public function index(LoggerInterface $logger)
     {
-        dd($this->getUser()->getFirstName());
+        $logger->debug('Checking account page for '.$this->getUser()->getEmail());
         return $this->render('account/index.html.twig', [
 
         ]);
