diff --git a/src/Entity/User.php b/src/Entity/User.php
index b7ed9d40..7894f726 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -27,6 +27,11 @@ class User implements UserInterface
      */
     private $roles = [];
 
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $firstName;
+
     public function getId(): ?int
     {
         return $this->id;
@@ -97,4 +102,16 @@ class User implements UserInterface
         // If you store any temporary, sensitive data on the user, clear it here
         // $this->plainPassword = null;
     }
+
+    public function getFirstName(): ?string
+    {
+        return $this->firstName;
+    }
+
+    public function setFirstName(string $firstName): self
+    {
+        $this->firstName = $firstName;
+
+        return $this;
+    }
 }
