diff --git a/src/DataFixtures/UserFixture.php b/src/DataFixtures/UserFixture.php
index 5a8d46b0..cfb03741 100644
--- a/src/DataFixtures/UserFixture.php
+++ b/src/DataFixtures/UserFixture.php
@@ -2,12 +2,11 @@
 
 namespace App\DataFixtures;
 
-use Doctrine\Bundle\FixturesBundle\Fixture;
 use Doctrine\Common\Persistence\ObjectManager;
 
-class UserFixture extends Fixture
+class UserFixture extends BaseFixture
 {
-    public function load(ObjectManager $manager)
+    protected function loadData(ObjectManager $manager)
     {
         // $product = new Product();
         // $manager->persist($product);
