diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index 37b399723..3d6554503 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -57,7 +57,7 @@ class ArticleFormType extends AbstractType
                 'required' => false,
                 'constraints' => [
                     new Image([
-                        'maxSize' => '5k'
+                        'maxSize' => '5M'
                     ])
                 ]
             ])
