diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index 628c6ffd..671720a8 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -38,6 +38,7 @@ class ArticleFormType extends AbstractType
                 },
                 'placeholder' => 'Choose an author',
                 'choices' => $this->userRepository->findAllEmailAlphabetical(),
+                'invalid_message' => 'Symfony is too smart for your hacking!'
             ])
         ;
     }
