diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index 52cfa20..52d508a 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -17,6 +17,7 @@ use Symfony\Component\Form\FormEvents;
 use Symfony\Component\Form\FormInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 use Symfony\Component\Validator\Constraints\Image;
+use Symfony\Component\Validator\Constraints\NotNull;
 
 class ArticleFormType extends AbstractType
 {
@@ -59,6 +60,13 @@ class ArticleFormType extends AbstractType
                 'maxSize' => '5M'
             ])
         ];
+
+        if (!$isEdit || !$article->getImageFilename()) {
+            $imageConstraints[] = new NotNull([
+                'message' => 'Please upload an image',
+            ]);
+        }
+
         $builder
             ->add('imageFile', FileType::class, [
                 'mapped' => false,
