diff --git a/src/Form/ArticleFormType.php b/src/Form/ArticleFormType.php
index 4ad93341..37b39972 100644
--- a/src/Form/ArticleFormType.php
+++ b/src/Form/ArticleFormType.php
@@ -56,7 +56,9 @@ class ArticleFormType extends AbstractType
                 'mapped' => false,
                 'required' => false,
                 'constraints' => [
-                    new Image()
+                    new Image([
+                        'maxSize' => '5k'
+                    ])
                 ]
             ])
         ;
