diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 987d717e..f661c9da 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -31,9 +31,7 @@ class ArticleAdminController extends AbstractController
      */
     public function edit(Article $article)
     {
-        if (!$this->isGranted('MANAGE', $article)) {
-            throw $this->createAccessDeniedException('No access!');
-        }
+        $this->denyAccessUnlessGranted('MANAGE', $article);
 
         dd($article);
     }
