diff --git a/src/Entity/Article.php b/src/Entity/Article.php
index 7ae4db6d..bb25b938 100644
--- a/src/Entity/Article.php
+++ b/src/Entity/Article.php
@@ -45,11 +45,6 @@ class Article
      */
     private $publishedAt;
 
-    /**
-     * @ORM\Column(type="string", length=255)
-     */
-    private $author;
-
     /**
      * @ORM\Column(type="integer")
      */
@@ -131,18 +126,6 @@ class Article
         return $this;
     }
 
-    public function getAuthor(): ?string
-    {
-        return $this->author;
-    }
-
-    public function setAuthor(string $author): self
-    {
-        $this->author = $author;
-
-        return $this;
-    }
-
     public function getHeartCount(): ?int
     {
         return $this->heartCount;
diff --git a/src/Migrations/Version20180901184240.php b/src/Migrations/Version20180901184240.php
new file mode 100644
index 00000000..1737d174
--- /dev/null
+++ b/src/Migrations/Version20180901184240.php
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20180901184240 extends AbstractMigration
+{
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE article DROP author');
+    }
+
+    public function down(Schema $schema) : void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE article ADD author VARCHAR(255) NOT NULL COLLATE utf8mb4_unicode_ci');
+    }
+}
