diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 7e177471..987d717e 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -31,7 +31,7 @@ class ArticleAdminController extends AbstractController
      */
     public function edit(Article $article)
     {
-        if ($article->getAuthor() != $this->getUser() && !$this->isGranted('ROLE_ADMIN_ARTICLE')) {
+        if (!$this->isGranted('MANAGE', $article)) {
             throw $this->createAccessDeniedException('No access!');
         }
 
