diff --git a/src/Controller/ArticleAdminController.php b/src/Controller/ArticleAdminController.php
index 455e672d..b3f83827 100644
--- a/src/Controller/ArticleAdminController.php
+++ b/src/Controller/ArticleAdminController.php
@@ -27,4 +27,12 @@ class ArticleAdminController extends AbstractController
             $article->getSlug()
         ));
     }
+
+    /**
+     * @Route("/admin/article/{id}/edit")
+     */
+    public function edit(Article $article)
+    {
+        dd($article);
+    }
 }
