diff --git a/templates/email/emailBase.html.twig b/templates/email/emailBase.html.twig
index 2edeed117..401ac0188 100644
--- a/templates/email/emailBase.html.twig
+++ b/templates/email/emailBase.html.twig
@@ -1,4 +1,4 @@
-{% apply inky_to_html|inline_css(source('@styles/foundation-emails.css'), source('@styles/email.css')) %}
+{% apply inky_to_html|inline_css(encore_entry_css_source('email')) %}
     <container>
         <row class="header">
             <columns>
